<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Artemis S. Louyakis" />


<title>hvo_mating</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/bootstrap.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>




<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">My Website</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="hvo_mating.html">Hvo mating</a>
</li>
<li>
  <a href="manhattan_agl.html">agl manhattan</a>
</li>
<li>
  <a href="manhattan_crisprs.html">crispr manhattan</a>
</li>
<li>
  <a href="manhattan_transposases.html">transposases manhattan</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">hvo_mating</h1>
<h4 class="author">Artemis S. Louyakis</h4>
<h4 class="date">6/8/2020</h4>

</div>


<p><em>Haloferax volcanii</em> was grown over 24 hours in five increments - 0, 2, 4, 8, and 24 - with a non-mating, shaking control, p.</p>
<p>Programs: <a href="https://genome.sph.umich.edu/wiki/Code_Sample:_Generating_Manhattan_Plots_in_R" class="uri">https://genome.sph.umich.edu/wiki/Code_Sample:_Generating_Manhattan_Plots_in_R</a></p>
<div id="import-count-matrix" class="section level3">
<h3>Import count matrix</h3>
</div>
<div id="prepare-sample-metadata" class="section level3">
<h3>Prepare sample metadata</h3>
</div>
<div id="construct-annotation-map" class="section level3">
<h3>Construct annotation map</h3>
<pre><code>## Warning: Expected 3 pieces. Missing pieces filled with `NA` in 110 rows [21, 87, 212, 305, 344, 430, 491, 497, 501, 551, 553, 594, 627, 628, 639,
## 880, 978, 1048, 1172, 1342, ...].</code></pre>
<pre><code>##  [1] &quot;wp&quot;                   &quot;dbxref&quot;               &quot;locus_tag&quot;            &quot;ncbi_id&quot;              &quot;genomic_accession&quot;    &quot;feature&quot;             
##  [7] &quot;start&quot;                &quot;stop&quot;                 &quot;strand&quot;               &quot;gene&quot;                 &quot;gbkey&quot;                &quot;gene_biotype&quot;        
## [13] &quot;old_locus_tag.x&quot;      &quot;length&quot;               &quot;protein&quot;              &quot;old_locus_tag.y&quot;      &quot;kegg1&quot;                &quot;kegg2&quot;               
## [19] &quot;kegg3&quot;                &quot;gene_id&quot;              &quot;gene_description&quot;     &quot;kegg_id&quot;              &quot;kegg_gene&quot;            &quot;kegg_description&quot;    
## [25] &quot;enzyme&quot;               &quot;keggidprot&quot;           &quot;Gene.Name&quot;            &quot;Species&quot;              &quot;EC_NUMBER&quot;            &quot;GOTERM_BP_DIRECT&quot;    
## [31] &quot;GOTERM_CC_DIRECT&quot;     &quot;GOTERM_MF_DIRECT&quot;     &quot;INTERPRO&quot;             &quot;KEGG_PATHWAY&quot;         &quot;OFFICIAL_GENE_SYMBOL&quot; &quot;PFAM&quot;                
## [37] &quot;PIR_SUPERFAMILY&quot;      &quot;SMART&quot;                &quot;UP_KEYWORDS&quot;          &quot;chr&quot;                  &quot;arCOG_name&quot;           &quot;E.value&quot;             
## [43] &quot;score&quot;                &quot;arCOG_familiy&quot;        &quot;Gene_name&quot;            &quot;Description&quot;          &quot;HVO&quot;</code></pre>
<pre class="r"><code>duplicated(hvo_map$ncbi_id)</code></pre>
<pre><code>##    [1] FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE  TRUE FALSE  TRUE FALSE  TRUE FALSE FALSE FALSE
##   [24] FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE  TRUE FALSE  TRUE FALSE  TRUE FALSE FALSE FALSE FALSE FALSE FALSE FALSE
##   [47] FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE  TRUE  TRUE FALSE
##   [70]  TRUE  TRUE FALSE  TRUE  TRUE FALSE FALSE FALSE FALSE FALSE FALSE  TRUE  TRUE  TRUE  TRUE  TRUE  TRUE  TRUE  TRUE  TRUE  TRUE  TRUE  TRUE
##   [93]  TRUE FALSE FALSE  TRUE  TRUE FALSE FALSE FALSE  TRUE  TRUE  TRUE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE
##  [116] FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE  TRUE FALSE  TRUE FALSE  TRUE  TRUE  TRUE FALSE FALSE
##  [139]  TRUE  TRUE  TRUE FALSE FALSE FALSE FALSE  TRUE  TRUE  TRUE  TRUE  TRUE FALSE  TRUE  TRUE FALSE  TRUE  TRUE FALSE  TRUE  TRUE FALSE  TRUE
##  [162]  TRUE FALSE  TRUE  TRUE FALSE FALSE FALSE FALSE  TRUE  TRUE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE  TRUE  TRUE FALSE FALSE FALSE
##  [185] FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE  TRUE FALSE  TRUE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE
##  [208] FALSE FALSE FALSE FALSE FALSE FALSE  TRUE FALSE FALSE FALSE FALSE FALSE  TRUE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE
##  [231] FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE
##  [254] FALSE FALSE FALSE FALSE FALSE FALSE  TRUE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE  TRUE FALSE FALSE FALSE  TRUE FALSE FALSE FALSE
##  [277] FALSE  TRUE FALSE  TRUE FALSE  TRUE FALSE  TRUE FALSE  TRUE FALSE FALSE FALSE FALSE FALSE FALSE  TRUE FALSE  TRUE FALSE  TRUE FALSE  TRUE
##  [300] FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE  TRUE  TRUE  TRUE  TRUE  TRUE  TRUE FALSE  TRUE  TRUE  TRUE  TRUE FALSE FALSE FALSE
##  [323] FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE  TRUE FALSE FALSE FALSE FALSE
##  [346] FALSE FALSE FALSE  TRUE FALSE FALSE  TRUE FALSE  TRUE FALSE  TRUE FALSE  TRUE FALSE  TRUE FALSE FALSE FALSE FALSE FALSE FALSE FALSE  TRUE
##  [369] FALSE  TRUE FALSE FALSE FALSE  TRUE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE  TRUE FALSE  TRUE  TRUE  TRUE  TRUE FALSE
##  [392] FALSE  TRUE FALSE FALSE  TRUE  TRUE FALSE  TRUE FALSE  TRUE FALSE FALSE FALSE FALSE FALSE FALSE FALSE  TRUE  TRUE  TRUE  TRUE  TRUE  TRUE
##  [415]  TRUE  TRUE  TRUE  TRUE  TRUE FALSE  TRUE FALSE  TRUE FALSE FALSE FALSE FALSE  TRUE FALSE  TRUE FALSE  TRUE FALSE FALSE FALSE FALSE FALSE
##  [438] FALSE  TRUE  TRUE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE  TRUE FALSE FALSE FALSE FALSE FALSE  TRUE  TRUE FALSE FALSE
##  [461] FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE  TRUE FALSE FALSE FALSE  TRUE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE  TRUE FALSE
##  [484] FALSE  TRUE FALSE  TRUE FALSE  TRUE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE  TRUE  TRUE  TRUE FALSE FALSE  TRUE FALSE  TRUE FALSE
##  [507] FALSE FALSE  TRUE FALSE  TRUE FALSE  TRUE FALSE  TRUE FALSE FALSE  TRUE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE
##  [530] FALSE FALSE FALSE FALSE FALSE FALSE FALSE  TRUE  TRUE  TRUE  TRUE FALSE FALSE FALSE FALSE  TRUE FALSE  TRUE  TRUE  TRUE  TRUE FALSE FALSE
##  [553] FALSE FALSE  TRUE FALSE FALSE FALSE  TRUE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE  TRUE FALSE FALSE FALSE FALSE
##  [576] FALSE FALSE FALSE FALSE FALSE FALSE  TRUE  TRUE FALSE  TRUE  TRUE FALSE  TRUE  TRUE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE  TRUE
##  [599] FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE  TRUE FALSE  TRUE FALSE  TRUE FALSE  TRUE
##  [622] FALSE  TRUE FALSE FALSE FALSE FALSE FALSE FALSE  TRUE FALSE FALSE FALSE FALSE FALSE  TRUE FALSE FALSE FALSE FALSE FALSE  TRUE FALSE  TRUE
##  [645] FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE  TRUE FALSE  TRUE FALSE FALSE FALSE FALSE FALSE FALSE FALSE
##  [668] FALSE FALSE  TRUE  TRUE FALSE FALSE  TRUE  TRUE FALSE FALSE FALSE FALSE FALSE  TRUE FALSE FALSE  TRUE FALSE FALSE FALSE FALSE  TRUE  TRUE
##  [691] FALSE FALSE FALSE FALSE FALSE FALSE  TRUE  TRUE  TRUE  TRUE FALSE FALSE FALSE FALSE FALSE FALSE FALSE  TRUE  TRUE FALSE FALSE FALSE  TRUE
##  [714]  TRUE  TRUE  TRUE  TRUE FALSE FALSE FALSE FALSE FALSE FALSE  TRUE FALSE  TRUE FALSE FALSE FALSE  TRUE FALSE FALSE FALSE FALSE FALSE FALSE
##  [737] FALSE FALSE FALSE  TRUE  TRUE  TRUE  TRUE  TRUE  TRUE FALSE FALSE FALSE FALSE  TRUE FALSE  TRUE  TRUE  TRUE  TRUE FALSE  TRUE FALSE FALSE
##  [760] FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE  TRUE  TRUE  TRUE  TRUE  TRUE  TRUE  TRUE  TRUE FALSE FALSE FALSE FALSE FALSE FALSE FALSE
##  [783] FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE  TRUE FALSE FALSE FALSE
##  [806] FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE  TRUE  TRUE FALSE  TRUE  TRUE FALSE FALSE FALSE  TRUE FALSE FALSE FALSE
##  [829] FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE  TRUE FALSE FALSE  TRUE FALSE FALSE FALSE  TRUE
##  [852] FALSE FALSE  TRUE  TRUE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE
##  [875]  TRUE FALSE  TRUE FALSE  TRUE FALSE FALSE  TRUE FALSE FALSE FALSE FALSE FALSE FALSE  TRUE FALSE  TRUE FALSE  TRUE FALSE FALSE FALSE  TRUE
##  [898]  TRUE  TRUE FALSE FALSE FALSE  TRUE  TRUE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE
##  [921] FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE  TRUE FALSE
##  [944]  TRUE  TRUE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE
##  [967] FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE  TRUE FALSE FALSE FALSE  TRUE FALSE FALSE FALSE
##  [990] FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE  TRUE  TRUE
##  [ reached getOption(&quot;max.print&quot;) -- omitted 4163 entries ]</code></pre>
<pre class="r"><code>any(duplicated(colnames(count_data)))</code></pre>
<pre><code>## [1] FALSE</code></pre>
<pre class="r"><code># any(duplicated(count_data$ncbi_id))
any(duplicated(rownames(count_data)))</code></pre>
<pre><code>## [1] FALSE</code></pre>
<pre class="r"><code>length(unique(hvo_map$ncbi_id))</code></pre>
<pre><code>## [1] 3964</code></pre>
<pre class="r"><code>length(unique(hvo_map$wp))</code></pre>
<pre><code>## [1] 3828</code></pre>
<pre class="r"><code>length(unique(rownames(count_data_raw)))</code></pre>
<pre><code>## [1] 3941</code></pre>
<pre class="r"><code>length(unique(rownames(count_data)))</code></pre>
<pre><code>## [1] 3582</code></pre>
<pre class="r"><code>#keggmap &lt;- dplyr::filter(keggmap,  !is.na(kegg_id))
#sub.keggmap &lt;- aggregate(keggmap,by=list(keggmap$hvo),head,n=1)
n_occur &lt;- data.frame(table(rownames(count_data)))
n_occur[n_occur$Freq &gt; 1,]</code></pre>
<pre><code>## [1] Var1 Freq
## &lt;0 rows&gt; (or 0-length row.names)</code></pre>
<pre class="r"><code>count_data[rownames(count_data) %in% n_occur$Var1[n_occur$Freq &gt; 1],]</code></pre>
<pre><code>##  [1] ap0_quant a00_quant a02_quant a04_quant a08_quant a24_quant bp0_quant b00_quant b02_quant b04_quant b08_quant b24_quant
## &lt;0 rows&gt; (or 0-length row.names)</code></pre>
<pre class="r"><code># rownames(count_data) &lt;- count_data$ncbi_id
# count_data$ncbi_id &lt;- NULL</code></pre>
</div>
<div id="run-differential-expression-analysis-with-noiseqbio" class="section level3">
<h3>Run differential expression analysis with noiseqbio</h3>
<pre><code>## Warning in dir.create(&quot;noiseq&quot;): &#39;noiseq&#39; already exists</code></pre>
</div>
<div id="volcano-plots" class="section level3">
<h3>Volcano plots</h3>
<pre><code>## [1] &quot;00&quot; &quot;p0&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/volcano%20plots-1.png" width="50%" /></p>
<pre><code>## [1] &quot;00&quot; &quot;02&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/volcano%20plots-2.png" width="50%" /></p>
<pre><code>## [1] &quot;00&quot; &quot;04&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/volcano%20plots-3.png" width="50%" /></p>
<pre><code>## [1] &quot;00&quot; &quot;08&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/volcano%20plots-4.png" width="50%" /></p>
<pre><code>## [1] &quot;00&quot; &quot;24&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/volcano%20plots-5.png" width="50%" /></p>
<pre><code>## [1] &quot;02&quot; &quot;p0&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/volcano%20plots-6.png" width="50%" /></p>
<pre><code>## [1] &quot;02&quot; &quot;04&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/volcano%20plots-7.png" width="50%" /></p>
<pre><code>## [1] &quot;02&quot; &quot;08&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/volcano%20plots-8.png" width="50%" /></p>
<pre><code>## [1] &quot;02&quot; &quot;24&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/volcano%20plots-9.png" width="50%" /></p>
<pre><code>## [1] &quot;04&quot; &quot;p0&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/volcano%20plots-10.png" width="50%" /></p>
<pre><code>## [1] &quot;04&quot; &quot;08&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/volcano%20plots-11.png" width="50%" /></p>
<pre><code>## [1] &quot;04&quot; &quot;24&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/volcano%20plots-12.png" width="50%" /></p>
<pre><code>## [1] &quot;08&quot; &quot;p0&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/volcano%20plots-13.png" width="50%" /></p>
<pre><code>## [1] &quot;08&quot; &quot;24&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/volcano%20plots-14.png" width="50%" /></p>
<pre><code>## [1] &quot;24&quot; &quot;p0&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/volcano%20plots-15.png" width="50%" /> ###Interactive volcano plots labeled with <em>H. volcanii</em> identifier</p>
<pre><code>## [1] &quot;00&quot; &quot;p0&quot;
## [1] &quot;00&quot; &quot;02&quot;
## [1] &quot;00&quot; &quot;04&quot;
## [1] &quot;00&quot; &quot;08&quot;
## [1] &quot;00&quot; &quot;24&quot;
## [1] &quot;02&quot; &quot;p0&quot;
## [1] &quot;02&quot; &quot;04&quot;
## [1] &quot;02&quot; &quot;08&quot;
## [1] &quot;02&quot; &quot;24&quot;
## [1] &quot;04&quot; &quot;p0&quot;
## [1] &quot;04&quot; &quot;08&quot;
## [1] &quot;04&quot; &quot;24&quot;
## [1] &quot;08&quot; &quot;p0&quot;
## [1] &quot;08&quot; &quot;24&quot;
## [1] &quot;24&quot; &quot;p0&quot;</code></pre>
</div>
<div id="interactive-volcano-plotes-labeled-with-kegg-orthologous-heirarchal-categories" class="section level3">
<h3>Interactive volcano plotes labeled with kegg orthologous heirarchal categories</h3>
<pre><code>## [1] &quot;00&quot; &quot;p0&quot;
## [1] &quot;00&quot; &quot;02&quot;
## [1] &quot;00&quot; &quot;04&quot;
## [1] &quot;00&quot; &quot;08&quot;
## [1] &quot;00&quot; &quot;24&quot;
## [1] &quot;02&quot; &quot;p0&quot;
## [1] &quot;02&quot; &quot;04&quot;
## [1] &quot;02&quot; &quot;08&quot;
## [1] &quot;02&quot; &quot;24&quot;
## [1] &quot;04&quot; &quot;p0&quot;
## [1] &quot;04&quot; &quot;08&quot;
## [1] &quot;04&quot; &quot;24&quot;
## [1] &quot;08&quot; &quot;p0&quot;
## [1] &quot;08&quot; &quot;24&quot;
## [1] &quot;24&quot; &quot;p0&quot;</code></pre>
</div>
<div id="principle-component-analysis" class="section level3">
<h3>Principle component analysis</h3>
<embed src="hvo_mating_files/figure-html/pca-1.pdf" width="600px" type="application/pdf" />
<embed src="hvo_mating_files/figure-html/pca-2.pdf" width="600px" type="application/pdf" />
<embed src="hvo_mating_files/figure-html/pca-3.pdf" width="600px" type="application/pdf" />
<pre><code>## pdf 
##   2</code></pre>
<pre><code>## pdf 
##   2</code></pre>
<pre><code>## pdf 
##   2</code></pre>
</div>
<div id="stats" class="section level3">
<h3>Stats</h3>
</div>
<div id="correlation-tests-on-replicates---pearson-and-spearman" class="section level3">
<h3>Correlation tests on replicates - Pearson and Spearman</h3>
<pre><code>##           ap0_quant a00_quant a02_quant a04_quant a08_quant a24_quant bp0_quant b00_quant b02_quant b04_quant b08_quant b24_quant
## ap0_quant 1.0000000 0.9983682 0.9630373 0.9697688 0.9718433 0.9281991 0.9967648 0.9960929 0.9642690 0.9676181 0.9634593 0.8734208
## a00_quant 0.9983682 1.0000000 0.9680972 0.9742907 0.9757129 0.9279349 0.9930267 0.9939833 0.9698671 0.9710704 0.9680990 0.8692608
## a02_quant 0.9630373 0.9680972 1.0000000 0.9940406 0.9811301 0.9221282 0.9534413 0.9608966 0.9990026 0.9887269 0.9765050 0.8529000
## a04_quant 0.9697688 0.9742907 0.9940406 1.0000000 0.9920891 0.9438075 0.9636056 0.9704396 0.9935932 0.9974309 0.9887996 0.8801673
## a08_quant 0.9718433 0.9757129 0.9811301 0.9920891 1.0000000 0.9748164 0.9727446 0.9792258 0.9800030 0.9960012 0.9987643 0.9266369
## a24_quant 0.9281991 0.9279349 0.9221282 0.9438075 0.9748164 1.0000000 0.9451313 0.9513462 0.9175970 0.9607964 0.9775691 0.9853785
## bp0_quant 0.9967648 0.9930267 0.9534413 0.9636056 0.9727446 0.9451313 1.0000000 0.9990981 0.9532253 0.9661029 0.9654620 0.9020862
## b00_quant 0.9960929 0.9939833 0.9608966 0.9704396 0.9792258 0.9513462 0.9990981 1.0000000 0.9603221 0.9733631 0.9729999 0.9068787
## b02_quant 0.9642690 0.9698671 0.9990026 0.9935932 0.9800030 0.9175970 0.9532253 0.9603221 1.0000000 0.9870377 0.9755970 0.8462585
## b04_quant 0.9676181 0.9710704 0.9887269 0.9974309 0.9960012 0.9607964 0.9661029 0.9733631 0.9870377 1.0000000 0.9945229 0.9052150
## b08_quant 0.9634593 0.9680990 0.9765050 0.9887996 0.9987643 0.9775691 0.9654620 0.9729999 0.9755970 0.9945229 1.0000000 0.9314757
## b24_quant 0.8734208 0.8692608 0.8529000 0.8801673 0.9266369 0.9853785 0.9020862 0.9068787 0.8462585 0.9052150 0.9314757 1.0000000</code></pre>
<pre><code>##           ap0_quant a00_quant a02_quant a04_quant a08_quant a24_quant bp0_quant b00_quant b02_quant b04_quant b08_quant b24_quant
## ap0_quant 1.0000000 0.9102327 0.8040093 0.7850077 0.8058361 0.8180921 0.9405341 0.9161027 0.8018557 0.7907638 0.7951065 0.7961772
## a00_quant 0.9102327 1.0000000 0.8576140 0.8500299 0.8641262 0.8596782 0.9065479 0.9464284 0.8596870 0.8518531 0.8532209 0.8465695
## a02_quant 0.8040093 0.8576140 1.0000000 0.9349972 0.9258556 0.8790469 0.7953985 0.8499472 0.9534460 0.9304953 0.9105425 0.8833257
## a04_quant 0.7850077 0.8500299 0.9349972 1.0000000 0.9508534 0.9070316 0.7833358 0.8431677 0.9375753 0.9519754 0.9363643 0.9106234
## a08_quant 0.8058361 0.8641262 0.9258556 0.9508534 1.0000000 0.9246176 0.8043446 0.8568174 0.9298750 0.9435493 0.9456457 0.9277200
## a24_quant 0.8180921 0.8596782 0.8790469 0.9070316 0.9246176 1.0000000 0.8145453 0.8541075 0.8821310 0.9046773 0.9166922 0.9339101
## bp0_quant 0.9405341 0.9065479 0.7953985 0.7833358 0.8043446 0.8145453 1.0000000 0.9271970 0.7977462 0.7878423 0.7888875 0.8022131
## b00_quant 0.9161027 0.9464284 0.8499472 0.8431677 0.8568174 0.8541075 0.9271970 1.0000000 0.8517179 0.8492244 0.8469294 0.8439018
## b02_quant 0.8018557 0.8596870 0.9534460 0.9375753 0.9298750 0.8821310 0.7977462 0.8517179 1.0000000 0.9326793 0.9164104 0.8898089
## b04_quant 0.7907638 0.8518531 0.9304953 0.9519754 0.9435493 0.9046773 0.7878423 0.8492244 0.9326793 1.0000000 0.9409360 0.9097714
## b08_quant 0.7951065 0.8532209 0.9105425 0.9363643 0.9456457 0.9166922 0.7888875 0.8469294 0.9164104 0.9409360 1.0000000 0.9230216
## b24_quant 0.7961772 0.8465695 0.8833257 0.9106234 0.9277200 0.9339101 0.8022131 0.8439018 0.8898089 0.9097714 0.9230216 1.0000000</code></pre>
<pre><code>##  [1] &quot;old_locus_tag&quot;    &quot;hvo.x&quot;            &quot;ncbi_id&quot;          &quot;ap0_quant&quot;        &quot;a00_quant&quot;        &quot;a02_quant&quot;        &quot;a04_quant&quot;       
##  [8] &quot;a08_quant&quot;        &quot;a24_quant&quot;        &quot;bp0_quant&quot;        &quot;b00_quant&quot;        &quot;b02_quant&quot;        &quot;b04_quant&quot;        &quot;b08_quant&quot;       
## [15] &quot;b24_quant&quot;        &quot;protein&quot;          &quot;kegg1&quot;            &quot;kegg2&quot;            &quot;kegg3&quot;            &quot;gene_id&quot;          &quot;gene_description&quot;
## [22] &quot;kegg_id&quot;          &quot;kegg_gene&quot;        &quot;kegg_description&quot; &quot;enzyme&quot;           &quot;hvo.y&quot;</code></pre>
<pre><code>## [1] &quot;ap0_quant&quot; &quot;bp0_quant&quot;</code></pre>
<pre><code>## Saving 7 x 5 in image</code></pre>
<p><img src="hvo_mating_files/figure-html/correlations-1.png" width="672" /></p>
<pre><code>## [1] &quot;a00_quant&quot; &quot;b00_quant&quot;</code></pre>
<pre><code>## Saving 7 x 5 in image</code></pre>
<p><img src="hvo_mating_files/figure-html/correlations-2.png" width="672" /></p>
<pre><code>## [1] &quot;a02_quant&quot; &quot;b02_quant&quot;</code></pre>
<pre><code>## Saving 7 x 5 in image</code></pre>
<p><img src="hvo_mating_files/figure-html/correlations-3.png" width="672" /></p>
<pre><code>## [1] &quot;a04_quant&quot; &quot;b04_quant&quot;</code></pre>
<pre><code>## Saving 7 x 5 in image</code></pre>
<p><img src="hvo_mating_files/figure-html/correlations-4.png" width="672" /></p>
<pre><code>## [1] &quot;a08_quant&quot; &quot;b08_quant&quot;</code></pre>
<pre><code>## Saving 7 x 5 in image</code></pre>
<p><img src="hvo_mating_files/figure-html/correlations-5.png" width="672" /></p>
<pre><code>## [1] &quot;a24_quant&quot; &quot;b24_quant&quot;</code></pre>
<pre><code>## Saving 7 x 5 in image</code></pre>
<p><img src="hvo_mating_files/figure-html/correlations-6.png" width="672" /></p>
</div>
<div id="differential-expression-bar-plots-show-higher-expression-in-each-timepoint-compared-to-shaking-as-positive-value-and-lower-expression-compared-to-shaking-as-negative-value" class="section level3">
<h3>Differential expression bar plots show higher expression in each timepoint compared to shaking as positive value and lower expression compared to shaking as negative value</h3>
<pre class="r"><code>### This set of lists contains all original genes - hypotheticals/unused were removed in the list in the below chunk
lists &lt;- list(
  `Glycosylation`=c(&quot;HVO_0727&quot;, &quot;HVO_1524&quot;, &quot;HVO_1527&quot;, &quot;HVO_1528&quot;, &quot;HVO_1529&quot;, &quot;HVO_1530&quot;, &quot;HVO_1531&quot;, &quot;HVO_A0586&quot;, 
                    &quot;HVO_1517&quot;, &quot;HVO_1523A&quot;, &quot;HVO_0798&quot;, &quot;HVO_1526&quot;, &quot;HVO_1522&quot;, &quot;HVO_1523&quot;, &quot;HVO_2072&quot;),
  `Predicted Amyloid`=c(&quot;HVO_0066&quot;, &quot;HVO_A0362&quot;, &quot;HVO_A0410&quot;, &quot;HVO_B0132&quot;, &quot;HVO_B0138&quot;, &quot;HVO_B0139&quot;, &quot;HVO_B0140&quot;, 
                        &quot;HVO_B0141&quot;, &quot;HVO_B0222&quot;),
  `Tubulin`=c(&quot;HVO_0581&quot;, &quot;HVO_0717&quot;, &quot;HVO_0745&quot;, &quot;HVO_1113&quot;, &quot;HVO_2013&quot;, &quot;HVO_2068&quot;, &quot;HVO_2204&quot;, &quot;HVO_A0035&quot;),
  `Chemotaxis`=c(&quot;HVO_0553&quot;, &quot;HVO_0554&quot;, &quot;HVO_1205&quot;, &quot;HVO_1206&quot;, &quot;HVO_1207&quot;, &quot;HVO_1219&quot;, &quot;HVO_1221&quot;, &quot;HVO_1222&quot;, 
                 &quot;HVO_1223&quot;, &quot;HVO_1224&quot;, &quot;HVO_1225&quot;, &quot;HVO_2993&quot;),
  `SNARE dedA Motif`=c(&quot;HVO_RS05125&quot;, &quot;HVO_0934&quot;, &quot;HVO_1969&quot;, &quot;HVO_2517&quot;, &quot;HVO_B0204&quot;),
  `Quorum Sensing`=c(&quot;HVO_0204&quot;, &quot;HVO_0205&quot;, &quot;HVO_1811&quot;, &quot;HVO_1812&quot;, &quot;HVO_1917&quot;, &quot;HVO_2337&quot;, &quot;HVO_2338&quot;, &quot;HVO_2339&quot;),
  `Low-Salt Glycosylation`=c(&quot;HVO_2043&quot;, &quot;HVO_2044&quot;, &quot;HVO_2045&quot;, &quot;HVO_2046&quot;, &quot;HVO_2047&quot;, &quot;HVO_2048&quot;, &quot;HVO_2049&quot;, 
                             &quot;HVO_2051&quot;, &quot;HVO_2052&quot;, &quot;HVO_2053&quot;, &quot;HVO_2054&quot;, &quot;HVO_2055&quot;, &quot;HVO_2056&quot;, &quot;HVO_2057&quot;, 
                             &quot;HVO_2058&quot;, &quot;HVO_2059&quot;, &quot;HVO_2060&quot;, &quot;HVO_2061&quot;, &quot;HVO_2062&quot;, &quot;HVO_2063&quot;, &quot;HVO_2064&quot;),
  `Molecular Parasites`=c(&quot;HVO_0858&quot;, &quot;HVO_B0152&quot;, &quot;HVO_C0080&quot;, &quot;HVO_B0155&quot;, &quot;HVO_A0209&quot;),
  `MATE Family Transporters`=c(&quot;HVO_1524&quot;, &quot;HVO_2055&quot;, &quot;HVO_2342&quot;, &quot;HVO_2652&quot;, &quot;HVO_A0162&quot;, &quot;HVO_A0197&quot;),
  `Annotated Adhesins`=c(&quot;HVO_2397&quot;, &quot;HVO_A0611&quot;),
  `Bacterio-opsin Activator`=c(&quot;HVO_0318&quot;, &quot;HVO_0492&quot;, &quot;HVO_0513&quot;, &quot;HVO_1117&quot;, &quot;HVO_1190&quot;, &quot;HVO_1356&quot;, &quot;HVO_1357&quot;, 
                               &quot;HVO_1952&quot;, &quot;HVO_1953&quot;, &quot;HVO_2835&quot;, &quot;HVO_2836&quot;, &quot;HVO_2894&quot;, &quot;HVO_A0168&quot;, &quot;HVO_A0169&quot;, 
                               &quot;HVO_A0424&quot;, &quot;HVO_A0495&quot;, &quot;HVO_A0506&quot;, &quot;HVO_B0020&quot;, &quot;HVO_B0159&quot;, &quot;HVO_B0272&quot;),
  `Cell Division`=c(&quot;HVO_1725&quot;, &quot;HVO_2042&quot;, &quot;HVO_2293&quot;, &quot;HVO_B0001&quot;, &quot;HVO_A0001&quot;, &quot;HVO_A0064&quot;, &quot;HVO_A0257&quot;, &quot;HVO_0220&quot;),
  `Secretion System`=c(&quot;HVO_2166&quot;, &quot;HVO_2167&quot;, &quot;HVO_2168&quot;, &quot;HVO_2169&quot;, &quot;HVO_2170&quot;, &quot;HVO_2171&quot;, &quot;HVO_2172&quot;, &quot;HVO_2173&quot;, 
                     &quot;HVO_1477&quot;),
  #`DNA Uptake`=c(&quot;HVO_1477&quot;),
  `Type IV Secretion`=c(&quot;HVO_0421&quot;, &quot;HVO_0620&quot;, &quot;HVO_0748&quot;, &quot;HVO_1160&quot;, &quot;HVO_1217&quot;, &quot;HVO_2278&quot;, &quot;HVO_2385&quot;, &quot;HVO_C0036&quot;),
  `IS4 Family Transposase (set 1/3)`=c(&quot;HVO_B0152&quot;, &quot;HVO_B0157&quot;, &quot;HVO_B0234&quot;, &quot;HVO_A0077&quot;, &quot;HVO_A0081&quot;, &quot;HVO_A0110&quot;, &quot;HVO_A0125&quot;, 
                           &quot;HVO_A0144&quot;, &quot;HVO_A0151&quot;, &quot;HVO_A0250&quot;, &quot;HVO_A0260&quot;, &quot;HVO_A0262&quot;, &quot;HVO_A0367&quot;, &quot;HVO_A0420&quot;, 
                           &quot;HVO_A0423&quot;, &quot;HVO_A0431&quot;, &quot;HVO_A0433&quot;, &quot;HVO_A0449&quot;, &quot;HVO_0275&quot;, &quot;HVO_0276&quot;, &quot;HVO_0528&quot;), 
  `IS4 Family Transposase (set 2/3)`=c(&quot;HVO_0922&quot;, &quot;HVO_0926&quot;, &quot;HVO_1519&quot;, &quot;HVO_RS12025&quot;, &quot;HVO_1525&quot;, &quot;HVO_1782&quot;, &quot;HVO_1835&quot;, 
                           &quot;HVO_1845&quot;, &quot;HVO_2051&quot;, &quot;HVO_2054&quot;, &quot;HVO_2252&quot;, &quot;HVO_2331&quot;, &quot;HVO_2466&quot;, &quot;HVO_2817&quot;, 
                           &quot;HVO_2820&quot;, &quot;HVO_C0004&quot;, &quot;HVO_C0021&quot;, &quot;HVO_C0039&quot;, &quot;HVO_C0053&quot;, &quot;HVO_C0070&quot;, &quot;HVO_C0071&quot;), 
  `IS4 Family Transposase (set 3/3)`=c(&quot;HVO_A0007&quot;, &quot;HVO_A0238&quot;, &quot;HVO_A0355&quot;, &quot;HVO_0924&quot;, &quot;HVO_A0080&quot;, &quot;HVO_A0456&quot;, &quot;HVO_1106&quot;, 
                           &quot;HVO_2830&quot;, &quot;HVO_C0013&quot;, &quot;HVO_A0116&quot;),
  `IS5 Family Transposase`=c(&quot;HVO_A0117&quot;, &quot;HVO_A0057&quot;, &quot;HVO_A0240&quot;, &quot;HVO_C0068&quot;, &quot;HVO_0864&quot;, &quot;HVO_B0249&quot;, &quot;HVO_A0134&quot;),
  `IS6 Family Transposase`=c(&quot;HVO_A0265&quot;, &quot;HVO_C0065&quot;, &quot;HVO_C0020A&quot;),
  `ISH3 Family Transposase`=c(&quot;HVO_A0354&quot;, &quot;HVO_A0408&quot;, &quot;HVO_A0483&quot;, &quot;HVO_2050&quot;, &quot;HVO_A0353&quot;, &quot;HVO_1837&quot;, &quot;HVO_0263&quot;, 
                            &quot;HVO_A0364&quot;, &quot;HVO_A0445&quot;, &quot;HVO_C0037&quot;, &quot;HVO_A0349&quot;),
  `IS630 Family Transposase`=c(&quot;HVO_A0100&quot;, &quot;HVO_A0403&quot;, &quot;HVO_C0078&quot;, &quot;HVO_C0080&quot;),
  `IS110 Family Transposase`=c(&quot;HVO_0556&quot;, &quot;HVO_A0016&quot;, &quot;HVO_A0128&quot;, &quot;HVO_A0014&quot;, &quot;HVO_A0279&quot;, &quot;HVO_0278&quot;, &quot;HVO_2821&quot;),
  `IS1595 Family Transposase`=c(&quot;HVO_RS19485&quot;, &quot;HVO_C0064&quot;, &quot;HVO_A0062&quot;, &quot;HVO_A0258&quot;, &quot;HVO_RS19515&quot;),
  `IS200/IS605 Family Transposase`=c(&quot;HVO_1150&quot;, &quot;HVO_B0242&quot;),
  `Other Transposase`=c(&quot;HVO_B0155&quot;, &quot;HVO_B0152A&quot;, &quot;HVO_B0241&quot;, &quot;HVO_A0008&quot;, &quot;HVO_A0018&quot;, &quot;HVO_A0066&quot;, &quot;HVO_A0309&quot;, 
                       &quot;HVO_1151&quot;, &quot;HVO_2075&quot;, &quot;HVO_2434&quot;, &quot;HVO_RS19915&quot;, &quot;HVO_2090A&quot;),
  `misc`=c(&quot;HVO_2380&quot;, &quot;HVO_2700&quot;, &quot;HVO_1327&quot;, &quot;HVO_0915&quot;),
  `new glyco`=c(&quot;HVO_2166&quot;, &quot;HVO_2167&quot;, &quot;HVO_2168&quot;, &quot;HVO_2169&quot;, &quot;HVO_2170&quot;, &quot;HVO_2171&quot;, &quot;HVO_2172&quot;),
  `virus proteins`=c(&quot;HVO_0268&quot;, &quot;HVO_0269&quot;, &quot;HVO_0270&quot;, &quot;HVO_0271&quot;, &quot;HVO_0272&quot;, &quot;HVO_0273&quot;, &quot;HVO_0274&quot;, &quot;HVO_0275&quot;, &quot;HVO_0276&quot;,
                     &quot;HVO_0279&quot;, &quot;HVO_0258&quot;, &quot;HVO_0260&quot;, &quot;HVO_0261&quot;, &quot;HVO_2278&quot;, &quot;HVO_2824&quot;),
  `RM System`=c(&quot;HVO_0794&quot;, &quot;HVO_2269&quot;, &quot;HVO_2270&quot;, &quot;HVO_2271&quot;, &quot;HVO_C0040&quot;, &quot;HVO_A0006&quot;, &quot;HVO_A0079&quot;, &quot;HVO_A0237&quot;)
)</code></pre>
<pre><code>## [1] &quot;Glycosylation&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/bar%20plots-1.png" width="720" /></p>
<pre><code>## [1] &quot;Predicted trans-membrane&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/bar%20plots-2.png" width="720" /></p>
<pre><code>## [1] &quot;DNA binding&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/bar%20plots-3.png" width="720" /></p>
<pre><code>## [1] &quot;Low-Salt Glycosylation&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/bar%20plots-4.png" width="720" /></p>
<pre><code>## [1] &quot;Quorum Sensing&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/bar%20plots-5.png" width="720" /></p>
<pre><code>## [1] &quot;Secretion System&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/bar%20plots-6.png" width="720" /></p>
<pre><code>## [1] &quot;Chemotaxis&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/bar%20plots-7.png" width="720" /></p>
<pre><code>## [1] &quot;flagella&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/bar%20plots-8.png" width="720" /></p>
<pre><code>## [1] &quot;EPS&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/bar%20plots-9.png" width="720" /></p>
<pre><code>## [1] &quot;dessication&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/bar%20plots-10.png" width="720" /></p>
<pre><code>## [1] &quot;stress&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/bar%20plots-11.png" width="720" /></p>
<pre><code>## [1] &quot;division1&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/bar%20plots-12.png" width="720" /></p>
<pre><code>## [1] &quot;division2&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/bar%20plots-13.png" width="720" /></p>
<pre><code>## [1] &quot;Selfish genes&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/bar%20plots-14.png" width="720" /></p>
<pre><code>## [1] &quot;RM System&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/bar%20plots-15.png" width="720" /></p>
<pre><code>## [1] &quot;Tubulin&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/bar%20plots-16.png" width="720" /></p>
<pre><code>## [1] &quot;Cell Division&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/bar%20plots-17.png" width="720" /></p>
<pre><code>## [1] &quot;SNARE dedA Motif&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/bar%20plots-18.png" width="720" /></p>
<pre><code>## [1] &quot;excreted stuff&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/bar%20plots-19.png" width="720" /></p>
<pre><code>## [1] &quot;aglD&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/bar%20plots-20.png" width="720" /></p>
<pre><code>## [1] &quot;crispr&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/bar%20plots-21.png" width="720" /></p>
<pre><code>## [1] &quot;all QS&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/bar%20plots-22.png" width="720" /></p>
<pre><code>## [1] &quot;andreas&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/bar%20plots-23.png" width="720" /></p>
<pre><code>## [1] &quot;Predicted Amyloid&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/bar%20plots-24.png" width="720" /></p>
<pre><code>## [1] &quot;Secretion System&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/bar%20plots-25.png" width="720" /></p>
<pre><code>## [1] &quot;Type IV Secretion&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/bar%20plots-26.png" width="720" /></p>
<pre><code>## [1] &quot;IS4 Family Transposase 1&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/bar%20plots-27.png" width="720" /></p>
<pre><code>## [1] &quot;IS4 Family Transposase 2&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/bar%20plots-28.png" width="720" /></p>
<pre><code>## [1] &quot;IS4 Family Transposase 3&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/bar%20plots-29.png" width="720" /></p>
<pre><code>## [1] &quot;IS5 Family Transposase&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/bar%20plots-30.png" width="720" /></p>
<pre><code>## [1] &quot;IS6 Family Transposase&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/bar%20plots-31.png" width="720" /></p>
<pre><code>## [1] &quot;ISH3 Family Transposase&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/bar%20plots-32.png" width="720" /></p>
<pre><code>## [1] &quot;IS630 Family Transposase&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/bar%20plots-33.png" width="720" /></p>
<pre><code>## [1] &quot;IS110 Family Transposase&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/bar%20plots-34.png" width="720" /></p>
<pre><code>## [1] &quot;IS1595 Family Transposase&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/bar%20plots-35.png" width="720" /></p>
<pre><code>## [1] &quot;IS200-IS605 Family Transposase&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/bar%20plots-36.png" width="720" /></p>
<pre><code>## [1] &quot;Other Transposase&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/bar%20plots-37.png" width="720" /></p>
<pre class="r"><code>load(&quot;data/p0.RData&quot;)
load(&quot;data/coords.RData&quot;)
load(&quot;data/coords01.RData&quot;)

coords$valt &lt;- NULL
coords01$val01t &lt;- NULL
coords$prot &lt;- NULL
coords01$prot &lt;- NULL

lists &lt;- list(
  `snare_deda`=c(&quot;HVO_RS05125&quot;, &quot;HVO_0934&quot;, &quot;HVO_1969&quot;, &quot;HVO_2517&quot;, &quot;HVO_B0204&quot;),
  `cetz_ftsz`=c(&quot;HVO_0581&quot;, &quot;HVO_0717&quot;, &quot;HVO_0745&quot;, &quot;HVO_1113&quot;, &quot;HVO_2013&quot;, &quot;HVO_2068&quot;, &quot;HVO_2204&quot;, &quot;HVO_A0035&quot;),
  `cell_div`=c(&quot;HVO_1725&quot;, &quot;HVO_2042&quot;, &quot;HVO_2293&quot;, &quot;HVO_B0001&quot;, &quot;HVO_A0001&quot;, &quot;HVO_A0064&quot;, &quot;HVO_A0257&quot;, &quot;HVO_0220&quot;),
  `rms`=c(&quot;HVO_0794&quot;, &quot;HVO_2269&quot;, &quot;HVO_2270&quot;, &quot;HVO_2271&quot;, &quot;HVO_C0040&quot;, &quot;HVO_A0006&quot;, &quot;HVO_A0079&quot;, &quot;HVO_A0237&quot;),
  `tivss`=c(&quot;HVO_0421&quot;, &quot;HVO_0620&quot;, &quot;HVO_0748&quot;, &quot;HVO_1160&quot;, &quot;HVO_1217&quot;, &quot;HVO_2278&quot;, &quot;HVO_2385&quot;, &quot;HVO_C0036&quot;,
  &quot;HVO_2281&quot;, &quot;HVO_C0036&quot;),
  `crispr_cas`=c(&quot;HVO_A0205&quot;, &quot;HVO_A0206&quot;, &quot;HVO_A0207&quot;, &quot;HVO_A0208&quot;,&quot;HVO_A0209&quot;, &quot;HVO_A0210&quot;, &quot;HVO_A0211&quot;, &quot;HVO_A0212&quot;),
  `glyc`=c(&quot;HVO_0727&quot;, &quot;HVO_1524&quot;, &quot;HVO_1527&quot;, &quot;HVO_1528&quot;, &quot;HVO_1529&quot;, &quot;HVO_1530&quot;, &quot;HVO_1531&quot;, &quot;HVO_A0586&quot;,
                    &quot;HVO_1517&quot;, &quot;HVO_1523A&quot;, &quot;HVO_0798&quot;, &quot;HVO_1526&quot;, &quot;HVO_1522&quot;, &quot;HVO_1523&quot;, &quot;HVO_2072&quot;, &quot;HVO_1734A&quot;),
  `lsglyc`=c(&quot;HVO_2043&quot;, &quot;HVO_2044&quot;, &quot;HVO_2045&quot;, &quot;HVO_2046&quot;, &quot;HVO_2047&quot;, &quot;HVO_2048&quot;, &quot;HVO_2049&quot;,
  &quot;HVO_2051&quot;, &quot;HVO_2052&quot;, &quot;HVO_2053&quot;, &quot;HVO_2054&quot;, &quot;HVO_2055&quot;, &quot;HVO_2056&quot;, &quot;HVO_2057&quot;,
  &quot;HVO_2058&quot;, &quot;HVO_2059&quot;, &quot;HVO_2060&quot;, &quot;HVO_2061&quot;, &quot;HVO_2062&quot;, &quot;HVO_2063&quot;, &quot;HVO_2064&quot;)
)

## relabel specific items in p0 based on corrections discussed in lab meetings
p0$prot &lt;- as.character(p0$prot)
p0[2604, 9] &lt;- &quot;cell division protein FtsZ2&quot;
p0[2514, 9] &lt;- &quot;cell division protein FtsZ1&quot;
p0[2491, 9] &lt;- &quot;cell division protein CetZ2&quot;
p0[2193, 9] &lt;- &quot;cell division protein CetZ3&quot;
p0[1248, 9] &lt;- &quot;cell division protein CetZ5&quot;
p0[1294, 9] &lt;- &quot;cell division protein CetZ6&quot;
p0[3425, 9] &lt;- &quot;cell division protein CetZ1&quot;
p0[635, 9] &lt;- &quot;cell division protein CetZ4&quot;
p0[3406, 9] &lt;- &quot;type I RMS restriction subunit&quot;
p0[3243, 9] &lt;- &quot;type I RMS methylation subunit&quot;
p0[3342, 9] &lt;- &quot;type I RMS specificity subunit&quot;
p0[3542, 9] &lt;- &quot;DNA cytosine methyltransferase&quot;
p0[454, 9] &lt;- &quot;methyltransferase (hypothetical)&quot;
p0[607, 9] &lt;- &quot;N-6 DNA methylase (hypothetical)&quot;
p0[779, 9] &lt;- &quot;N-6 DNA methylase (hypothetical)&quot;
p0[2113, 9] &lt;- &quot;type II/IV secretion system protein (hypothetical)&quot;
p0[3556, 9] &lt;- &quot;type IV secretion system coupling protein DNA-binding domain (hypothetical)&quot;
p0[1445, 9] &lt;- &quot;type IV secretion system TraD (conjugal transfer protein)&quot;
p0[2729, 9] &lt;- &quot;type II/IV secretion system protein&quot;
p0[1448, 9] &lt;- &quot;type IV secretory pathway, VirB4 component&quot;
p0[2161, 9] &lt;- &quot;type II/IV secretion system protein&quot;
p0[497, 9] &lt;- &quot;helicase Cas3&quot;
p0$prot &lt;- as.factor(p0$prot)

for(i in 1:length(lists)){
  nm &lt;- (names(lists)[i])
  print(nm)
  df_sub &lt;- filter(p0, grepl(paste((lists[[i]]), collapse=&quot;|&quot;), old))
  coords_sub &lt;- filter(coords, grepl(paste((lists[[i]]), collapse=&quot;|&quot;), old))
  coords01_sub &lt;- filter(coords01, grepl(paste((lists[[i]]), collapse=&quot;|&quot;), old))
  df_sub$Row.names &lt;- NULL
  df_sub &lt;- melt(df_sub, id.vars=c(&quot;old&quot;, &quot;new&quot;, &quot;prot&quot;, &quot;geneid&quot;, &quot;arcog&quot;, &quot;cog&quot;, &quot;gene_name&quot;))
  df_sub1 &lt;- merge(df_sub, coords_sub, all.x = TRUE)
  df_sub1 &lt;- merge(df_sub1, coords01_sub, all.x = TRUE)
  p &lt;- ggplot(data=df_sub1, aes(x= paste(new, 
                                         # prot, sep = &#39;\n&#39;), # single line prot
                                         str_wrap(prot, width = 40), sep = &#39;\n&#39;), # multiline x-axis
                                y=value, fill = variable)) +
    theme_bw() +
    geom_bar(stat=&quot;identity&quot;, position = position_dodge(width = NULL), width = 0.7) +
    geom_hline(yintercept = 0) +
    theme(panel.grid.major = element_blank(), panel.grid.minor = element_blank()) +
    labs(title = paste(nm,&quot;genes&quot;), 
         x = NULL, y = expression(&quot;log&quot;[&quot;2&quot;]*&quot; fold change&quot;), fill = &quot;time vs shaking&quot;) +
    theme(text = element_text(size=12), axis.text.x = element_text(size  = 12, angle = 45, hjust = 1, vjust = 1)) +
    theme(plot.title = element_blank()) +
    theme(plot.margin = unit(c(0.5,0.5,0.5,2),&quot;cm&quot;)) + ## use this for most plot margins
    # theme(plot.margin = unit(c(0.5,1,0,1.5),&quot;cm&quot;)) + ## use this for cowplot for rm
    scale_fill_manual(labels = c(&quot;t00&quot;, &quot;t02&quot;, &quot;t04&quot;, &quot;t08&quot;, &quot;t24&quot;), 
                      values = c(&quot;#000000&quot;, &quot;#E69F00&quot;, &quot;#009E73&quot;, &quot;#0072B2&quot;, &quot;#D55E00&quot;)) +
    theme(legend.background = element_rect(fill = &quot;transparent&quot;)) +
    theme(legend.position = c(0.9, 0.85)) +
    theme(legend.title = element_text(size = 10), legend.text = element_text(size = 10)) +
    theme(legend.key.size = unit(0.07,&quot;in&quot;)) +
    geom_text(aes(y = val), label = &quot;*&quot;, position = position_dodge(width = 0.7)) +
    geom_text(aes(y = val01), label = &quot;**&quot;, position = position_dodge(width = 0.7))
  print(p)
  nm2 &lt;- gsub(&quot; &quot;, &quot;_&quot;, nm)
  assign(paste(&quot;plot&quot;, nm2, sep = &quot;_&quot;), p)
  ggsave(filename = paste(&quot;plots/bar_&quot;,nm2,&quot;_histogram.pdf&quot;, sep = &quot;&quot;), 
         # height = 6, width = 8, units = &quot;in&quot; ## for snare
         height = 6, width = 7, units = &quot;in&quot; ## for tivss
         # height = 8, width = 10, units = &quot;in&quot; ## for ftsz/cetz system
         )
  # ggsave(filename = paste(&quot;exp_mating/gene_plots/&quot;,nm2,&quot;_histogram.png&quot;, sep = &quot;&quot;))
}</code></pre>
<pre><code>## [1] &quot;snare_deda&quot;</code></pre>
<pre><code>## Warning: Removed 12 rows containing missing values (geom_text).</code></pre>
<pre><code>## Warning: Removed 11 rows containing missing values (geom_text).</code></pre>
<pre><code>## Warning: Removed 12 rows containing missing values (geom_text).</code></pre>
<pre><code>## Warning: Removed 11 rows containing missing values (geom_text).</code></pre>
<p><img src="hvo_mating_files/figure-html/manuscript%20barplots-1.png" width="672" /></p>
<pre><code>## [1] &quot;cetz_ftsz&quot;</code></pre>
<pre><code>## Warning: Removed 27 rows containing missing values (geom_text).</code></pre>
<pre><code>## Warning: Removed 32 rows containing missing values (geom_text).</code></pre>
<pre><code>## Warning: Removed 27 rows containing missing values (geom_text).</code></pre>
<pre><code>## Warning: Removed 32 rows containing missing values (geom_text).</code></pre>
<p><img src="hvo_mating_files/figure-html/manuscript%20barplots-2.png" width="672" /></p>
<pre><code>## [1] &quot;cell_div&quot;</code></pre>
<pre><code>## Warning: Removed 33 rows containing missing values (geom_text).

## Warning: Removed 32 rows containing missing values (geom_text).</code></pre>
<pre><code>## Warning: Removed 33 rows containing missing values (geom_text).</code></pre>
<pre><code>## Warning: Removed 32 rows containing missing values (geom_text).</code></pre>
<p><img src="hvo_mating_files/figure-html/manuscript%20barplots-3.png" width="672" /></p>
<pre><code>## [1] &quot;rms&quot;</code></pre>
<pre><code>## Warning: Removed 3 rows containing missing values (geom_bar).</code></pre>
<pre><code>## Warning: Removed 30 rows containing missing values (geom_text).</code></pre>
<pre><code>## Warning: Removed 27 rows containing missing values (geom_text).</code></pre>
<pre><code>## Warning: Removed 3 rows containing missing values (geom_bar).</code></pre>
<pre><code>## Warning: Removed 30 rows containing missing values (geom_text).</code></pre>
<pre><code>## Warning: Removed 27 rows containing missing values (geom_text).</code></pre>
<p><img src="hvo_mating_files/figure-html/manuscript%20barplots-4.png" width="672" /></p>
<pre><code>## [1] &quot;tivss&quot;</code></pre>
<pre><code>## Warning: Removed 9 rows containing missing values (geom_bar).</code></pre>
<pre><code>## Warning: Removed 19 rows containing missing values (geom_text).</code></pre>
<pre><code>## Warning: Removed 26 rows containing missing values (geom_text).</code></pre>
<pre><code>## Warning: Removed 9 rows containing missing values (geom_bar).</code></pre>
<pre><code>## Warning: Removed 19 rows containing missing values (geom_text).</code></pre>
<pre><code>## Warning: Removed 26 rows containing missing values (geom_text).</code></pre>
<p><img src="hvo_mating_files/figure-html/manuscript%20barplots-5.png" width="672" /></p>
<pre><code>## [1] &quot;crispr_cas&quot;</code></pre>
<pre><code>## Warning: Removed 35 rows containing missing values (geom_text).</code></pre>
<pre><code>## Warning: Removed 33 rows containing missing values (geom_text).</code></pre>
<pre><code>## Warning: Removed 35 rows containing missing values (geom_text).</code></pre>
<pre><code>## Warning: Removed 33 rows containing missing values (geom_text).</code></pre>
<p><img src="hvo_mating_files/figure-html/manuscript%20barplots-6.png" width="672" /></p>
<pre><code>## [1] &quot;glyc&quot;</code></pre>
<pre><code>## Warning: Removed 1 rows containing missing values (geom_bar).</code></pre>
<pre><code>## Warning: Removed 67 rows containing missing values (geom_text).</code></pre>
<pre><code>## Warning: Removed 47 rows containing missing values (geom_text).</code></pre>
<pre><code>## Warning: Removed 1 rows containing missing values (geom_bar).</code></pre>
<pre><code>## Warning: Removed 67 rows containing missing values (geom_text).</code></pre>
<pre><code>## Warning: Removed 47 rows containing missing values (geom_text).</code></pre>
<p><img src="hvo_mating_files/figure-html/manuscript%20barplots-7.png" width="672" /></p>
<pre><code>## [1] &quot;lsglyc&quot;</code></pre>
<pre><code>## Warning: Removed 76 rows containing missing values (geom_text).</code></pre>
<pre><code>## Warning: Removed 72 rows containing missing values (geom_text).</code></pre>
<pre><code>## Warning: Removed 76 rows containing missing values (geom_text).</code></pre>
<pre><code>## Warning: Removed 72 rows containing missing values (geom_text).</code></pre>
<p><img src="hvo_mating_files/figure-html/manuscript%20barplots-8.png" width="672" /></p>
<pre class="r"><code>### for final plots, edit script as needed - slightly modified for each</code></pre>
<pre class="r"><code>### gene names manually adjusted in p0 table
p0 &lt;- read.delim(&quot;data/p0_newgene.txt&quot;)
coords &lt;- read.delim(&quot;data/coords.txt&quot;)
coords01 &lt;- read.delim(&quot;data/coords01.txt&quot;)

coords$valt &lt;- ifelse(coords$val&gt;0, coords$val, coords$val + 0.1)
coords01$val01t &lt;- ifelse(coords01$val01&gt;0, coords01$val01, coords01$val01 + 0.1)

p0$prot &lt;- as.character(p0$prot)
p0[840, 9] &lt;- &quot;ATPase FlaH&quot;
p0[834, 9] &lt;- &quot;archaellar protein FlaCE&quot;
p0[842, 9] &lt;- &quot;type II secretion system protein PilB3&quot;
p0[1544, 9] &lt;- &quot;secretion system protein PilC3&quot;
p0[2055, 9] &lt;- &quot;transducer protein Htr15b&quot;
p0[2054, 9] &lt;- &quot;transducer protein Htr15a&quot;
p0[2053, 9] &lt;- &quot;transducer protein Htr15a&quot;
p0[751, 9] &lt;- &quot;peptidase A24 PibD&quot;
p0[2059, 9] &lt;- &quot;methyl-accepting chemotaxis protein Htr7&quot;
p0[553, 9] &lt;- &quot;chemotaxis response regulator protein CheB&quot;
p0[554, 9] &lt;- &quot;chemotaxis response regulator protein CheB&quot;
p0[839, 9] &lt;- &quot;archaellin FlgA2&quot;
p0[2418, 9] &lt;- &quot;archaellin FlgA1&quot;
p0[555, 9] &lt;- &quot;response regulator CheY&quot;
p0[556, 9] &lt;- &quot;response regulator CheY&quot;
p0[2058, 9] &lt;- &quot;methyl−accepting chemotaxis protein Htr39&quot;
p0[2435, 9] &lt;- &quot;ATPase MoxR3&quot;
p0[843, 9] &lt;- &quot;secretion system protein PilB4&quot;
p0[3075, 9] &lt;- &quot;Flp pilus assembly complex ATPase component TadA&quot;
p0[2356, 9] &lt;- &quot;archaeosortase A ArtA&quot;

p0[2469, 9] &lt;- &quot;archaellin pilA2&quot;
p0[2470, 9] &lt;- &quot;archaellin pilA3&quot;
p0[2471, 9] &lt;- &quot;archaellin pilA4&quot;
p0[2472, 9] &lt;- &quot;archaellin pilA5&quot;
p0[2473, 9] &lt;- &quot;archaellin pilA6&quot;
p0[2814, 9] &lt;- &quot;archaellin type IV pilin&quot;
p0$prot &lt;- as.factor(p0$prot)

secretion_system &lt;- c(&quot;HVO_2168&quot;, &quot;HVO_0421&quot;, &quot;HVO_2278&quot;, &quot;HVO_0915&quot;)
chemotaxis &lt;- c(&quot;HVO_2993&quot;, &quot;HVO_0420&quot;, &quot;HVO_0555&quot;, &quot;HVO_0969&quot;,
               &quot;HVO_1207&quot;, &quot;HVO_1219&quot;, &quot;HVO_1222&quot;, &quot;HVO_1224&quot;,
               &quot;HVO_1225&quot;, &quot;HVO_1779&quot;, &quot;HVO_1999&quot;, &quot;HVO_2214&quot;, &quot;HVO_3005&quot;)
flagella &lt;- c(&quot;HVO_1033&quot;, &quot;HVO_1034&quot;, &quot;HVO_1160&quot;, &quot;HVO_1200&quot;, &quot;HVO_1203&quot;, 
              &quot;HVO_1210&quot;, &quot;HVO_1211&quot;, &quot;HVO_1213&quot;, &quot;HVO_1214&quot;, &quot;HVO_1215&quot;, 
              &quot;HVO_1216&quot;, &quot;HVO_2993&quot;, &quot;HVO_2062&quot;, &quot;HVO_2450&quot;, &quot;HVO_2451&quot;, 
              &quot;HVO_A0632&quot;, &quot;HVO_A0633&quot;, &quot;HVO_2704&quot;, &quot;HVO_1160&quot;)
# other_flagella &lt;- c(&quot;HVO_2062&quot;, &quot;HVO_2450&quot;, &quot;HVO_2451&quot;, &quot;HVO_A0632&quot;, &quot;HVO_A0633&quot;, &quot;HVO_2704&quot;) # added to flagella

df &lt;- stack(mget(c(&#39;secretion_system&#39;, &#39;chemotaxis&#39;, &#39;flagella&#39;)))
# df &lt;- stack(mget(c(&#39;secretion_system&#39;, &#39;chemotaxis&#39;, &#39;flagella&#39;, &#39;eps&#39;, &#39;other_flagella&#39;)))
df &lt;- plyr::rename(df, c(&quot;values&quot;=&quot;old&quot;, &quot;ind&quot;=&quot;category&quot;))
df_sub &lt;- dplyr::distinct(merge(df, p0, all.x = TRUE, by = &quot;old&quot;))
df_sub$Row.names &lt;- NULL
df_sub &lt;- reshape::melt(df_sub, id.vars=c(&quot;old&quot;, &quot;category&quot;, &quot;new&quot;, &quot;prot&quot;, &quot;gene_name&quot;, &quot;arcog&quot;))
coords_sub &lt;- dplyr::distinct(merge(df, coords, all.x = TRUE, by = &quot;old&quot;))
coords01_sub &lt;- dplyr::distinct(merge(df, coords01, all.x = TRUE, by = &quot;old&quot;))
df_sub1 &lt;- merge(df_sub, coords_sub, all.x = TRUE)
df_sub1 &lt;- merge(df_sub1, coords01_sub, all.x = TRUE)


ggplot(data=arrange(df_sub1,category), aes(x = old, y=value, fill = variable, group = variable)) +
# ggplot(data=arrange(df_sub1,category), aes(x = paste(new, prot, sep=&#39;\n&#39;), y=value, fill = variable)) +
    theme_bw() +
    geom_bar(stat=&quot;identity&quot;, position = position_dodge2(width = NULL, reverse=TRUE, padding = 0), width = 0.7) +
    # geom_signif(stat=&quot;identity&quot;, aes(y = val, x = new, annotation = &quot;*&quot;, xend = new, yend = val)) +
    geom_hline(yintercept = 0) +
    geom_vline(xintercept = 4.5, col=&#39;grey47&#39;, lwd=1) +
    geom_vline(xintercept = 17.5, col=&#39;grey47&#39;, lwd=1) +
    theme(panel.grid.major = element_blank(), panel.grid.minor = element_blank()) +
    theme(axis.text.x = element_text(size = 8, hjust = 1, vjust = 1)) +
    theme(plot.title = element_text(hjust = 0.5), plot.margin = unit(c(0.5,0,0.5,1),&quot;cm&quot;)) +
    scale_fill_manual(labels = c(&quot;t00&quot;, &quot;t02&quot;, &quot;t04&quot;, &quot;t08&quot;, &quot;t24&quot;), 
                      values=c(&quot;#000000&quot;, &quot;#E69F00&quot;, &quot;#009E73&quot;, &quot;#0072B2&quot;, &quot;#D55E00&quot;)) +
    # geom_text(aes(y = valt), label = &quot;*&quot;, position = position_dodge(width = 0.7)) +
    # geom_text(aes(y = val01t), label = &quot;**&quot;, position = position_dodge(width = 0.7)) +
    aes(x = fct_inorder(paste(new, prot, sep=&#39;\n&#39;), category)) +
    labs(x = NULL, y = expression(&quot;log&quot;[&quot;2&quot;]*&quot; fold change&quot;), fill = &quot;time vs shaking&quot;) +
    theme(legend.position = c(0.94, 0.87)) +
    coord_flip()</code></pre>
<pre><code>## Warning in if (is.na(ordered)) {: the condition has length &gt; 1 and only the first element will be used</code></pre>
<pre><code>## Warning in if (ordered) &quot;ordered&quot;: the condition has length &gt; 1 and only the first element will be used</code></pre>
<pre><code>## Warning in if (is.na(ordered)) {: the condition has length &gt; 1 and only the first element will be used</code></pre>
<pre><code>## Warning in if (ordered) &quot;ordered&quot;: the condition has length &gt; 1 and only the first element will be used</code></pre>
<pre><code>## Warning: Removed 8 rows containing missing values (geom_bar).</code></pre>
<p><img src="hvo_mating_files/figure-html/long%20plot%20biofilm-1.png" width="624" /></p>
<pre class="r"><code># longplot
ggsave(&quot;plots/fig3_community.pdf&quot;, height = 12, width = 10)</code></pre>
<pre><code>## Warning in if (is.na(ordered)) {: the condition has length &gt; 1 and only the first element will be used</code></pre>
<pre><code>## Warning in if (ordered) &quot;ordered&quot;: the condition has length &gt; 1 and only the first element will be used</code></pre>
<pre><code>## Warning in if (is.na(ordered)) {: the condition has length &gt; 1 and only the first element will be used</code></pre>
<pre><code>## Warning in if (ordered) &quot;ordered&quot;: the condition has length &gt; 1 and only the first element will be used</code></pre>
<pre><code>## Warning: Removed 8 rows containing missing values (geom_bar).</code></pre>
<pre><code>## Warning in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y, : conversion failure on &#39;methyl−accepting chemotaxis protein Htr39&#39; in
## &#39;mbcsToSbcs&#39;: dot substituted for &lt;e2&gt;</code></pre>
<pre><code>## Warning in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y, : conversion failure on &#39;methyl−accepting chemotaxis protein Htr39&#39; in
## &#39;mbcsToSbcs&#39;: dot substituted for &lt;88&gt;</code></pre>
<pre><code>## Warning in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y, : conversion failure on &#39;methyl−accepting chemotaxis protein Htr39&#39; in
## &#39;mbcsToSbcs&#39;: dot substituted for &lt;92&gt;</code></pre>
<pre><code>## Warning in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y, : conversion failure on &#39;methyl−accepting chemotaxis protein Htr39&#39; in
## &#39;mbcsToSbcs&#39;: dot substituted for &lt;e2&gt;</code></pre>
<pre><code>## Warning in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y, : conversion failure on &#39;methyl−accepting chemotaxis protein Htr39&#39; in
## &#39;mbcsToSbcs&#39;: dot substituted for &lt;88&gt;</code></pre>
<pre><code>## Warning in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y, : conversion failure on &#39;methyl−accepting chemotaxis protein Htr39&#39; in
## &#39;mbcsToSbcs&#39;: dot substituted for &lt;92&gt;</code></pre>
<pre><code>## Warning in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y, : conversion failure on &#39;methyl−accepting chemotaxis protein Htr39&#39; in
## &#39;mbcsToSbcs&#39;: dot substituted for &lt;e2&gt;</code></pre>
<pre><code>## Warning in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y, : conversion failure on &#39;methyl−accepting chemotaxis protein Htr39&#39; in
## &#39;mbcsToSbcs&#39;: dot substituted for &lt;88&gt;</code></pre>
<pre><code>## Warning in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y, : conversion failure on &#39;methyl−accepting chemotaxis protein Htr39&#39; in
## &#39;mbcsToSbcs&#39;: dot substituted for &lt;92&gt;</code></pre>
<pre><code>## Warning in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y, : conversion failure on &#39;methyl−accepting chemotaxis protein Htr39&#39; in
## &#39;mbcsToSbcs&#39;: dot substituted for &lt;e2&gt;</code></pre>
<pre><code>## Warning in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y, : conversion failure on &#39;methyl−accepting chemotaxis protein Htr39&#39; in
## &#39;mbcsToSbcs&#39;: dot substituted for &lt;88&gt;</code></pre>
<pre><code>## Warning in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y, : conversion failure on &#39;methyl−accepting chemotaxis protein Htr39&#39; in
## &#39;mbcsToSbcs&#39;: dot substituted for &lt;92&gt;</code></pre>
<pre><code>## Warning in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y, : conversion failure on &#39;methyl−accepting chemotaxis protein Htr39&#39; in
## &#39;mbcsToSbcs&#39;: dot substituted for &lt;e2&gt;</code></pre>
<pre><code>## Warning in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y, : conversion failure on &#39;methyl−accepting chemotaxis protein Htr39&#39; in
## &#39;mbcsToSbcs&#39;: dot substituted for &lt;88&gt;</code></pre>
<pre><code>## Warning in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y, : conversion failure on &#39;methyl−accepting chemotaxis protein Htr39&#39; in
## &#39;mbcsToSbcs&#39;: dot substituted for &lt;92&gt;</code></pre>
<pre><code>## Warning in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y, : conversion failure on &#39;methyl−accepting chemotaxis protein Htr39&#39; in
## &#39;mbcsToSbcs&#39;: dot substituted for &lt;e2&gt;</code></pre>
<pre><code>## Warning in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y, : conversion failure on &#39;methyl−accepting chemotaxis protein Htr39&#39; in
## &#39;mbcsToSbcs&#39;: dot substituted for &lt;88&gt;</code></pre>
<pre><code>## Warning in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y, : conversion failure on &#39;methyl−accepting chemotaxis protein Htr39&#39; in
## &#39;mbcsToSbcs&#39;: dot substituted for &lt;92&gt;</code></pre>
<pre><code>## Warning in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y, : conversion failure on &#39;methyl−accepting chemotaxis protein Htr39&#39; in
## &#39;mbcsToSbcs&#39;: dot substituted for &lt;e2&gt;</code></pre>
<pre><code>## Warning in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y, : conversion failure on &#39;methyl−accepting chemotaxis protein Htr39&#39; in
## &#39;mbcsToSbcs&#39;: dot substituted for &lt;88&gt;</code></pre>
<pre><code>## Warning in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y, : conversion failure on &#39;methyl−accepting chemotaxis protein Htr39&#39; in
## &#39;mbcsToSbcs&#39;: dot substituted for &lt;92&gt;</code></pre>
<pre><code>## Warning in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y, : conversion failure on &#39;methyl−accepting chemotaxis protein Htr39&#39; in
## &#39;mbcsToSbcs&#39;: dot substituted for &lt;e2&gt;</code></pre>
<pre><code>## Warning in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y, : conversion failure on &#39;methyl−accepting chemotaxis protein Htr39&#39; in
## &#39;mbcsToSbcs&#39;: dot substituted for &lt;88&gt;</code></pre>
<pre><code>## Warning in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y, : conversion failure on &#39;methyl−accepting chemotaxis protein Htr39&#39; in
## &#39;mbcsToSbcs&#39;: dot substituted for &lt;92&gt;</code></pre>
<pre><code>## Warning in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y, : conversion failure on &#39;methyl−accepting chemotaxis protein Htr39&#39; in
## &#39;mbcsToSbcs&#39;: dot substituted for &lt;e2&gt;</code></pre>
<pre><code>## Warning in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y, : conversion failure on &#39;methyl−accepting chemotaxis protein Htr39&#39; in
## &#39;mbcsToSbcs&#39;: dot substituted for &lt;88&gt;</code></pre>
<pre><code>## Warning in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y, : conversion failure on &#39;methyl−accepting chemotaxis protein Htr39&#39; in
## &#39;mbcsToSbcs&#39;: dot substituted for &lt;92&gt;</code></pre>
<pre><code>## Warning in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y, : conversion failure on &#39;methyl−accepting chemotaxis protein Htr39&#39; in
## &#39;mbcsToSbcs&#39;: dot substituted for &lt;e2&gt;</code></pre>
<pre><code>## Warning in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y, : conversion failure on &#39;methyl−accepting chemotaxis protein Htr39&#39; in
## &#39;mbcsToSbcs&#39;: dot substituted for &lt;88&gt;</code></pre>
<pre><code>## Warning in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y, : conversion failure on &#39;methyl−accepting chemotaxis protein Htr39&#39; in
## &#39;mbcsToSbcs&#39;: dot substituted for &lt;92&gt;</code></pre>
<pre><code>## Warning in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y, : conversion failure on &#39;methyl−accepting chemotaxis protein Htr39&#39; in
## &#39;mbcsToSbcs&#39;: dot substituted for &lt;e2&gt;</code></pre>
<pre><code>## Warning in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y, : conversion failure on &#39;methyl−accepting chemotaxis protein Htr39&#39; in
## &#39;mbcsToSbcs&#39;: dot substituted for &lt;88&gt;</code></pre>
<pre><code>## Warning in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y, : conversion failure on &#39;methyl−accepting chemotaxis protein Htr39&#39; in
## &#39;mbcsToSbcs&#39;: dot substituted for &lt;92&gt;</code></pre>
<pre><code>## Warning in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y, : conversion failure on &#39;methyl−accepting chemotaxis protein Htr39&#39; in
## &#39;mbcsToSbcs&#39;: dot substituted for &lt;e2&gt;</code></pre>
<pre><code>## Warning in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y, : conversion failure on &#39;methyl−accepting chemotaxis protein Htr39&#39; in
## &#39;mbcsToSbcs&#39;: dot substituted for &lt;88&gt;</code></pre>
<pre><code>## Warning in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y, : conversion failure on &#39;methyl−accepting chemotaxis protein Htr39&#39; in
## &#39;mbcsToSbcs&#39;: dot substituted for &lt;92&gt;</code></pre>
<pre><code>## Warning in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y, : conversion failure on &#39;methyl−accepting chemotaxis protein Htr39&#39; in
## &#39;mbcsToSbcs&#39;: dot substituted for &lt;e2&gt;</code></pre>
<pre><code>## Warning in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y, : conversion failure on &#39;methyl−accepting chemotaxis protein Htr39&#39; in
## &#39;mbcsToSbcs&#39;: dot substituted for &lt;88&gt;</code></pre>
<pre><code>## Warning in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y, : conversion failure on &#39;methyl−accepting chemotaxis protein Htr39&#39; in
## &#39;mbcsToSbcs&#39;: dot substituted for &lt;92&gt;</code></pre>
<pre><code>## Warning in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y, : conversion failure on &#39;methyl−accepting chemotaxis protein Htr39&#39; in
## &#39;mbcsToSbcs&#39;: dot substituted for &lt;e2&gt;</code></pre>
<pre><code>## Warning in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y, : conversion failure on &#39;methyl−accepting chemotaxis protein Htr39&#39; in
## &#39;mbcsToSbcs&#39;: dot substituted for &lt;88&gt;</code></pre>
<pre><code>## Warning in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y, : conversion failure on &#39;methyl−accepting chemotaxis protein Htr39&#39; in
## &#39;mbcsToSbcs&#39;: dot substituted for &lt;92&gt;</code></pre>
<pre><code>## Warning in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y, : conversion failure on &#39;methyl−accepting chemotaxis protein Htr39&#39; in
## &#39;mbcsToSbcs&#39;: dot substituted for &lt;e2&gt;</code></pre>
<pre><code>## Warning in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y, : conversion failure on &#39;methyl−accepting chemotaxis protein Htr39&#39; in
## &#39;mbcsToSbcs&#39;: dot substituted for &lt;88&gt;</code></pre>
<pre><code>## Warning in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y, : conversion failure on &#39;methyl−accepting chemotaxis protein Htr39&#39; in
## &#39;mbcsToSbcs&#39;: dot substituted for &lt;92&gt;</code></pre>
<pre><code>## Warning in grid.Call.graphics(C_text, as.graphicsAnnot(x$label), x$x, x$y, : conversion failure on &#39;methyl−accepting chemotaxis protein Htr39&#39; in
## &#39;mbcsToSbcs&#39;: dot substituted for &lt;e2&gt;</code></pre>
<pre><code>## Warning in grid.Call.graphics(C_text, as.graphicsAnnot(x$label), x$x, x$y, : conversion failure on &#39;methyl−accepting chemotaxis protein Htr39&#39; in
## &#39;mbcsToSbcs&#39;: dot substituted for &lt;88&gt;</code></pre>
<pre><code>## Warning in grid.Call.graphics(C_text, as.graphicsAnnot(x$label), x$x, x$y, : conversion failure on &#39;methyl−accepting chemotaxis protein Htr39&#39; in
## &#39;mbcsToSbcs&#39;: dot substituted for &lt;92&gt;</code></pre>
<pre><code>## Warning in grid.Call.graphics(C_text, as.graphicsAnnot(x$label), x$x, x$y, : conversion failure on &#39;methyl−accepting chemotaxis protein Htr39&#39; in
## &#39;mbcsToSbcs&#39;: dot substituted for &lt;e2&gt;</code></pre>
<pre><code>## Warning in grid.Call.graphics(C_text, as.graphicsAnnot(x$label), x$x, x$y, : conversion failure on &#39;methyl−accepting chemotaxis protein Htr39&#39; in
## &#39;mbcsToSbcs&#39;: dot substituted for &lt;88&gt;</code></pre>
<pre><code>## Warning in grid.Call.graphics(C_text, as.graphicsAnnot(x$label), x$x, x$y, : conversion failure on &#39;methyl−accepting chemotaxis protein Htr39&#39; in
## &#39;mbcsToSbcs&#39;: dot substituted for &lt;92&gt;</code></pre>
<pre class="r"><code>## added significance stars by hand due to misalignment with above script</code></pre>
<pre class="r"><code>p0 &lt;- read.delim(&quot;data/p0_newgene.txt&quot;)
coords &lt;- read.delim(&quot;data/coords.txt&quot;)
coords01 &lt;- read.delim(&quot;data/coords01.txt&quot;)

coords$valt &lt;- ifelse(coords$val&gt;0, coords$val, coords$val + 0.1)
coords01$val01t &lt;- ifelse(coords01$val01&gt;0, coords01$val01, coords01$val01 + 0.1)


`Other Transposase`=c(&quot;HVO_B0155&quot;, &quot;HVO_B0152A&quot;, &quot;HVO_B0241&quot;, &quot;HVO_A0008&quot;, &quot;HVO_A0018&quot;, &quot;HVO_A0066&quot;, &quot;HVO_A0309&quot;,
                     &quot;HVO_1151&quot;, &quot;HVO_2075&quot;, &quot;HVO_2434&quot;, &quot;HVO_RS19915&quot;, &quot;HVO_2090A&quot;) # 12
`IS200-IS605 Family Transposase`=c(&quot;HVO_1150&quot;, &quot;HVO_B0242&quot;) # 2
`IS1595 Family Transposase`=c(&quot;HVO_RS19485&quot;, &quot;HVO_C0064&quot;, &quot;HVO_A0062&quot;, &quot;HVO_A0258&quot;, &quot;HVO_RS19515&quot;) # 4
`IS110 Family Transposase`=c(&quot;HVO_0556&quot;, &quot;HVO_A0016&quot;, &quot;HVO_A0128&quot;, &quot;HVO_A0014&quot;, &quot;HVO_A0279&quot;, &quot;HVO_0278&quot;, &quot;HVO_2821&quot;) # 3
`IS630 Family Transposase`=c(&quot;HVO_A0100&quot;, &quot;HVO_A0403&quot;, &quot;HVO_C0078&quot;, &quot;HVO_C0080&quot;) # 4
`ISH3 Family Transposase`=c(&quot;HVO_A0354&quot;, &quot;HVO_A0408&quot;, &quot;HVO_A0483&quot;, &quot;HVO_2050&quot;, &quot;HVO_A0353&quot;, &quot;HVO_1837&quot;, &quot;HVO_0263&quot;,
                          &quot;HVO_A0364&quot;, &quot;HVO_A0445&quot;, &quot;HVO_C0037&quot;, &quot;HVO_A0349&quot;) # 10
`IS6 Family Transposase`=c(&quot;HVO_A0265&quot;, &quot;HVO_C0065&quot;, &quot;HVO_C0020A&quot;) # 3
`IS5 Family Transposase`=c(&quot;HVO_A0117&quot;, &quot;HVO_A0057&quot;, &quot;HVO_A0240&quot;, &quot;HVO_C0068&quot;, &quot;HVO_0864&quot;, &quot;HVO_B0249&quot;, &quot;HVO_A0134&quot;) # 6
`IS4 Family Transposase`=c(&quot;HVO_B0152&quot;, &quot;HVO_B0157&quot;, &quot;HVO_B0234&quot;, &quot;HVO_A0077&quot;, &quot;HVO_A0081&quot;, &quot;HVO_A0110&quot;, &quot;HVO_A0125&quot;,
                         &quot;HVO_A0144&quot;, &quot;HVO_A0151&quot;, &quot;HVO_A0250&quot;, &quot;HVO_A0260&quot;, &quot;HVO_A0262&quot;, &quot;HVO_A0367&quot;, &quot;HVO_A0420&quot;,
                         &quot;HVO_A0423&quot;, &quot;HVO_A0431&quot;, &quot;HVO_A0433&quot;, &quot;HVO_A0449&quot;, &quot;HVO_0275&quot;, &quot;HVO_0276&quot;, &quot;HVO_0528&quot;,
                         &quot;HVO_0922&quot;, &quot;HVO_0926&quot;, &quot;HVO_1519&quot;, &quot;HVO_RS12025&quot;, &quot;HVO_1525&quot;, &quot;HVO_1782&quot;, &quot;HVO_1835&quot;,
                         &quot;HVO_1845&quot;, &quot;HVO_2051&quot;, &quot;HVO_2054&quot;, &quot;HVO_2252&quot;, &quot;HVO_2331&quot;, &quot;HVO_2466&quot;, &quot;HVO_2817&quot;,
                         &quot;HVO_2820&quot;, &quot;HVO_C0004&quot;, &quot;HVO_C0021&quot;, &quot;HVO_C0039&quot;, &quot;HVO_C0053&quot;, &quot;HVO_C0070&quot;, &quot;HVO_C0071&quot;,
                         &quot;HVO_A0007&quot;, &quot;HVO_A0238&quot;, &quot;HVO_A0355&quot;, &quot;HVO_0924&quot;, &quot;HVO_A0080&quot;, &quot;HVO_A0456&quot;, &quot;HVO_1106&quot;,
                         &quot;HVO_2830&quot;, &quot;HVO_C0013&quot;, &quot;HVO_A0116&quot;)

df &lt;- stack(mget(c(&#39;Other Transposase&#39;, &#39;IS200-IS605 Family Transposase&#39;, &#39;IS1595 Family Transposase&#39;, 
                   &#39;IS110 Family Transposase&#39;, &#39;IS630 Family Transposase&#39;, &#39;ISH3 Family Transposase&#39;,
                   &#39;IS6 Family Transposase&#39;, &#39;IS5 Family Transposase&#39;, &#39;IS4 Family Transposase&#39;)))
df &lt;- plyr::rename(df, c(&quot;values&quot;=&quot;old&quot;, &quot;ind&quot;=&quot;category&quot;))
df_sub &lt;- dplyr::distinct(merge(df, p0, all.x = TRUE, by = &quot;old&quot;))
df_sub$Row.names &lt;- NULL
df_sub &lt;- reshape::melt(df_sub, id.vars=c(&quot;old&quot;, &quot;category&quot;, &quot;new&quot;, &quot;prot&quot;, &quot;gene_name&quot;, &quot;arcog&quot;))
coords_sub &lt;- dplyr::distinct(merge(df, coords, all.x = TRUE, by = &quot;old&quot;))
coords01_sub &lt;- dplyr::distinct(merge(df, coords01, all.x = TRUE, by = &quot;old&quot;))
df_sub1 &lt;- merge(df_sub, coords_sub, all.x = TRUE)
df_sub1 &lt;- merge(df_sub1, coords01_sub, all.x = TRUE)
df_sub1 &lt;- df_sub1[!is.na(df_sub1$value), ]

ggplot(data=arrange(df_sub1, category), aes(x = old, y=value, fill = variable, group = variable)) +
# ggplot(data=arrange(df_sub1,category), aes(x = paste(new, prot, sep=&#39;\n&#39;), y=value, fill = variable)) +
    theme_bw() +
    geom_bar(stat=&quot;identity&quot;, position = position_dodge2(width = NULL, reverse=TRUE, padding = 0), width = 0.7) +
    # geom_text(position = position_dodge(width = 1), aes(x=category, y=0)) +
    geom_hline(yintercept = 0) +
    # geom_text(aes(0, 4.5, label = &quot;other&quot;, vjust = -0.7), color = &quot;grey47&quot;, size = 3) +
    geom_vline(xintercept = 12.5, col=&#39;grey47&#39;, lwd=0.5) + 
  geom_text(aes(12.5, 5, label = &quot;other&quot;, vjust = 1.2, hjust = 1), color = &quot;grey47&quot;, size = 3) +
    geom_vline(xintercept = 14.5, col=&#39;grey47&#39;, lwd=0.5) + 
  geom_text(aes(14.5, 5, label = &quot;IS200-IS605 Family&quot;, vjust = 1.2, hjust = 1), color = &quot;grey47&quot;, size = 3)+
    geom_vline(xintercept = 18.5, col=&#39;grey47&#39;, lwd=0.5) + 
  geom_text(aes(18.5, 5, label = &quot;IS1595 Family&quot;, vjust = 1.2, hjust = 1), color = &quot;grey47&quot;, size = 3) +
    geom_vline(xintercept = 21.5, col=&#39;grey47&#39;, lwd=0.5) + 
  geom_text(aes(21.5, 5, label = &quot;IS110 Family&quot;, vjust = 1.2, hjust = 1), color = &quot;grey47&quot;, size = 3) +
    geom_vline(xintercept = 25.5, col=&#39;grey47&#39;, lwd=0.5) + 
  geom_text(aes(25.5, 5, label = &quot;IS630 Family&quot;, vjust = 1.2, hjust = 1), color = &quot;grey47&quot;, size = 3) +
    geom_vline(xintercept = 35.5, col=&#39;grey47&#39;, lwd=0.5) + 
  geom_text(aes(35.5, 5, label = &quot;ISH3 Family&quot;, vjust = 1.2, hjust = 1), color = &quot;grey47&quot;, size = 3) +
    geom_vline(xintercept = 38.5, col=&#39;grey47&#39;, lwd=0.5) + 
  geom_text(aes(38.5, 5, label = &quot;IS6 Family&quot;, vjust = 1.2, hjust = 1), color = &quot;grey47&quot;, size = 3) +
    geom_vline(xintercept = 44.5, col=&#39;grey47&#39;, lwd=0.5) + 
  geom_text(aes(44.5, 5, label = &quot;IS5 Family&quot;, vjust = 1.2, hjust = 1), color = &quot;grey47&quot;, size = 3) +
    geom_text(aes(81.5, 5, label = &quot;IS4 Family&quot;, vjust = 1.1, hjust = 1), color = &quot;grey47&quot;, size = 3) +
    ylim(-5, 5) +
    theme(legend.position = c(0.1,0.94)) +
    theme(panel.grid.major = element_blank(), panel.grid.minor = element_blank()) +
    theme(axis.text.x = element_text(size = 8, hjust = 1, vjust = 1)) +
    theme(plot.title = element_text(hjust = 0.5), plot.margin = unit(c(0.5,0,0.5,1),&quot;cm&quot;)) +
    scale_fill_manual(labels = c(&quot;t00&quot;, &quot;t02&quot;, &quot;t04&quot;, &quot;t08&quot;, &quot;t24&quot;), 
                      values=c(&quot;#000000&quot;, &quot;#E69F00&quot;, &quot;#009E73&quot;, &quot;#0072B2&quot;, &quot;#D55E00&quot;)) +
    # geom_text(aes(y = valt), label = &quot;*&quot;, position = position_dodge(width = 0.7)) +
    # geom_text(aes(y = val01t), label = &quot;**&quot;, position = position_dodge(width = 0.7)) +
    aes(x = fct_inorder(paste(new), category)) +
    labs(x = NULL, y = expression(&quot;log&quot;[&quot;2&quot;]*&quot; fold change&quot;), fill = &quot;time vs shaking&quot;) +
    coord_flip()</code></pre>
<pre><code>## Warning in if (is.na(ordered)) {: the condition has length &gt; 1 and only the first element will be used</code></pre>
<pre><code>## Warning in if (ordered) &quot;ordered&quot;: the condition has length &gt; 1 and only the first element will be used</code></pre>
<pre><code>## Warning in if (is.na(ordered)) {: the condition has length &gt; 1 and only the first element will be used</code></pre>
<pre><code>## Warning in if (ordered) &quot;ordered&quot;: the condition has length &gt; 1 and only the first element will be used</code></pre>
<pre><code>## Warning: Removed 2 rows containing missing values (geom_bar).</code></pre>
<p><img src="hvo_mating_files/figure-html/long%20plot%20insertion%20seqs-1.png" width="480" /></p>
<pre class="r"><code>  # facet_wrap(~category, nrow = 1, scales = &quot;free_x&quot;)
# longplot
ggsave(&quot;plots/figS3_transposases.pdf&quot;, height = 15, width = 8)</code></pre>
<pre><code>## Warning in if (is.na(ordered)) {: the condition has length &gt; 1 and only the first element will be used</code></pre>
<pre><code>## Warning in if (ordered) &quot;ordered&quot;: the condition has length &gt; 1 and only the first element will be used</code></pre>
<pre><code>## Warning in if (is.na(ordered)) {: the condition has length &gt; 1 and only the first element will be used</code></pre>
<pre><code>## Warning in if (ordered) &quot;ordered&quot;: the condition has length &gt; 1 and only the first element will be used</code></pre>
<pre><code>## Warning: Removed 2 rows containing missing values (geom_bar).</code></pre>
<pre class="r"><code>### violin plot for transposases
## this chunk should run independently of rest of the script
library(beanplot)
library(purrr)
library(NOISeq)
library(ggplot2)
library(wesanderson)
library(grid)

count_data_raw &lt;- read.table(&quot;data/mating.mat&quot;, 
                             header=T, com=&#39;&#39;, check.names=F)
  count_data &lt;- round(count_data_raw, 0)
  count_data &lt;- count_data_raw[rowSums(count_data_raw) &gt; 10,]
  p0tmm &lt;- tmm(count_data, long = 1000, lc = 0, k = 0, refColumn = 1, logratioTrim = 0.3, 
               sumTrim = 0.05, doWeighting = TRUE, Acutoff = -1e+10)
  p0tmm &lt;- as.data.frame(p0tmm)
  p0tmm$Row.names &lt;- rownames(p0tmm)

p0 &lt;- read.delim(&quot;data/p0.txt&quot;)
p0mod &lt;- p0
  p0mod$de00 &lt;- NULL
  p0mod$de02 &lt;- NULL
  p0mod$de04 &lt;- NULL
  p0mod$de08 &lt;- NULL
  p0mod$de24 &lt;- NULL
p0mer &lt;- merge(p0mod, p0tmm, by = &quot;Row.names&quot;)

transposase &lt;- c(&quot;HVO_B0152&quot;, &quot;HVO_B0157&quot;, &quot;HVO_B0234&quot;, &quot;HVO_A0077&quot;, &quot;HVO_A0081&quot;, 
                 &quot;HVO_A0110&quot;, &quot;HVO_A0125&quot;, &quot;HVO_A0144&quot;, &quot;HVO_A0151&quot;, &quot;HVO_A0250&quot;, 
                 &quot;HVO_A0260&quot;, &quot;HVO_A0262&quot;, &quot;HVO_A0367&quot;, &quot;HVO_A0420&quot;, &quot;HVO_A0423&quot;, 
                 &quot;HVO_A0431&quot;, &quot;HVO_A0433&quot;, &quot;HVO_A0449&quot;, &quot;HVO_0275&quot;, &quot;HVO_0276&quot;, 
                 &quot;HVO_0528&quot;, &quot;HVO_0922&quot;, &quot;HVO_0926&quot;, &quot;HVO_1519&quot;, &quot;HVO_RS12025&quot;, 
                 &quot;HVO_1525&quot;, &quot;HVO_1782&quot;, &quot;HVO_1835&quot;, &quot;HVO_1845&quot;, &quot;HVO_2051&quot;, &quot;HVO_2054&quot;, 
                 &quot;HVO_2252&quot;, &quot;HVO_2331&quot;, &quot;HVO_2466&quot;, &quot;HVO_2817&quot;, &quot;HVO_2820&quot;, &quot;HVO_C0004&quot;, 
                 &quot;HVO_C0021&quot;, &quot;HVO_C0039&quot;, &quot;HVO_C0053&quot;, &quot;HVO_C0070&quot;, &quot;HVO_C0071&quot;, 
                 &quot;HVO_A0007&quot;, &quot;HVO_A0238&quot;, &quot;HVO_A0355&quot;, &quot;HVO_0924&quot;, &quot;HVO_A0080&quot;, 
                 &quot;HVO_A0456&quot;, &quot;HVO_1106&quot;, &quot;HVO_2830&quot;, &quot;HVO_C0013&quot;, &quot;HVO_A0116&quot;, 
                 &quot;HVO_A0117&quot;, &quot;HVO_A0057&quot;, &quot;HVO_A0240&quot;, &quot;HVO_C0068&quot;, &quot;HVO_0864&quot;, 
                 &quot;HVO_B0249&quot;, &quot;HVO_A0134&quot;, &quot;HVO_A0265&quot;, &quot;HVO_C0065&quot;, &quot;HVO_C0020A&quot;, 
                 &quot;HVO_A0354&quot;, &quot;HVO_A0408&quot;, &quot;HVO_A0483&quot;, &quot;HVO_2050&quot;, &quot;HVO_A0353&quot;, 
                 &quot;HVO_1837&quot;, &quot;HVO_0263&quot;, &quot;HVO_A0364&quot;, &quot;HVO_A0445&quot;, &quot;HVO_C0037&quot;, 
                 &quot;HVO_A0349&quot;,&quot;HVO_A0100&quot;, &quot;HVO_A0403&quot;, &quot;HVO_C0078&quot;, &quot;HVO_C0080&quot;,
                 &quot;HVO_0556&quot;, &quot;HVO_A0016&quot;, &quot;HVO_A0128&quot;, &quot;HVO_A0014&quot;, &quot;HVO_A0279&quot;, 
                 &quot;HVO_0278&quot;, &quot;HVO_2821&quot;,&quot;HVO_RS19485&quot;, &quot;HVO_C0064&quot;, &quot;HVO_A0062&quot;, 
                 &quot;HVO_A0258&quot;, &quot;HVO_RS19515&quot;,&quot;HVO_1150&quot;, &quot;HVO_B0242&quot;,&quot;HVO_B0155&quot;, 
                 &quot;HVO_B0152A&quot;, &quot;HVO_B0241&quot;, &quot;HVO_A0008&quot;, &quot;HVO_A0018&quot;, &quot;HVO_A0066&quot;, 
                 &quot;HVO_A0309&quot;, &quot;HVO_1151&quot;, &quot;HVO_2075&quot;, &quot;HVO_2434&quot;, &quot;HVO_RS19915&quot;, &quot;HVO_2090A&quot;)

## tlist is a list of transposases by family
tlist &lt;- read.delim(&quot;annotations/tlist.txt&quot;, header = FALSE)
  tlist &lt;- melt(tlist, id.vars=&quot;V1&quot;)
  tlist$variable &lt;- NULL
p0sub &lt;- subset(p0mer, old %in% transposase)
  p0sub$Row.names &lt;- NULL
  p0sub$shaking &lt;- rowMeans(p0sub[c(&#39;ap0_quant&#39;, &#39;bp0_quant&#39;)], na.rm=TRUE)
  p0sub$t00 &lt;- rowMeans(p0sub[c(&#39;a00_quant&#39;, &#39;b00_quant&#39;)], na.rm=TRUE)
  p0sub$t02 &lt;- rowMeans(p0sub[c(&#39;a02_quant&#39;, &#39;b02_quant&#39;)], na.rm=TRUE)
  p0sub$t04 &lt;- rowMeans(p0sub[c(&#39;a04_quant&#39;, &#39;b04_quant&#39;)], na.rm=TRUE)
  p0sub$t08 &lt;- rowMeans(p0sub[c(&#39;a08_quant&#39;, &#39;b08_quant&#39;)], na.rm=TRUE)
  p0sub$t24 &lt;- rowMeans(p0sub[c(&#39;a24_quant&#39;, &#39;b24_quant&#39;)], na.rm=TRUE)
  p0sub &lt;- merge(p0sub, tlist, by.x = &quot;old&quot;, by.y = &quot;value&quot;, all.x = TRUE)
  p0sub &lt;- p0sub[, -c(4:19)]
  p0sub &lt;- melt(p0sub, id.vars=c(&quot;old&quot;, &quot;new&quot;, &quot;prot&quot;, &quot;V1&quot;))

#beanplot(value~V1, data = p0sub, log = &quot;&quot;, col = &quot;bisque&quot;, method = &quot;jitter&quot;)

viol &lt;- ggplot(data = p0sub, aes(x = variable, y = value)) +
  geom_violin() +
  geom_jitter(height = 0, width = 0.2, aes(color = V1), alpha = 0.5) +
  theme(axis.text.x = element_text(angle = 90, hjust = 1, vjust = 0.5)) +
  scale_color_manual(values = wes_palette(n = 9, name = &quot;Darjeeling1&quot;, type = &quot;continuous&quot;), name = &quot;&quot;) +
  labs(x = NULL, y = expression(&quot;log&quot;[&quot;10&quot;]*&quot; TMM values&quot;)) +
  scale_y_continuous(trans=&#39;log10&#39;) +
  theme(legend.position = c(0.82, 0.18)) +
  theme(legend.key.height=unit(0.3,&quot;line&quot;)) +
  theme(legend.key.width=unit(1,&quot;line&quot;)) +
  theme(legend.background = element_rect(fill = &quot;transparent&quot;)) +
  theme(plot.margin = unit(c(0,1,0,0.3),&quot;cm&quot;)) +
  theme(panel.grid.major = element_blank(), panel.grid.minor = element_blank())
ggsave(&quot;plots/violin_transposase.pdf&quot;, height = 4, width = 7, units = &quot;in&quot;, dpi = 600)</code></pre>
<pre><code>## Warning: Transformation introduced infinite values in continuous y-axis

## Warning: Transformation introduced infinite values in continuous y-axis</code></pre>
<pre><code>## Warning: Removed 5 rows containing non-finite values (stat_ydensity).</code></pre>
<pre class="r"><code>viol</code></pre>
<pre><code>## Warning: Transformation introduced infinite values in continuous y-axis</code></pre>
<pre><code>## Warning: Transformation introduced infinite values in continuous y-axis</code></pre>
<pre><code>## Warning: Removed 5 rows containing non-finite values (stat_ydensity).</code></pre>
<p><img src="hvo_mating_files/figure-html/violin%20plot-1.png" width="672" /></p>
<pre class="r"><code># manhattan plot
#### this chunk should run independently of the rest of the script
library(manhattanly)
library(reshape)
library(dplyr)

### files made in earlier chunks that were saved for read-in
hvo_map &lt;- read.delim(&quot;annotations/hvo_rmap.txt&quot;)
logfc &lt;- read.delim(&quot;data/logfc.txt&quot;)
sig &lt;- read.delim(&quot;data/sig.txt&quot;)

## use hvo_map to start the merge dataframe for the manhattan plot input (see HapMap)
man_map &lt;- NULL
man_map$CHR &lt;- hvo_map$genomic_accession
man_map$CHR2 &lt;- hvo_map$genomic_accession
man_map$BP &lt;- hvo_map$start
man_map$SNP &lt;- hvo_map$locus_tag
man_map$tmp &lt;- hvo_map$ncbi_id
man_map$gene &lt;- hvo_map$protein
man_map &lt;- as.data.frame(man_map)

## use the significance and fold change data from each timepoint compared to a shaking control to make input
sig_melt &lt;- melt(sig, id.vars = &quot;Row.names&quot;)
  sig_melt &lt;- dplyr::rename(sig_melt, prob = value, timepoint = variable)
  sig_melt$P &lt;- with(sig_melt, 1 - prob)
logfc_melt &lt;- melt(logfc, id.vars = &quot;Row.names&quot;)
  logfc_melt &lt;- dplyr::rename(logfc_melt, EFFECTSIZE = value, timepoint = variable)

man_in &lt;- merge(logfc_melt, sig_melt, by = c(&quot;Row.names&quot;, &quot;timepoint&quot;))
man_in &lt;- merge(man_in, man_map, by.x = &quot;Row.names&quot;, by.y = &quot;tmp&quot;)

## rename the chromosome/plasmids and clean up the dataframe
man_in$CHR &lt;- gsub(&#39;NC_013967.1&#39;, 1, man_in$CHR)
  man_in$CHR &lt;- gsub(&#39;NC_013968.1&#39;, 10, man_in$CHR)
  man_in$CHR &lt;- gsub(&#39;NC_013965.1&#39;, 20, man_in$CHR)
  man_in$CHR &lt;- gsub(&#39;NC_013964.1&#39;, 30, man_in$CHR)
  man_in$CHR &lt;- gsub(&#39;NC_013966.1&#39;, 40, man_in$CHR)
  man_in$CHR &lt;- as.numeric(man_in$CHR)
man_in$CHR2 &lt;- gsub(&#39;NC_013967.1&#39;, &#39;Chromosome&#39;, man_in$CHR2)
  man_in$CHR2 &lt;- gsub(&#39;NC_013968.1&#39;, &#39;pHV1&#39;, man_in$CHR2)
  man_in$CHR2 &lt;- gsub(&#39;NC_013965.1&#39;, &#39;pHV2&#39;, man_in$CHR2)
  man_in$CHR2 &lt;- gsub(&#39;NC_013964.1&#39;, &#39;pHV3&#39;, man_in$CHR2)
  man_in$CHR2 &lt;- gsub(&#39;NC_013966.1&#39;, &#39;pHV4&#39;, man_in$CHR2)
man_in$Row.names &lt;- NULL
man_in &lt;- man_in[c(&quot;CHR&quot;, &quot;CHR2&quot;, &quot;BP&quot;, &quot;P&quot;, &quot;SNP&quot;, &quot;EFFECTSIZE&quot;, &quot;timepoint&quot;, &quot;prob&quot;, &quot;gene&quot;)]

manobj &lt;- manhattanr(man_in, chr = &quot;CHR&quot;, bp = &quot;BP&quot;, p = &quot;P&quot;, snp = &quot;SNP&quot;, 
                     annotation1 = &quot;timepoint&quot;, annotation2 = &quot;EFFECTSIZE&quot;, gene = &quot;gene&quot;, logp = TRUE)
intplot &lt;- manhattanly(manobj, point_size = 5, labelChr = c(&quot;Chr&quot;, &quot;pHV1&quot;, &quot;pHV3&quot;, &quot;pHV4&quot;), xlab = &quot;Haloferax volcanii&quot;,
            highlight = c(&quot;HVO_RS00735&quot;, &quot;HVO_RS00740&quot;, &quot;HVO_RS00765&quot;, &quot;HVO_RS00775&quot;, &quot;HVO_RS01150&quot;, &quot;HVO_RS01185&quot;, &quot;HVO_RS01190&quot;, &quot;HVO_RS01220&quot;, &quot;HVO_RS01895&quot;, &quot;HVO_RS01900&quot;, &quot;HVO_RS01930&quot;, &quot;HVO_RS01935&quot;, &quot;HVO_RS01940&quot;, &quot;HVO_RS02130&quot;, &quot;HVO_RS02155&quot;, &quot;HVO_RS02175&quot;, &quot;HVO_RS02220&quot;, &quot;HVO_RS02235&quot;, &quot;HVO_RS02240&quot;, &quot;HVO_RS02380&quot;, &quot;HVO_RS02395&quot;, &quot;HVO_RS02400&quot;, &quot;HVO_RS02430&quot;, &quot;HVO_RS02445&quot;, &quot;HVO_RS02500&quot;, &quot;HVO_RS02535&quot;, &quot;HVO_RS02910&quot;, &quot;HVO_RS02920&quot;, &quot;HVO_RS02960&quot;, &quot;HVO_RS02990&quot;, &quot;HVO_RS02995&quot;, &quot;HVO_RS03005&quot;, &quot;HVO_RS03015&quot;, &quot;HVO_RS03080&quot;, &quot;HVO_RS03235&quot;, &quot;HVO_RS03435&quot;, &quot;HVO_RS03440&quot;, &quot;HVO_RS03480&quot;, &quot;HVO_RS03490&quot;, &quot;HVO_RS03670&quot;, &quot;HVO_RS03720&quot;, &quot;HVO_RS03730&quot;, &quot;HVO_RS03765&quot;, &quot;HVO_RS03770&quot;, &quot;HVO_RS03825&quot;, &quot;HVO_RS03835&quot;, &quot;HVO_RS03870&quot;, &quot;HVO_RS04000&quot;, &quot;HVO_RS05975&quot;, &quot;HVO_RS06035&quot;, &quot;HVO_RS06040&quot;, &quot;HVO_RS06050&quot;, &quot;HVO_RS07250&quot;, &quot;HVO_RS07375&quot;, &quot;HVO_RS08840&quot;, &quot;HVO_RS09115&quot;, &quot;HVO_RS09120&quot;, &quot;HVO_RS09130&quot;, &quot;HVO_RS10010&quot;, &quot;HVO_RS10230&quot;, &quot;HVO_RS10235&quot;, &quot;HVO_RS12020&quot;, &quot;HVO_RS12025&quot;, &quot;HVO_RS12030&quot;, &quot;HVO_RS13270&quot;, &quot;HVO_RS13510&quot;, &quot;HVO_RS13560&quot;, &quot;HVO_RS14565&quot;, &quot;HVO_RS14570&quot;, &quot;HVO_RS14575&quot;, &quot;HVO_RS14680&quot;, &quot;HVO_RS15525&quot;, &quot;HVO_RS15925&quot;, &quot;HVO_RS16425&quot;, &quot;HVO_RS16590&quot;, &quot;HVO_RS18325&quot;, &quot;HVO_RS18335&quot;, &quot;HVO_RS18340&quot;, &quot;HVO_RS18385&quot;, &quot;HVO_RS19305&quot;, &quot;HVO_RS19350&quot;, &quot;HVO_RS19370&quot;, &quot;HVO_RS19430&quot;, &quot;HVO_RS19435&quot;, &quot;HVO_RS19485&quot;, &quot;HVO_RS19490&quot;, &quot;HVO_RS19515&quot;, &quot;HVO_RS19535&quot;, &quot;HVO_RS19540&quot;, &quot;HVO_RS19550&quot;, &quot;HVO_RS19560&quot;, &quot;HVO_RS19565&quot;, &quot;HVO_RS19605&quot;, &quot;HVO_RS19775&quot;, &quot;HVO_RS19810&quot;, &quot;HVO_RS19915&quot;, &quot;HVO_RS19920&quot;, &quot;HVO_RS19935&quot;, &quot;HVO_RS19970&quot;, &quot;HVO_RS20195&quot;, &quot;HVO_RS20245&quot;, &quot;HVO_RS20355&quot;, &quot;HVO_RS20415&quot;))</code></pre>
<pre><code>## Warning in manhattanly.manhattanr(manobj, point_size = 5, labelChr = c(&quot;Chr&quot;, : You&#39;re trying to highlight SNPs that don&#39;t exist in your results.</code></pre>
<pre class="r"><code>htmlwidgets::saveWidget(widget = intplot, file = &quot;manhattan_transposases.html&quot;, selfcontained = TRUE)</code></pre>
<pre><code>## Warning: Ignoring 455 observations</code></pre>
<pre><code>## Warning: Ignoring 12 observations</code></pre>
<pre><code>## Warning: Ignoring 284 observations</code></pre>
<pre><code>## Warning: Ignoring 166 observations</code></pre>
<pre><code>## Warning: Ignoring 3 observations</code></pre>
<pre><code>## Warning: Ignoring 1 observations</code></pre>
<pre class="r"><code>intplotc &lt;- manhattanly(manobj, point_size = 5, labelChr = c(&quot;Chr&quot;, &quot;pHV1&quot;, &quot;pHV3&quot;, &quot;pHV4&quot;), xlab = &quot;Haloferax volcanii&quot;,
            highlight = c(&quot;HVO_RS02755&quot;, &quot;HVO_RS02760&quot;, &quot;HVO_RS02765&quot;, &quot;HVO_RS02770&quot;, &quot;HVO_RS02780&quot;, &quot;HVO_RS02785&quot;, &quot;HVO_RS02790&quot;),
            highlight_color = &quot;blue&quot;)
htmlwidgets::saveWidget(widget = intplotc, file = &quot;manhattan_crisprs.html&quot;, selfcontained = TRUE)</code></pre>
<pre><code>## Warning: Ignoring 455 observations</code></pre>
<pre><code>## Warning: Ignoring 12 observations</code></pre>
<pre><code>## Warning: Ignoring 284 observations</code></pre>
<pre><code>## Warning: Ignoring 166 observations</code></pre>
<pre class="r"><code>hvos &lt;- c(&quot;HVO_RS08530&quot;, &quot;HVO_RS09750&quot;, &quot;HVO_RS12015&quot;, &quot;HVO_RS12035&quot;, &quot;HVO_RS12055&quot;, &quot;HVO_RS13185&quot;, &quot;HVO_RS13625&quot;,
                          &quot;HVO_RS14535&quot;, &quot;HVO_RS15825&quot;, &quot;HVO_RS20150&quot;, &quot;HVO_RS20160&quot;,
                          &quot;HVO_RS04470&quot;, &quot;HVO_RS08195&quot;, &quot;HVO_RS08530&quot;, &quot;HVO_RS12015&quot;, &quot;HVO_RS12035&quot;, &quot;HVO_RS12040&quot;, &quot;HVO_RS12045&quot;, 
                          &quot;HVO_RS12050&quot;, &quot;HVO_RS12055&quot;, &quot;HVO_RS14660&quot;, &quot;HVO_RS20145&quot;, &quot;HVO_RS20150&quot;, &quot;HVO_RS20155&quot;, &quot;HVO_RS20160&quot;,
                          &quot;HVO_RS20165&quot;,
                          &quot;HVO_RS14530&quot;, &quot;HVO_RS14535&quot;, &quot;HVO_RS14540&quot;, &quot;HVO_RS14545&quot;, &quot;HVO_RS14550&quot;, &quot;HVO_RS14555&quot;, &quot;HVO_RS14560&quot;, 
                          &quot;HVO_RS14570&quot;, &quot;HVO_RS14580&quot;, &quot;HVO_RS14585&quot;, &quot;HVO_RS14595&quot;, &quot;HVO_RS14600&quot;, &quot;HVO_RS14605&quot;, &quot;HVO_RS14610&quot;,
                          &quot;HVO_RS14615&quot;, &quot;HVO_RS14620&quot;, &quot;HVO_RS20230&quot;, &quot;HVO_RS20235&quot;, &quot;HVO_RS20240&quot;)
#count(man_in, vars = hvos)
library(expss)
for(i in hvos){
tmp &lt;- count_if(i, man_in)
print(paste(i, &quot;=&quot;, tmp))
}</code></pre>
<pre><code>## [1] &quot;HVO_RS08530 = 5&quot;
## [1] &quot;HVO_RS09750 = 10&quot;
## [1] &quot;HVO_RS12015 = 5&quot;
## [1] &quot;HVO_RS12035 = 15&quot;
## [1] &quot;HVO_RS12055 = 15&quot;
## [1] &quot;HVO_RS13185 = 5&quot;
## [1] &quot;HVO_RS13625 = 5&quot;
## [1] &quot;HVO_RS14535 = 5&quot;
## [1] &quot;HVO_RS15825 = 5&quot;
## [1] &quot;HVO_RS20150 = 5&quot;
## [1] &quot;HVO_RS20160 = 5&quot;
## [1] &quot;HVO_RS04470 = 15&quot;
## [1] &quot;HVO_RS08195 = 5&quot;
## [1] &quot;HVO_RS08530 = 5&quot;
## [1] &quot;HVO_RS12015 = 5&quot;
## [1] &quot;HVO_RS12035 = 15&quot;
## [1] &quot;HVO_RS12040 = 5&quot;
## [1] &quot;HVO_RS12045 = 5&quot;
## [1] &quot;HVO_RS12050 = 5&quot;
## [1] &quot;HVO_RS12055 = 15&quot;
## [1] &quot;HVO_RS14660 = 5&quot;
## [1] &quot;HVO_RS20145 = 5&quot;
## [1] &quot;HVO_RS20150 = 5&quot;
## [1] &quot;HVO_RS20155 = 5&quot;
## [1] &quot;HVO_RS20160 = 5&quot;
## [1] &quot;HVO_RS20165 = 5&quot;
## [1] &quot;HVO_RS14530 = 5&quot;
## [1] &quot;HVO_RS14535 = 5&quot;
## [1] &quot;HVO_RS14540 = 5&quot;
## [1] &quot;HVO_RS14545 = 5&quot;
## [1] &quot;HVO_RS14550 = 5&quot;
## [1] &quot;HVO_RS14555 = 5&quot;
## [1] &quot;HVO_RS14560 = 5&quot;
## [1] &quot;HVO_RS14570 = 5&quot;
## [1] &quot;HVO_RS14580 = 10&quot;
## [1] &quot;HVO_RS14585 = 15&quot;
## [1] &quot;HVO_RS14595 = 10&quot;
## [1] &quot;HVO_RS14600 = 20&quot;
## [1] &quot;HVO_RS14605 = 5&quot;
## [1] &quot;HVO_RS14610 = 5&quot;
## [1] &quot;HVO_RS14615 = 5&quot;
## [1] &quot;HVO_RS14620 = 5&quot;
## [1] &quot;HVO_RS20230 = 5&quot;
## [1] &quot;HVO_RS20235 = 5&quot;
## [1] &quot;HVO_RS20240 = 5&quot;</code></pre>
<pre class="r"><code>intplot_agl &lt;- manhattanly(manobj, point_size = 5, labelChr = c(&quot;Chr&quot;, &quot;pHV1&quot;, &quot;pHV3&quot;, &quot;pHV4&quot;), xlab = &quot;Haloferax volcanii&quot;,
            highlight = c(&quot;HVO_RS09750&quot;, &quot;HVO_RS13185&quot;, &quot;HVO_RS13625&quot;, &quot;HVO_RS15825&quot;,
                          &quot;HVO_RS04470&quot;, &quot;HVO_RS08195&quot;, &quot;HVO_RS08530&quot;, &quot;HVO_RS12015&quot;, &quot;HVO_RS12035&quot;, &quot;HVO_RS12040&quot;, &quot;HVO_RS12045&quot;, 
                          &quot;HVO_RS12050&quot;, &quot;HVO_RS12055&quot;, &quot;HVO_RS14660&quot;, &quot;HVO_RS20145&quot;, &quot;HVO_RS20150&quot;, &quot;HVO_RS20155&quot;, &quot;HVO_RS20160&quot;,
                          &quot;HVO_RS20165&quot;,
                          &quot;HVO_RS14530&quot;, &quot;HVO_RS14535&quot;, &quot;HVO_RS14540&quot;, &quot;HVO_RS14545&quot;, &quot;HVO_RS14550&quot;, &quot;HVO_RS14555&quot;, &quot;HVO_RS14560&quot;, 
                          &quot;HVO_RS14570&quot;, &quot;HVO_RS14580&quot;, &quot;HVO_RS14585&quot;, &quot;HVO_RS14595&quot;, &quot;HVO_RS14600&quot;, &quot;HVO_RS14605&quot;, &quot;HVO_RS14610&quot;,
                          &quot;HVO_RS14615&quot;, &quot;HVO_RS14620&quot;, &quot;HVO_RS20230&quot;, &quot;HVO_RS20235&quot;, &quot;HVO_RS20240&quot;),
            highlight_color = c(&quot;blue&quot;, &quot;blue&quot;, &quot;blue&quot;, &quot;blue&quot;,
                                &quot;blue&quot;, &quot;blue&quot;, &quot;blue&quot;, &quot;blue&quot;,
                                &quot;blue&quot;, &quot;blue&quot;, &quot;blue&quot;, &quot;blue&quot;,
                                &quot;blue&quot;, &quot;blue&quot;, &quot;blue&quot;, &quot;blue&quot;,
                                &quot;blue&quot;, &quot;blue&quot;, &quot;blue&quot;, &quot;blue&quot;,
                                &quot;green&quot;, &quot;green&quot;, &quot;green&quot;, &quot;green&quot;, &quot;green&quot;, &quot;green&quot;, &quot;green&quot;, &quot;green&quot;, &quot;green&quot;, &quot;green&quot;, &quot;green&quot;, 
                                &quot;green&quot;, &quot;green&quot;, &quot;green&quot;, &quot;green&quot;,
                                &quot;green&quot;, &quot;green&quot;, &quot;green&quot;, &quot;green&quot;, &quot;green&quot;, &quot;green&quot;, &quot;green&quot;, &quot;green&quot;, &quot;green&quot;, &quot;green&quot;, &quot;green&quot;, 
                                &quot;green&quot;, &quot;green&quot;, &quot;green&quot;, &quot;green&quot;,
                                &quot;green&quot;, &quot;green&quot;, &quot;green&quot;, &quot;green&quot;, &quot;green&quot;, &quot;green&quot;, &quot;green&quot;, &quot;green&quot;, &quot;green&quot;, &quot;green&quot;, &quot;green&quot;, 
                                &quot;green&quot;, &quot;green&quot;, &quot;green&quot;, &quot;green&quot;,
                                &quot;green&quot;, &quot;green&quot;, &quot;green&quot;, &quot;green&quot;, &quot;green&quot;, &quot;green&quot;, &quot;green&quot;, &quot;green&quot;, &quot;green&quot;, &quot;green&quot;, &quot;green&quot;, 
                                &quot;green&quot;, &quot;green&quot;, &quot;green&quot;, &quot;green&quot;,
                                &quot;green&quot;, &quot;green&quot;, &quot;green&quot;, &quot;green&quot;, &quot;green&quot;, &quot;green&quot;, &quot;green&quot;, &quot;green&quot;, &quot;green&quot;, &quot;green&quot;, &quot;green&quot;, 
                                &quot;green&quot;, &quot;green&quot;, &quot;green&quot;, &quot;green&quot;,
                                &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;,
                                &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;,
                                &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;,
                                &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;,
                                &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;,
                                &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;,
                                &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;,
                                &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;,
                                &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;,
                                &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;, &quot;orange&quot;))
htmlwidgets::saveWidget(widget = intplot_agl, file = &quot;manhattan_agl.html&quot;, selfcontained = TRUE)</code></pre>
<pre><code>## Warning: Ignoring 455 observations</code></pre>
<pre><code>## Warning: Ignoring 12 observations</code></pre>
<pre><code>## Warning: Ignoring 284 observations</code></pre>
<pre><code>## Warning: Ignoring 166 observations</code></pre>
<pre><code>## Warning: Ignoring 1 observations</code></pre>
<pre class="r"><code>### from https://genome.sph.umich.edu/wiki/Code_Sample:_Generating_Manhattan_Plots_in_R
library(lattice)
manhattan.plot &lt;- function(chr, pos, pvalue, 
    sig.level=NA, annotate=NULL, ann.default=list(),
    should.thin=T, thin.pos.places=2, thin.logp.places=2, 
    xlab=&quot;Chromosome&quot;, ylab=expression(-log[10](p-value)),
    col=c(&quot;gray&quot;,&quot;darkgray&quot;), panel.extra=NULL, pch=20, cex=0.8,...) {

    if (length(chr)==0) stop(&quot;chromosome vector is empty&quot;)
    if (length(pos)==0) stop(&quot;position vector is empty&quot;)
    if (length(pvalue)==0) stop(&quot;pvalue vector is empty&quot;)

    #make sure we have an ordered factor
    if(!is.ordered(chr)) {
        chr &lt;- ordered(chr)
    } else {
        chr &lt;- chr[,drop=T]
    }

    #make sure positions are in kbp
    if (any(pos&gt;1e6)) pos&lt;-pos/1e6;

    #calculate absolute genomic position
    #from relative chromosomal positions
    posmin &lt;- tapply(pos,chr, min);
    posmax &lt;- tapply(pos,chr, max);
    posshift &lt;- head(c(0,cumsum(posmax)),-1);
    names(posshift) &lt;- levels(chr)
    genpos &lt;- pos + posshift[chr];
    getGenPos&lt;-function(cchr, cpos) {
        p&lt;-posshift[as.character(cchr)]+cpos
        return(p)
    }

    #parse annotations
    grp &lt;- NULL
    ann.settings &lt;- list()
    label.default&lt;-list(x=&quot;peak&quot;,y=&quot;peak&quot;,adj=NULL, pos=3, offset=0.5, 
        col=NULL, fontface=NULL, fontsize=NULL, show=F)
    parse.label&lt;-function(rawval, groupname) {
        r&lt;-list(text=groupname)
        if(is.logical(rawval)) {
            if(!rawval) {r$show &lt;- F}
        } else if (is.character(rawval) || is.expression(rawval)) {
            if(nchar(rawval)&gt;=1) {
                r$text &lt;- rawval
            }
        } else if (is.list(rawval)) {
            r &lt;- modifyList(r, rawval)
        }
        return(r)
    }

    if(!is.null(annotate)) {
        if (is.list(annotate)) {
            grp &lt;- annotate[[1]]
        } else {
            grp &lt;- annotate
        } 
        if (!is.factor(grp)) {
            grp &lt;- factor(grp)
        }
    } else {
        grp &lt;- factor(rep(1, times=length(pvalue)))
    }
  
    ann.settings&lt;-vector(&quot;list&quot;, length(levels(grp)))
    ann.settings[[1]]&lt;-list(pch=pch, col=col, cex=cex, fill=col, label=label.default)

    if (length(ann.settings)&gt;1) { 
        lcols&lt;-trellis.par.get(&quot;superpose.symbol&quot;)$col 
        lfills&lt;-trellis.par.get(&quot;superpose.symbol&quot;)$fill
        for(i in 2:length(levels(grp))) {
            ann.settings[[i]]&lt;-list(pch=pch, 
                col=lcols[(i-2) %% length(lcols) +1 ], 
                fill=lfills[(i-2) %% length(lfills) +1 ], 
                cex=cex, label=label.default);
            ann.settings[[i]]$label$show &lt;- T
        }
        names(ann.settings)&lt;-levels(grp)
    }
    for(i in 1:length(ann.settings)) {
        if (i&gt;1) {ann.settings[[i]] &lt;- modifyList(ann.settings[[i]], ann.default)}
        ann.settings[[i]]$label &lt;- modifyList(ann.settings[[i]]$label, 
            parse.label(ann.settings[[i]]$label, levels(grp)[i]))
    }
    if(is.list(annotate) &amp;&amp; length(annotate)&gt;1) {
        user.cols &lt;- 2:length(annotate)
        ann.cols &lt;- c()
        if(!is.null(names(annotate[-1])) &amp;&amp; all(names(annotate[-1])!=&quot;&quot;)) {
            ann.cols&lt;-match(names(annotate)[-1], names(ann.settings))
        } else {
            ann.cols&lt;-user.cols-1
        }
        for(i in seq_along(user.cols)) {
            if(!is.null(annotate[[user.cols[i]]]$label)) {
                annotate[[user.cols[i]]]$label&lt;-parse.label(annotate[[user.cols[i]]]$label, 
                    levels(grp)[ann.cols[i]])
            }
            ann.settings[[ann.cols[i]]]&lt;-modifyList(ann.settings[[ann.cols[i]]], 
                annotate[[user.cols[i]]])
        }
    }
    rm(annotate)

    #reduce number of points plotted
    if(should.thin) {
        thinned &lt;- unique(data.frame(
            logp=round(-log10(pvalue),thin.logp.places), 
            pos=round(genpos,thin.pos.places), 
            chr=chr,
            grp=grp)
        )
        logp &lt;- thinned$logp
        genpos &lt;- thinned$pos
        chr &lt;- thinned$chr
        grp &lt;- thinned$grp
        rm(thinned)
    } else {
        logp &lt;- -log10(pvalue)
    }
    rm(pos, pvalue)
    gc()

    #custom axis to print chromosome names
    axis.chr &lt;- function(side,...) {
        if(side==&quot;bottom&quot;) {
            panel.axis(side=side, outside=T,
                at=((posmax+posmin)/2+posshift),
                labels=levels(chr), 
                ticks=F, rot=0,
                check.overlap=F
            )
        } else if (side==&quot;top&quot; || side==&quot;right&quot;) {
            panel.axis(side=side, draw.labels=F, ticks=F);
        }
        else {
            axis.default(side=side,...);
        }
     }

    #make sure the y-lim covers the range (plus a bit more to look nice)
    prepanel.chr&lt;-function(x,y,...) { 
        A&lt;-list();
        maxy&lt;-ceiling(max(y, ifelse(!is.na(sig.level), -log10(sig.level), 0)))+.5;
        A$ylim=c(0,maxy);
        A;
    }

    xyplot(logp~genpos, chr=chr, groups=grp,
        axis=axis.chr, ann.settings=ann.settings, 
#       auto.key=TRUE,
        prepanel=prepanel.chr, scales=list(axs=&quot;i&quot;),
        panel=function(x, y, ..., getgenpos) {
            if(!is.na(sig.level)) {
                #add significance line (if requested)
                panel.abline(h=-log10(sig.level), lty=2);
            }
            panel.superpose(x, y, ..., getgenpos=getgenpos);
            if(!is.null(panel.extra)) {
                panel.extra(x,y, getgenpos, ...)
            }
        },
        panel.groups = function(x,y,..., subscripts, group.number) {
            A&lt;-list(...)
            #allow for different annotation settings
            gs &lt;- ann.settings[[group.number]]
            A$col.symbol &lt;- gs$col[(as.numeric(chr[subscripts])-1) %% length(gs$col) + 1]    
            A$cex &lt;- gs$cex[(as.numeric(chr[subscripts])-1) %% length(gs$cex) + 1]
            A$alpha &lt;- 0.4
            A$pch &lt;- gs$pch[(as.numeric(chr[subscripts])-1) %% length(gs$pch) + 1]
            A$fill &lt;- gs$fill[(as.numeric(chr[subscripts])-1) %% length(gs$fill) + 1]
            A$x &lt;- x
            A$y &lt;- y
            do.call(&quot;panel.xyplot&quot;, A)
            #draw labels (if requested)
            if(gs$label$show) {
                gt&lt;-gs$label
                names(gt)[which(names(gt)==&quot;text&quot;)]&lt;-&quot;labels&quot;
                gt$show&lt;-NULL
                if(is.character(gt$x) | is.character(gt$y)) {
                    peak = which.max(y)
                    center = mean(range(x))
                    if (is.character(gt$x)) {
                        if(gt$x==&quot;peak&quot;) {gt$x&lt;-x[peak]}
                        if(gt$x==&quot;center&quot;) {gt$x&lt;-center}
                    }
                    if (is.character(gt$y)) {
                        if(gt$y==&quot;peak&quot;) {gt$y&lt;-y[peak]}
                    }
                }
                if(is.list(gt$x)) {
                    gt$x&lt;-A$getgenpos(gt$x[[1]],gt$x[[2]])
                }
                do.call(&quot;panel.text&quot;, gt)
            }
        },
        xlab=xlab, ylab=ylab, 
        panel.extra=panel.extra, getgenpos=getGenPos, ...
    );
}

transposases &lt;- list(&quot;HVO_RS00735&quot;, &quot;HVO_RS00740&quot;, &quot;HVO_RS00765&quot;, &quot;HVO_RS00775&quot;, &quot;HVO_RS01150&quot;, 
                     &quot;HVO_RS01185&quot;, &quot;HVO_RS01190&quot;, &quot;HVO_RS01220&quot;, &quot;HVO_RS01895&quot;, &quot;HVO_RS01900&quot;, 
                     &quot;HVO_RS01930&quot;, &quot;HVO_RS01935&quot;, &quot;HVO_RS01940&quot;, &quot;HVO_RS02130&quot;, &quot;HVO_RS02155&quot;, 
                     &quot;HVO_RS02175&quot;, &quot;HVO_RS02220&quot;, &quot;HVO_RS02235&quot;, &quot;HVO_RS02240&quot;, &quot;HVO_RS02380&quot;, 
                     &quot;HVO_RS02395&quot;, &quot;HVO_RS02400&quot;, &quot;HVO_RS02430&quot;, &quot;HVO_RS02445&quot;, &quot;HVO_RS02500&quot;, 
                     &quot;HVO_RS02535&quot;, &quot;HVO_RS02910&quot;, &quot;HVO_RS02920&quot;, &quot;HVO_RS02960&quot;, &quot;HVO_RS02990&quot;, 
                     &quot;HVO_RS02995&quot;, &quot;HVO_RS03005&quot;, &quot;HVO_RS03015&quot;, &quot;HVO_RS03080&quot;, &quot;HVO_RS03235&quot;, 
                     &quot;HVO_RS03435&quot;, &quot;HVO_RS03440&quot;, &quot;HVO_RS03480&quot;, &quot;HVO_RS03490&quot;, &quot;HVO_RS03670&quot;, 
                     &quot;HVO_RS03720&quot;, &quot;HVO_RS03730&quot;, &quot;HVO_RS03765&quot;, &quot;HVO_RS03770&quot;, &quot;HVO_RS03825&quot;, 
                     &quot;HVO_RS03835&quot;, &quot;HVO_RS03870&quot;, &quot;HVO_RS04000&quot;, &quot;HVO_RS05975&quot;, &quot;HVO_RS06035&quot;, 
                     &quot;HVO_RS06040&quot;, &quot;HVO_RS06050&quot;, &quot;HVO_RS07250&quot;, &quot;HVO_RS07375&quot;, &quot;HVO_RS08840&quot;, 
                     &quot;HVO_RS09115&quot;, &quot;HVO_RS09120&quot;, &quot;HVO_RS09130&quot;, &quot;HVO_RS10010&quot;, &quot;HVO_RS10230&quot;, 
                     &quot;HVO_RS10235&quot;, &quot;HVO_RS12020&quot;, &quot;HVO_RS12025&quot;, &quot;HVO_RS12030&quot;, &quot;HVO_RS13270&quot;, 
                     &quot;HVO_RS13510&quot;, &quot;HVO_RS13560&quot;, &quot;HVO_RS14565&quot;, &quot;HVO_RS14570&quot;, &quot;HVO_RS14575&quot;, 
                     &quot;HVO_RS14680&quot;, &quot;HVO_RS15525&quot;, &quot;HVO_RS15925&quot;, &quot;HVO_RS16425&quot;, &quot;HVO_RS16590&quot;, 
                     &quot;HVO_RS18325&quot;, &quot;HVO_RS18335&quot;, &quot;HVO_RS18340&quot;, &quot;HVO_RS18385&quot;, &quot;HVO_RS19305&quot;, 
                     &quot;HVO_RS19350&quot;, &quot;HVO_RS19370&quot;, &quot;HVO_RS19430&quot;, &quot;HVO_RS19435&quot;, &quot;HVO_RS19485&quot;, 
                     &quot;HVO_RS19490&quot;, &quot;HVO_RS19515&quot;, &quot;HVO_RS19535&quot;, &quot;HVO_RS19540&quot;, &quot;HVO_RS19550&quot;, 
                     &quot;HVO_RS19560&quot;, &quot;HVO_RS19565&quot;, &quot;HVO_RS19605&quot;, &quot;HVO_RS19775&quot;, &quot;HVO_RS19810&quot;, 
                     &quot;HVO_RS19915&quot;, &quot;HVO_RS19920&quot;, &quot;HVO_RS19935&quot;, &quot;HVO_RS19970&quot;, &quot;HVO_RS20195&quot;, 
                     &quot;HVO_RS20245&quot;, &quot;HVO_RS20355&quot;, &quot;HVO_RS20415&quot;)

ann&lt;-rep(1, length(man_in$SNP))
ann[with(man_in, timepoint==&quot;de00&quot; &amp; SNP %in% transposases)]&lt;-2
ann[with(man_in, timepoint==&quot;de02&quot; &amp; SNP %in% transposases)]&lt;-3
ann[with(man_in, timepoint==&quot;de04&quot; &amp; SNP %in% transposases)]&lt;-4
ann[with(man_in, timepoint==&quot;de08&quot; &amp; SNP %in% transposases)]&lt;-5
ann[with(man_in, timepoint==&quot;de24&quot; &amp; SNP %in% transposases)]&lt;-6
ann&lt;-factor(ann, levels=1:6)

manplot &lt;- manhattan.plot(man_in$CHR2, man_in$BP, man_in$P, xlab = &quot;&quot;,
              annotate = list(ann, &quot;2&quot; = list(label = &quot; &quot;, col = &quot;#000000&quot;),
                  &quot;3&quot; = list(label = &quot; &quot;, col = &quot;#E69F00&quot;),
                  &quot;4&quot; = list(label = &quot; &quot;, col = &quot;#009E73&quot;),
                  &quot;5&quot; = list(label = &quot; &quot;, col = &quot;#0072B2&quot;),
                  &quot;6&quot; = list(label = &quot; &quot;, col = &quot;#D55E00&quot;)),
              key = list(border=T, padding.text=2, 
                corner=c(.95, .95), text=list(lab=c(&quot;t00&quot;,&quot;t02&quot;,&quot;t04&quot;,&quot;t08&quot;,&quot;t24&quot;)), 
                points=list(col=c(&quot;#000000&quot;, &quot;#E69F00&quot;, &quot;#009E73&quot;, &quot;#0072B2&quot;, &quot;#D55E00&quot;), pch=20)))
manplot</code></pre>
<p><img src="hvo_mating_files/figure-html/manhattan-1.png" width="672" /></p>
<pre class="r"><code># pdf(&quot;plots/manplot_transposase.pdf&quot;, height = 4, width = 6)
# print(manplot)
# dev.off

### repeat for crispr/cas genes
crispr &lt;- list(&quot;HVO_RS02755&quot;, &quot;HVO_RS02760&quot;, &quot;HVO_RS02765&quot;, &quot;HVO_RS02770&quot;, &quot;HVO_RS02780&quot;, &quot;HVO_RS02785&quot;, &quot;HVO_RS02790&quot;)

annc&lt;-rep(1, length(man_in$SNP))
annc[with(man_in, timepoint==&quot;de00&quot; &amp; SNP %in% crispr)]&lt;-2
annc[with(man_in, timepoint==&quot;de02&quot; &amp; SNP %in% crispr)]&lt;-3
annc[with(man_in, timepoint==&quot;de04&quot; &amp; SNP %in% crispr)]&lt;-4
annc[with(man_in, timepoint==&quot;de08&quot; &amp; SNP %in% crispr)]&lt;-5
annc[with(man_in, timepoint==&quot;de24&quot; &amp; SNP %in% crispr)]&lt;-6
annc&lt;-factor(annc, levels=1:6)

manplotc &lt;- manhattan.plot(man_in$CHR2, man_in$BP, man_in$P, xlab = &quot;&quot;,
              annotate = list(annc, &quot;2&quot; = list(label = &quot; &quot;, col = &quot;#000000&quot;),
                              &quot;3&quot; = list(label = &quot; &quot;, col = &quot;#E69F00&quot;),
                              &quot;4&quot; = list(label = &quot; &quot;, col = &quot;#009E73&quot;),
                              &quot;5&quot; = list(label = &quot; &quot;, col = &quot;#0072B2&quot;),
                              &quot;6&quot; = list(label = &quot; &quot;, col = &quot;#D55E00&quot;)),
              key = list(border=T, padding.text=2, 
                corner=c(.95, .95), text=list(lab=c(&quot;t00&quot;,&quot;t02&quot;,&quot;t04&quot;,&quot;t08&quot;,&quot;t24&quot;)), 
                points=list(col=c(&quot;#000000&quot;, &quot;#E69F00&quot;, &quot;#009E73&quot;, &quot;#0072B2&quot;, &quot;#D55E00&quot;), pch=20)))
manplotc</code></pre>
<p><img src="hvo_mating_files/figure-html/manhattan-2.png" width="672" /></p>
<pre class="r"><code># pdf(&quot;plots/manplot_crispr.pdf&quot;, height = 4, width = 6)
# print(manplotc)
# dev.off
# c(&quot;#000000&quot;, &quot;#E69F00&quot;, &quot;#009E73&quot;, &quot;#0072B2&quot;, &quot;#D55E00&quot;)</code></pre>
<pre class="r"><code>## each plot made in individual chunks
manplot</code></pre>
<p><img src="hvo_mating_files/figure-html/cowplot-1.png" width="672" /></p>
<pre class="r"><code>plot_rms</code></pre>
<pre><code>## Warning: Removed 3 rows containing missing values (geom_bar).</code></pre>
<pre><code>## Warning: Removed 30 rows containing missing values (geom_text).</code></pre>
<pre><code>## Warning: Removed 27 rows containing missing values (geom_text).</code></pre>
<p><img src="hvo_mating_files/figure-html/cowplot-2.png" width="672" /></p>
<pre class="r"><code>viol</code></pre>
<pre><code>## Warning: Transformation introduced infinite values in continuous y-axis</code></pre>
<pre><code>## Warning: Transformation introduced infinite values in continuous y-axis</code></pre>
<pre><code>## Warning: Removed 5 rows containing non-finite values (stat_ydensity).</code></pre>
<p><img src="hvo_mating_files/figure-html/cowplot-3.png" width="672" /></p>
<pre class="r"><code>plot_grid(manplot, viol, plot_rms, labels = c(&#39;A&#39;, &#39;B&#39;, &#39;C&#39;), label_size = 14, ncol = 1, nrow = 3)</code></pre>
<pre><code>## Warning: Transformation introduced infinite values in continuous y-axis</code></pre>
<pre><code>## Warning: Transformation introduced infinite values in continuous y-axis</code></pre>
<pre><code>## Warning: Removed 5 rows containing non-finite values (stat_ydensity).</code></pre>
<pre><code>## Warning: Removed 3 rows containing missing values (geom_bar).</code></pre>
<pre><code>## Warning: Removed 30 rows containing missing values (geom_text).</code></pre>
<pre><code>## Warning: Removed 27 rows containing missing values (geom_text).</code></pre>
<p><img src="hvo_mating_files/figure-html/cowplot-4.png" width="672" /></p>
<pre class="r"><code># ggsave(&quot;plots/figure6_selfish_genes.pdf&quot;, height = 15, width = 8, units = &quot;in&quot;)


manplotc</code></pre>
<p><img src="hvo_mating_files/figure-html/cowplot-5.png" width="672" /></p>
<pre class="r"><code>plot_crispr_cas</code></pre>
<pre><code>## Warning: Removed 35 rows containing missing values (geom_text).</code></pre>
<pre><code>## Warning: Removed 33 rows containing missing values (geom_text).</code></pre>
<p><img src="hvo_mating_files/figure-html/cowplot-6.png" width="672" /></p>
<pre class="r"><code>plot_grid(manplotc, plot_crispr_cas, labels = c(&#39;A&#39;, &#39;B&#39;), label_size = 14, ncol = 1, nrow = 2)</code></pre>
<pre><code>## Warning: Removed 35 rows containing missing values (geom_text).

## Warning: Removed 33 rows containing missing values (geom_text).</code></pre>
<p><img src="hvo_mating_files/figure-html/cowplot-7.png" width="672" /></p>
<pre class="r"><code># ggsave(&quot;plots/figureS4_crispr.pdf&quot;, height = 12, width = 10, units = &quot;in&quot;)

plot_glyc</code></pre>
<pre><code>## Warning: Removed 1 rows containing missing values (geom_bar).</code></pre>
<pre><code>## Warning: Removed 67 rows containing missing values (geom_text).</code></pre>
<pre><code>## Warning: Removed 47 rows containing missing values (geom_text).</code></pre>
<p><img src="hvo_mating_files/figure-html/cowplot-8.png" width="672" /></p>
<pre class="r"><code>plot_lsglyc</code></pre>
<pre><code>## Warning: Removed 76 rows containing missing values (geom_text).</code></pre>
<pre><code>## Warning: Removed 72 rows containing missing values (geom_text).</code></pre>
<p><img src="hvo_mating_files/figure-html/cowplot-9.png" width="672" /></p>
<pre class="r"><code>plot_grid(plot_glyc, plot_lsglyc, labels = c(&#39;A&#39;, &#39;B&#39;), label_size = 18, ncol = 1, nrow = 2)</code></pre>
<pre><code>## Warning: Removed 1 rows containing missing values (geom_bar).</code></pre>
<pre><code>## Warning: Removed 67 rows containing missing values (geom_text).</code></pre>
<pre><code>## Warning: Removed 47 rows containing missing values (geom_text).</code></pre>
<pre><code>## Warning: Removed 76 rows containing missing values (geom_text).</code></pre>
<pre><code>## Warning: Removed 72 rows containing missing values (geom_text).</code></pre>
<p><img src="hvo_mating_files/figure-html/cowplot-10.png" width="672" /></p>
<pre class="r"><code># ggsave(&quot;plots/figure2_glycosylation.pdf&quot;, height = 18, width = 14, units = &quot;in&quot;)</code></pre>
<pre class="r"><code>#head(logfc)
#for(i in c(&quot;de00&quot;,&quot;de02&quot;,&quot;de04&quot;,&quot;de08&quot;,&quot;de24&quot;)) {
top_n(p0, 20, de00) %&gt;% arrange(-de00) %&gt;%
  write.table(file = &quot;tables/de00_top20.txt&quot;, quote = FALSE, sep = &quot;\t&quot;, row.names = FALSE) 
top_n(p0, 20, de02) %&gt;% arrange(-de02) %&gt;%
  write.table(file = &quot;tables/de02_top20.txt&quot;, quote = FALSE, sep = &quot;\t&quot;, row.names = FALSE)
top_n(p0, 20, de04) %&gt;% arrange(-de04) %&gt;%
  write.table(file = &quot;tables/de04_top20.txt&quot;, quote = FALSE, sep = &quot;\t&quot;, row.names = FALSE)
top_n(p0, 20, de08) %&gt;% arrange(-de08) %&gt;%
  write.table(file = &quot;tables/de08_top20.txt&quot;, quote = FALSE, sep = &quot;\t&quot;, row.names = FALSE)
top_n(p0, 20, de24) %&gt;% arrange(-de24) %&gt;%
  write.table(file = &quot;tables/de24_top20.txt&quot;, quote = FALSE, sep = &quot;\t&quot;, row.names = FALSE)
#max(logfc$de00, na.rm = TRUE)

top_n(p0, -20, de00) %&gt;% arrange(-de00) %&gt;%
  write.table(file = &quot;tables/de00_bottom20.txt&quot;, quote = FALSE, sep = &quot;\t&quot;, row.names = FALSE) 
top_n(p0, -20, de02) %&gt;% arrange(-de02) %&gt;%
  write.table(file = &quot;tables/de02_bottom20.txt&quot;, quote = FALSE, sep = &quot;\t&quot;, row.names = FALSE)
top_n(p0, -20, de04) %&gt;% arrange(-de04) %&gt;%
  write.table(file = &quot;tables/de04_bottom20.txt&quot;, quote = FALSE, sep = &quot;\t&quot;, row.names = FALSE)
top_n(p0, -20, de08) %&gt;% arrange(-de08) %&gt;%
  write.table(file = &quot;tables/de08_bottom20.txt&quot;, quote = FALSE, sep = &quot;\t&quot;, row.names = FALSE)
top_n(p0, -20, de24) %&gt;% arrange(-de24) %&gt;%
  write.table(file = &quot;tables/de24_bottom20.txt&quot;, quote = FALSE, sep = &quot;\t&quot;, row.names = FALSE)

#top 20
# lists &lt;- list(
#   `de00 top 20`=c(&quot;HVO_2375&quot;, &quot;HVO_RS19695&quot;, &quot;HVO_RS19895&quot;, &quot;HVO_B0212&quot;, &quot;HVO_0280A&quot;, &quot;HVO_A0184&quot;, &quot;HVO_2672&quot;, &quot;HVO_0925&quot;, &quot;HVO_A0319&quot;, &quot;HVO_RS04575&quot;, &quot;HVO_C0086&quot;, &quot;HVO_A0399&quot;, &quot;HVO_0365&quot;, &quot;HVO_RS11620&quot;, &quot;HVO_RS19980&quot;, &quot;HVO_1644&quot;, &quot;HVO_0956&quot;, &quot;HVO_0267&quot;, &quot;HVO_0852&quot;, &quot;HVO_RS03310&quot;),
#   `de02 top 20`=c(&quot;HVO_2550&quot;, &quot;HVO_2545&quot;, &quot;HVO_0925&quot;, &quot;HVO_RS19695&quot;, &quot;HVO_A0004&quot;, &quot;HVO_B0212&quot;, &quot;HVO_2544&quot;, &quot;HVO_2553&quot;, &quot;HVO_2562&quot;, &quot;HVO_C0086&quot;, &quot;HVO_0280A&quot;, &quot;HVO_0852&quot;, &quot;HVO_2649&quot;, &quot;HVO_A0399&quot;, &quot;HVO_2546&quot;, &quot;HVO_2543&quot;, &quot;HVO_2549&quot;, &quot;HVO_1981&quot;, &quot;HVO_2547&quot;, &quot;HVO_2551&quot;),
#   `de04 top 20`=c(&quot;HVO_0925&quot;, &quot;HVO_RS19695&quot;, &quot;HVO_A0004&quot;, &quot;HVO_A0399&quot;, &quot;HVO_B0212&quot;, &quot;HVO_2649&quot;, &quot;HVO_1644&quot;, &quot;HVO_A0272&quot;, &quot;HVO_RS19895&quot;, &quot;HVO_0280A&quot;, &quot;HVO_1705&quot;, &quot;HVO_0852&quot;, &quot;HVO_B0129&quot;, &quot;HVO_0133&quot;, &quot;HVO_A0059&quot;, &quot;HVO_0257&quot;, &quot;HVO_2550&quot;, &quot;HVO_C0086&quot;, &quot;HVO_A0098&quot;, &quot;HVO_2545&quot;),
#   `de08 top 20`=c(&quot;HVO_B0212&quot;, &quot;HVO_RS19895&quot;, &quot;HVO_1705&quot;, &quot;HVO_A0399&quot;, &quot;HVO_B0198&quot;, &quot;HVO_B0047&quot;, &quot;HVO_B0041&quot;, &quot;HVO_0383&quot;, &quot;HVO_1644&quot;, &quot;HVO_A0052&quot;, &quot;HVO_2550&quot;, &quot;HVO_0280A&quot;, &quot;HVO_A0272&quot;, &quot;HVO_A0059&quot;, &quot;HVO_B0129&quot;, &quot;HVO_A0319&quot;, &quot;HVO_2980&quot;, &quot;HVO_B0224&quot;, &quot;HVO_A0268&quot;, &quot;HVO_0925&quot;),
#   `de24 top 20`=c(&quot;HVO_RS19695&quot;, &quot;HVO_0665&quot;, &quot;HVO_A0399&quot;, &quot;HVO_A0611&quot;, &quot;HVO_B0041&quot;, &quot;HVO_B0212&quot;, &quot;HVO_A0319&quot;, &quot;HVO_RS19895&quot;, &quot;HVO_A0029&quot;, &quot;HVO_1705&quot;, &quot;HVO_0925&quot;, &quot;HVO_B0047&quot;, &quot;HVO_B0198&quot;, &quot;HVO_B0046&quot;, &quot;HVO_2038&quot;, &quot;HVO_B0042&quot;, &quot;HVO_B0129&quot;, &quot;HVO_2980&quot;, &quot;HVO_RS04575&quot;, &quot;HVO_0257&quot;)
# )

#bottom 20
# lists &lt;- list(
#   `de00 bottom 20`=c(&quot;HVO_2733&quot;, &quot;HVO_0493&quot;, &quot;HVO_1889&quot;, &quot;HVO_A0497&quot;, &quot;HVO_C0078&quot;, &quot;HVO_0177&quot;, &quot;HVO_0200&quot;, &quot;HVO_2718&quot;, &quot;HVO_0947&quot;, &quot;HVO_A0359&quot;, &quot;HVO_0368&quot;, &quot;HVO_A0434&quot;, &quot;HVO_1525&quot;, &quot;HVO_2741&quot;, &quot;HVO_0378&quot;, &quot;HVO_1805&quot;, &quot;HVO_RS11000&quot;, &quot;HVO_0171&quot;, &quot;HVO_0373&quot;, &quot;HVO_A0408A&quot;),
#   `de02 bottom 20`=c(&quot;HVO_0200&quot;, &quot;HVO_1757&quot;, &quot;HVO_1671&quot;, &quot;HVO_0855&quot;, &quot;HVO_2179&quot;, &quot;HVO_2243&quot;, &quot;HVO_0947&quot;, &quot;HVO_RS11000&quot;, &quot;HVO_0873&quot;, &quot;HVO_RS20250&quot;, &quot;HVO_1795&quot;, &quot;HVO_0515&quot;, &quot;HVO_2238&quot;, &quot;HVO_2569&quot;, &quot;HVO_0171&quot;, &quot;HVO_1801&quot;, &quot;HVO_2184&quot;, &quot;HVO_A0454&quot;, &quot;HVO_2987&quot;, &quot;HVO_A0408A&quot;),
#   `de04 bottom 20`=c(&quot;HVO_0096&quot;, &quot;HVO_2241&quot;, &quot;HVO_1537&quot;, &quot;HVO_0378&quot;, &quot;HVO_2179&quot;, &quot;HVO_0200&quot;, &quot;HVO_0515&quot;, &quot;HVO_2243&quot;, &quot;HVO_2372&quot;, &quot;HVO_0932&quot;, &quot;HVO_RS11000&quot;, &quot;HVO_2973&quot;, &quot;HVO_0373&quot;, &quot;HVO_1889&quot;, &quot;HVO_0947&quot;, &quot;HVO_RS19670&quot;, &quot;HVO_0931&quot;, &quot;HVO_A0592&quot;, &quot;HVO_0171&quot;, &quot;HVO_2121&quot;),
#   `de08 bottom 20`=c(&quot;HVO_0763&quot;, &quot;HVO_0249&quot;, &quot;HVO_0489&quot;, &quot;HVO_1489&quot;, &quot;HVO_2037&quot;, &quot;HVO_0488&quot;, &quot;HVO_A0142&quot;, &quot;HVO_A0548&quot;, &quot;HVO_1643&quot;, &quot;HVO_1525&quot;, &quot;HVO_RS20250&quot;, &quot;HVO_1795&quot;, &quot;HVO_B0362&quot;, &quot;HVO_2570&quot;, &quot;HVO_0947&quot;, &quot;HVO_2920&quot;, &quot;HVO_0171&quot;, &quot;HVO_2238&quot;, &quot;HVO_2184&quot;, &quot;HVO_A0408A&quot;),
#   `de24 bottom 20`=c(&quot;HVO_A0548&quot;, &quot;HVO_0355&quot;, &quot;HVO_0772&quot;, &quot;HVO_2973&quot;, &quot;HVO_1753&quot;, &quot;HVO_RS20250&quot;, &quot;HVO_1751&quot;, &quot;HVO_2282&quot;, &quot;HVO_0502&quot;, &quot;HVO_1108&quot;, &quot;HVO_1924&quot;, &quot;HVO_2178&quot;, &quot;HVO_1278&quot;, &quot;HVO_2920&quot;, &quot;HVO_RS20340&quot;, &quot;HVO_2569&quot;, &quot;HVO_0947&quot;, &quot;HVO_RS14225&quot;, &quot;HVO_A0408A&quot;, &quot;HVO_1325&quot;)
#   )

#top/bottom 10
lists &lt;- list(
  `t00 top 10`=c(&quot;HVO_2375&quot;, &quot;HVO_RS19695&quot;, &quot;HVO_RS19895&quot;, &quot;HVO_B0212&quot;, &quot;HVO_0280A&quot;, &quot;HVO_A0184&quot;, &quot;HVO_2672&quot;, &quot;HVO_0925&quot;, &quot;HVO_A0319&quot;, &quot;HVO_RS04575&quot;),
  `t02 top 10`=c(&quot;HVO_2550&quot;, &quot;HVO_2545&quot;, &quot;HVO_0925&quot;, &quot;HVO_RS19695&quot;, &quot;HVO_A0004&quot;, &quot;HVO_B0212&quot;, &quot;HVO_2544&quot;, &quot;HVO_2553&quot;, &quot;HVO_2562&quot;, &quot;HVO_C0086&quot;),
  `t04 top 10`=c(&quot;HVO_0925&quot;, &quot;HVO_RS19695&quot;, &quot;HVO_A0004&quot;, &quot;HVO_A0399&quot;, &quot;HVO_B0212&quot;, &quot;HVO_2649&quot;, &quot;HVO_1644&quot;, &quot;HVO_A0272&quot;, &quot;HVO_RS19895&quot;, &quot;HVO_0280A&quot;),
  `t08 top 10`=c(&quot;HVO_B0212&quot;, &quot;HVO_RS19895&quot;, &quot;HVO_1705&quot;, &quot;HVO_A0399&quot;, &quot;HVO_B0198&quot;, &quot;HVO_B0047&quot;, &quot;HVO_B0041&quot;, &quot;HVO_0383&quot;, &quot;HVO_1644&quot;, &quot;HVO_A0052&quot;),
  `t24 top 10`=c(&quot;HVO_RS19695&quot;, &quot;HVO_0665&quot;, &quot;HVO_A0399&quot;, &quot;HVO_A0611&quot;, &quot;HVO_B0041&quot;, &quot;HVO_B0212&quot;, &quot;HVO_A0319&quot;, &quot;HVO_RS19895&quot;, &quot;HVO_A0029&quot;, &quot;HVO_1705&quot;),
  `t00 bottom 10`=c(&quot;HVO_2733&quot;, &quot;HVO_0493&quot;, &quot;HVO_1889&quot;, &quot;HVO_A0497&quot;, &quot;HVO_C0078&quot;, &quot;HVO_0177&quot;, &quot;HVO_0200&quot;, &quot;HVO_2718&quot;, &quot;HVO_0947&quot;, &quot;HVO_A0359&quot;),
  `t02 bottom 10`=c(&quot;HVO_0200&quot;, &quot;HVO_1757&quot;, &quot;HVO_1671&quot;, &quot;HVO_0855&quot;, &quot;HVO_2179&quot;, &quot;HVO_2243&quot;, &quot;HVO_0947&quot;, &quot;HVO_RS11000&quot;, &quot;HVO_0873&quot;, &quot;HVO_RS20250&quot;),
  `t04 bottom 10`=c(&quot;HVO_0096&quot;, &quot;HVO_2241&quot;, &quot;HVO_1537&quot;, &quot;HVO_0378&quot;, &quot;HVO_2179&quot;, &quot;HVO_0200&quot;, &quot;HVO_0515&quot;, &quot;HVO_2243&quot;, &quot;HVO_2372&quot;, &quot;HVO_0932&quot;),
  `t08 bottom 10`=c(&quot;HVO_0763&quot;, &quot;HVO_0249&quot;, &quot;HVO_0489&quot;, &quot;HVO_1489&quot;, &quot;HVO_2037&quot;, &quot;HVO_0488&quot;, &quot;HVO_A0142&quot;, &quot;HVO_A0548&quot;, &quot;HVO_1643&quot;, &quot;HVO_1525&quot;),
  `t24 bottom 10`=c(&quot;HVO_A0548&quot;, &quot;HVO_0355&quot;, &quot;HVO_0772&quot;, &quot;HVO_2973&quot;, &quot;HVO_1753&quot;, &quot;HVO_RS20250&quot;, &quot;HVO_1751&quot;, &quot;HVO_2282&quot;, &quot;HVO_0502&quot;, &quot;HVO_1108&quot;)
  )

# p &lt;- htmltools::tagList()
for(i in 1:length(lists)){
  nm &lt;- (names(lists)[i])
  print(nm)
  df_sub &lt;- filter(p0, grepl(paste((lists[[i]]), collapse=&quot;|&quot;), old))
  coords_sub &lt;- filter(coords, grepl(paste((lists[[i]]), collapse=&quot;|&quot;), old))
  coords01_sub &lt;- filter(coords01, grepl(paste((lists[[i]]), collapse=&quot;|&quot;), old))
  df_sub$Row.names &lt;- NULL
  df_sub &lt;- melt(df_sub, id.vars=c(&quot;old&quot;, &quot;new&quot;, &quot;prot&quot;, &quot;gene_name&quot;, &quot;arcog&quot;, &quot;geneid&quot;, &quot;cog&quot;))
  df_sub1 &lt;- merge(df_sub, coords_sub, all.x = TRUE)
  df_sub1 &lt;- merge(df_sub1, coords01_sub, all.x = TRUE)
  p &lt;- ggplot(data=df_sub1, aes(x= paste(new, str_wrap(prot, width = 30), sep=&#39;\n&#39;), y=value, fill = variable)) +
    theme_bw() +
    geom_bar(stat=&quot;identity&quot;, position = position_dodge(width = NULL), width = 0.7) +
    geom_hline(yintercept = 0) +
    theme(panel.grid.major = element_blank(), panel.grid.minor = element_blank()) +
    labs(title = paste(nm,&quot;genes&quot;), 
         x = NULL, y = expression(&quot;log&quot;[&quot;2&quot;]*&quot; fold change&quot;), fill = &quot;time vs shaking&quot;) +
    theme(axis.text.x = element_text(size  = 16, angle = 45, hjust = 1, vjust = 1)) +
    theme(plot.title = element_text(hjust = 0.5), plot.margin = unit(c(0.5,0.5,0.5,1),&quot;cm&quot;)) +
    scale_fill_manual(labels = c(&quot;t00&quot;, &quot;t02&quot;, &quot;t04&quot;, &quot;t08&quot;, &quot;t24&quot;), 
                      values=c(&quot;#000000&quot;, &quot;#E69F00&quot;, &quot;#009E73&quot;, &quot;#0072B2&quot;, &quot;#D55E00&quot;)) +
    geom_text(aes(y = val), label = &quot;*&quot;, position = position_dodge(width = 0.7), size = 2) +
    geom_text(aes(y = val01), label = &quot;**&quot;, position = position_dodge(width = 0.7), size = 2) +
    theme(title = element_text(size = 16), axis.text = element_text(size = 16)) +
    # theme(legend.title = element_text(size = 16), legend.text = element_text(size = 16)) +
    # theme(legend.position = &quot;bottom&quot;)
    theme(legend.position = &quot;none&quot;)
  print(p)
  nm2 &lt;- gsub(&quot; &quot;, &quot;_&quot;, nm)
  assign(paste(&quot;p&quot;, nm2, sep=&quot;_&quot;),p)
  # ggsave(filename = paste(&quot;plots/arcog_&quot;,nm2,&quot;_histogram.pdf&quot;, sep = &quot;&quot;))
  # ggsave(filename = paste(&quot;plots/arcog_&quot;,nm2,&quot;_histogram.png&quot;, sep = &quot;&quot;))
}</code></pre>
<pre><code>## [1] &quot;t00 top 10&quot;</code></pre>
<pre><code>## Warning: Removed 6 rows containing missing values (geom_bar).</code></pre>
<pre><code>## Warning: Removed 46 rows containing missing values (geom_text).</code></pre>
<pre><code>## Warning: Removed 10 rows containing missing values (geom_text).</code></pre>
<p><img src="hvo_mating_files/figure-html/top%20and%20bottom%20de-1.png" width="672" /></p>
<pre><code>## [1] &quot;t02 top 10&quot;</code></pre>
<pre><code>## Warning: Removed 5 rows containing missing values (geom_bar).</code></pre>
<pre><code>## Warning: Removed 48 rows containing missing values (geom_text).</code></pre>
<pre><code>## Warning: Removed 12 rows containing missing values (geom_text).</code></pre>
<p><img src="hvo_mating_files/figure-html/top%20and%20bottom%20de-2.png" width="672" /></p>
<pre><code>## [1] &quot;t04 top 10&quot;</code></pre>
<pre><code>## Warning: Removed 7 rows containing missing values (geom_bar).</code></pre>
<pre><code>## Warning: Removed 46 rows containing missing values (geom_text).</code></pre>
<pre><code>## Warning: Removed 12 rows containing missing values (geom_text).</code></pre>
<p><img src="hvo_mating_files/figure-html/top%20and%20bottom%20de-3.png" width="672" /></p>
<pre><code>## [1] &quot;t08 top 10&quot;</code></pre>
<pre><code>## Warning: Removed 8 rows containing missing values (geom_bar).</code></pre>
<pre><code>## Warning: Removed 46 rows containing missing values (geom_text).</code></pre>
<pre><code>## Warning: Removed 16 rows containing missing values (geom_text).</code></pre>
<p><img src="hvo_mating_files/figure-html/top%20and%20bottom%20de-4.png" width="672" /></p>
<pre><code>## [1] &quot;t24 top 10&quot;</code></pre>
<pre><code>## Warning: Removed 7 rows containing missing values (geom_bar).</code></pre>
<pre><code>## Warning: Removed 46 rows containing missing values (geom_text).</code></pre>
<pre><code>## Warning: Removed 19 rows containing missing values (geom_text).</code></pre>
<p><img src="hvo_mating_files/figure-html/top%20and%20bottom%20de-5.png" width="672" /></p>
<pre><code>## [1] &quot;t00 bottom 10&quot;</code></pre>
<pre><code>## Warning: Removed 38 rows containing missing values (geom_text).</code></pre>
<pre><code>## Warning: Removed 26 rows containing missing values (geom_text).</code></pre>
<p><img src="hvo_mating_files/figure-html/top%20and%20bottom%20de-6.png" width="672" /></p>
<pre><code>## [1] &quot;t02 bottom 10&quot;</code></pre>
<pre><code>## Warning: Removed 44 rows containing missing values (geom_text).</code></pre>
<pre><code>## Warning: Removed 15 rows containing missing values (geom_text).</code></pre>
<p><img src="hvo_mating_files/figure-html/top%20and%20bottom%20de-7.png" width="672" /></p>
<pre><code>## [1] &quot;t04 bottom 10&quot;</code></pre>
<pre><code>## Warning: Removed 43 rows containing missing values (geom_text).</code></pre>
<pre><code>## Warning: Removed 14 rows containing missing values (geom_text).</code></pre>
<p><img src="hvo_mating_files/figure-html/top%20and%20bottom%20de-8.png" width="672" /></p>
<pre><code>## [1] &quot;t08 bottom 10&quot;</code></pre>
<pre><code>## Warning: Removed 48 rows containing missing values (geom_text).</code></pre>
<pre><code>## Warning: Removed 22 rows containing missing values (geom_text).</code></pre>
<p><img src="hvo_mating_files/figure-html/top%20and%20bottom%20de-9.png" width="672" /></p>
<pre><code>## [1] &quot;t24 bottom 10&quot;</code></pre>
<pre><code>## Warning: Removed 49 rows containing missing values (geom_text).</code></pre>
<pre><code>## Warning: Removed 23 rows containing missing values (geom_text).</code></pre>
<p><img src="hvo_mating_files/figure-html/top%20and%20bottom%20de-10.png" width="672" /></p>
<pre class="r"><code>## make legend separately from the plots
# leg &lt;- get_legend(p)
# as_ggplot(leg)


combined_plot &lt;- plot_grid(p_t00_top_10, p_t00_bottom_10, 
                           p_t02_top_10, p_t02_bottom_10, 
                           p_t04_top_10, p_t04_bottom_10, 
                           p_t08_top_10, p_t08_bottom_10, 
                           p_t24_top_10, p_t24_bottom_10, 
          labels = c(&#39;A&#39;, &#39;F&#39;, &#39;B&#39;, &#39;G&#39;, &#39;C&#39;, &#39;H&#39;, &#39;D&#39;, &#39;I&#39;, &#39;E&#39;, &#39;J&#39;), label_size = 20, ncol = 2, nrow = 5)</code></pre>
<pre><code>## Warning: Removed 6 rows containing missing values (geom_bar).</code></pre>
<pre><code>## Warning: Removed 46 rows containing missing values (geom_text).</code></pre>
<pre><code>## Warning: Removed 10 rows containing missing values (geom_text).</code></pre>
<pre><code>## Warning: Removed 38 rows containing missing values (geom_text).</code></pre>
<pre><code>## Warning: Removed 26 rows containing missing values (geom_text).</code></pre>
<pre><code>## Warning: Removed 5 rows containing missing values (geom_bar).</code></pre>
<pre><code>## Warning: Removed 48 rows containing missing values (geom_text).</code></pre>
<pre><code>## Warning: Removed 12 rows containing missing values (geom_text).</code></pre>
<pre><code>## Warning: Removed 44 rows containing missing values (geom_text).</code></pre>
<pre><code>## Warning: Removed 15 rows containing missing values (geom_text).</code></pre>
<pre><code>## Warning: Removed 7 rows containing missing values (geom_bar).</code></pre>
<pre><code>## Warning: Removed 46 rows containing missing values (geom_text).</code></pre>
<pre><code>## Warning: Removed 12 rows containing missing values (geom_text).</code></pre>
<pre><code>## Warning: Removed 43 rows containing missing values (geom_text).</code></pre>
<pre><code>## Warning: Removed 14 rows containing missing values (geom_text).</code></pre>
<pre><code>## Warning: Removed 8 rows containing missing values (geom_bar).</code></pre>
<pre><code>## Warning: Removed 46 rows containing missing values (geom_text).</code></pre>
<pre><code>## Warning: Removed 16 rows containing missing values (geom_text).</code></pre>
<pre><code>## Warning: Removed 48 rows containing missing values (geom_text).</code></pre>
<pre><code>## Warning: Removed 22 rows containing missing values (geom_text).</code></pre>
<pre><code>## Warning: Removed 7 rows containing missing values (geom_bar).</code></pre>
<pre><code>## Warning: Removed 46 rows containing missing values (geom_text).</code></pre>
<pre><code>## Warning: Removed 19 rows containing missing values (geom_text).</code></pre>
<pre><code>## Warning: Removed 49 rows containing missing values (geom_text).</code></pre>
<pre><code>## Warning: Removed 23 rows containing missing values (geom_text).</code></pre>
<pre class="r"><code>plot_grid(combined_plot, leg, ncol = 1, rel_heights = c(1, .02))</code></pre>
<p><img src="hvo_mating_files/figure-html/top%20and%20bottom%20de-11.png" width="672" /></p>
<pre class="r"><code>ggsave(&quot;plots/figureS6_top_bottom_10.pdf&quot;, height = 36, width = 24, units = &quot;in&quot;)</code></pre>
<pre class="r"><code>## clustering method suggested by Pariksheet
# https://hbctraining.github.io/DGE_workshop/lessons/08_DGE_LRT.html

load(&quot;noiseq/myfactors_mate.RData&quot;)

# May need to round count_data again... 
count_data_raw &lt;- read.table(&quot;data/mating.mat&quot;, header=T, com=&#39;&#39;, check.names=F)
count_data &lt;- count_data_raw[rowSums(count_data_raw) &gt; 10,]
count_data &lt;- round(count_data, 0)
dds &lt;- DESeqDataSetFromMatrix(countData = count_data, colData = myfactors, design = ~ time_de)</code></pre>
<pre><code>## converting counts to integer mode</code></pre>
<pre class="r"><code># Wald test
dds_wald &lt;- DESeq(dds, test=&quot;Wald&quot;)</code></pre>
<pre><code>## estimating size factors</code></pre>
<pre><code>## estimating dispersions</code></pre>
<pre><code>## gene-wise dispersion estimates</code></pre>
<pre><code>## mean-dispersion relationship</code></pre>
<pre><code>## final dispersion estimates</code></pre>
<pre><code>## fitting model and testing</code></pre>
<pre class="r"><code># Likelihood ratio test
dds_lrt &lt;- DESeq(dds, test=&quot;LRT&quot;, reduced = ~ 1)</code></pre>
<pre><code>## estimating size factors</code></pre>
<pre><code>## estimating dispersions</code></pre>
<pre><code>## gene-wise dispersion estimates</code></pre>
<pre><code>## mean-dispersion relationship</code></pre>
<pre><code>## final dispersion estimates</code></pre>
<pre><code>## fitting model and testing</code></pre>
<pre class="r"><code># Extract results
res_LRT &lt;- results(dds_lrt)

# Subset the LRT results to return genes with padj &lt; 0.05
sig_res_LRT &lt;- res_LRT %&gt;%
               data.frame() %&gt;%
               rownames_to_column(var=&quot;gene&quot;) %&gt;% 
               as_tibble() %&gt;% 
               filter(padj &lt; 0.05)
 
# Get sig gene lists
sigLRT_genes &lt;- sig_res_LRT %&gt;% 
                pull(gene)
                
length(sigLRT_genes)</code></pre>
<pre><code>## [1] 1240</code></pre>
<pre class="r"><code># Subset results for faster cluster finding (for classroom demo purposes)
clustering_sig_genes &lt;- sig_res_LRT %&gt;%
                  arrange(padj) %&gt;%
                  head(n=1000)

# Transform counts for data visualization
rld &lt;- rlog(dds, blind=TRUE)

# Plot PCA 
plotPCA(rld, intgroup=&quot;time&quot;)</code></pre>
<p><img src="hvo_mating_files/figure-html/clustering-1.png" width="1440" /></p>
<pre class="r"><code># Extract the rlog matrix from the object
rld_mat &lt;- assay(rld)

# Compute pairwise correlation values
rld_cor &lt;- cor(rld_mat)

# Plot heatmap
pheatmap(rld_cor)</code></pre>
<p><img src="hvo_mating_files/figure-html/clustering-2.png" width="1440" /></p>
<pre class="r"><code># Obtain rlog values for those significant genes
cluster_rlog &lt;- rld_mat[clustering_sig_genes$gene, ]

# Use the `degPatterns` function from the &#39;DEGreport&#39; package to show gene clusters across sample groups
clusters &lt;- degPatterns(cluster_rlog, metadata = myfactors, time = &quot;time_de&quot;, col=NULL, minc = 1)</code></pre>
<pre><code>## Working with 1000 genes.</code></pre>
<pre><code>## Working with 999 genes after filtering: minc &gt; 1</code></pre>
<pre><code>## Joining, by = &quot;merge&quot;Joining, by = &quot;merge&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/clustering-3.png" width="1440" /></p>
<pre class="r"><code>## cluster all genes instead of the top 1000
cluster_rlog_all &lt;- rld_mat[sig_res_LRT$gene, ]
cluster_all &lt;- degPatterns(cluster_rlog_all, metadata = myfactors, time = &quot;time_de&quot;, col=NULL, minc = 1)</code></pre>
<pre><code>## Working with 1240 genes.
## Working with 1240 genes after filtering: minc &gt; 1
## Joining, by = &quot;merge&quot;Joining, by = &quot;merge&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/clustering-4.png" width="1440" /></p>
<pre class="r"><code>#### ggplot manually
ggplot(cluster_all$plot$data) +
  geom_line(aes(x = time_de, y = value, group = genes), color = &quot;grey&quot;) +
  geom_boxplot(aes(x = time_de, y = value, alpha = 0.2)) +
  geom_point(position = &quot;jitter&quot;, aes(x = time_de, y = value, alpha = 0.2)) +
  theme_bw() +
  theme(panel.grid.major = element_blank(), panel.grid.minor = element_blank()) +
  theme(legend.position = &quot;none&quot;) +
  ylab(&quot;scaled expression&quot;) + xlab(NULL) +
  theme(text = element_text(size=24), axis.text.x = element_text(size  = 24, angle = 90, hjust = 1, vjust = 1)) +
  theme(plot.title = element_text(size = 24)) +
  facet_wrap(~cluster, ncol = 4)</code></pre>
<p><img src="hvo_mating_files/figure-html/clustering-5.png" width="1440" /></p>
<pre class="r"><code>ggsave(&quot;plots/figS7_clusters.pdf&quot;, height = 20, width = 14)
# ggsave(&quot;exp_mating/plots/supp_fig_clusters.tiff&quot;)
# What type of data structure is the `clusters` output?
class(clusters)</code></pre>
<pre><code>## [1] &quot;list&quot;</code></pre>
<pre class="r"><code># Let&#39;s see what is stored in the `df` component
head(clusters$df)</code></pre>
<pre><code>##                                                                           genes cluster
## lcl.NC_013967.1_cds_WP_004042603.1_3528 lcl.NC_013967.1_cds_WP_004042603.1_3528       1
## lcl.NC_013967.1_cds_WP_004042968.1_3704 lcl.NC_013967.1_cds_WP_004042968.1_3704       1
## lcl.NC_013967.1_cds_WP_004042130.1_3216 lcl.NC_013967.1_cds_WP_004042130.1_3216       2
## lcl.NC_013967.1_cds_WP_004044440.1_1543 lcl.NC_013967.1_cds_WP_004044440.1_1543       3
## lcl.NC_013967.1_cds_WP_004041922.1_3016 lcl.NC_013967.1_cds_WP_004041922.1_3016       3
## lcl.NC_013967.1_cds_WP_004042964.1_3702 lcl.NC_013967.1_cds_WP_004042964.1_3702       1</code></pre>
<pre class="r"><code># Extract the Group 3 genes
cluster_groups &lt;- cluster_all$df
group3 &lt;- cluster_all$df %&gt;%
          filter(cluster == 3)

group12 &lt;- cluster_all$df %&gt;%
          filter(cluster == 12)</code></pre>
<pre class="r"><code>#library(annotables)

group3 &lt;- data.frame(lapply(group3, function(x) {
  gsub(&quot;lcl.&quot;, &quot;lcl|&quot;, x)
}))
sub_counts &lt;- subset(annot_counts, ncbi_id %in% group3$genes)

# group12 &lt;- data.frame(lapply(group12, function(x) {
#   gsub(&quot;lcl.&quot;, &quot;lcl|&quot;, x)
# }))
# sub_counts &lt;- subset(annot_counts, ncbi_id %in% group12$genes)

hvo_counts &lt;- sub_counts[, c(4,45:56)]
rownames(hvo_counts) &lt;- NULL
hvo_counts &lt;- distinct(hvo_counts)
rownames(hvo_counts) &lt;- hvo_counts$locus_tag
  hvo_counts$locus_tag &lt;- NULL
  hvo_counts &lt;- data.matrix(hvo_counts)

print(sub_counts$old_locus_tag)</code></pre>
<pre><code>## NULL</code></pre>
<pre class="r"><code>print(rownames(hvo_counts))</code></pre>
<pre><code>##   [1] &quot;HVO_RS00705&quot; &quot;HVO_RS00635&quot; &quot;HVO_RS00570&quot; &quot;HVO_RS00350&quot; &quot;HVO_RS00090&quot; &quot;HVO_RS01720&quot; &quot;HVO_RS01710&quot; &quot;HVO_RS01330&quot; &quot;HVO_RS00950&quot; &quot;HVO_RS00945&quot;
##  [11] &quot;HVO_RS00895&quot; &quot;HVO_RS00770&quot; &quot;HVO_RS00755&quot; &quot;HVO_RS19935&quot; &quot;HVO_RS03435&quot; &quot;HVO_RS04090&quot; &quot;HVO_RS04280&quot; &quot;HVO_RS04545&quot; &quot;HVO_RS04675&quot; &quot;HVO_RS02480&quot;
##  [21] &quot;HVO_RS02940&quot; &quot;HVO_RS03305&quot; &quot;HVO_RS01920&quot; &quot;HVO_RS02130&quot; &quot;HVO_RS19745&quot; &quot;HVO_RS02045&quot; &quot;HVO_RS02040&quot; &quot;HVO_RS01975&quot; &quot;HVO_RS01970&quot; &quot;HVO_RS01960&quot;
##  [31] &quot;HVO_RS01955&quot; &quot;HVO_RS03500&quot; &quot;HVO_RS02535&quot; &quot;HVO_RS03835&quot; &quot;HVO_RS03085&quot; &quot;HVO_RS03550&quot; &quot;HVO_RS02965&quot; &quot;HVO_RS19945&quot; &quot;HVO_RS02380&quot; &quot;HVO_RS03005&quot;
##  [41] &quot;HVO_RS20080&quot; &quot;HVO_RS02165&quot; &quot;HVO_RS03490&quot; &quot;HVO_RS02455&quot; &quot;HVO_RS03695&quot; &quot;HVO_RS19720&quot; &quot;HVO_RS19805&quot; &quot;HVO_RS20145&quot; &quot;HVO_RS20160&quot; &quot;HVO_RS20165&quot;
##  [51] &quot;HVO_RS12285&quot; &quot;HVO_RS12345&quot; &quot;HVO_RS12475&quot; &quot;HVO_RS12560&quot; &quot;HVO_RS12625&quot; &quot;HVO_RS12645&quot; &quot;HVO_RS12715&quot; &quot;HVO_RS12795&quot; &quot;HVO_RS12855&quot; &quot;HVO_RS12955&quot;
##  [61] &quot;HVO_RS13045&quot; &quot;HVO_RS13100&quot; &quot;HVO_RS13115&quot; &quot;HVO_RS13545&quot; &quot;HVO_RS13795&quot; &quot;HVO_RS13805&quot; &quot;HVO_RS13810&quot; &quot;HVO_RS14080&quot; &quot;HVO_RS14085&quot; &quot;HVO_RS14205&quot;
##  [71] &quot;HVO_RS14460&quot; &quot;HVO_RS14600&quot; &quot;HVO_RS14635&quot; &quot;HVO_RS14640&quot; &quot;HVO_RS14935&quot; &quot;HVO_RS14985&quot; &quot;HVO_RS15030&quot; &quot;HVO_RS15300&quot; &quot;HVO_RS15530&quot; &quot;HVO_RS15550&quot;
##  [81] &quot;HVO_RS15585&quot; &quot;HVO_RS15625&quot; &quot;HVO_RS15645&quot; &quot;HVO_RS15650&quot; &quot;HVO_RS15690&quot; &quot;HVO_RS15710&quot; &quot;HVO_RS15770&quot; &quot;HVO_RS15840&quot; &quot;HVO_RS15990&quot; &quot;HVO_RS16010&quot;
##  [91] &quot;HVO_RS16125&quot; &quot;HVO_RS16415&quot; &quot;HVO_RS16505&quot; &quot;HVO_RS16635&quot; &quot;HVO_RS16710&quot; &quot;HVO_RS16735&quot; &quot;HVO_RS16740&quot; &quot;HVO_RS16790&quot; &quot;HVO_RS16820&quot; &quot;HVO_RS16925&quot;
## [101] &quot;HVO_RS16965&quot; &quot;HVO_RS16970&quot; &quot;HVO_RS16975&quot; &quot;HVO_RS16980&quot; &quot;HVO_RS16985&quot; &quot;HVO_RS16990&quot; &quot;HVO_RS17000&quot; &quot;HVO_RS17005&quot; &quot;HVO_RS17010&quot; &quot;HVO_RS17015&quot;
## [111] &quot;HVO_RS17020&quot; &quot;HVO_RS17025&quot; &quot;HVO_RS17030&quot; &quot;HVO_RS17040&quot; &quot;HVO_RS17045&quot; &quot;HVO_RS17050&quot; &quot;HVO_RS17055&quot; &quot;HVO_RS17060&quot; &quot;HVO_RS17065&quot; &quot;HVO_RS17070&quot;
## [121] &quot;HVO_RS17075&quot; &quot;HVO_RS17080&quot; &quot;HVO_RS17085&quot; &quot;HVO_RS17200&quot; &quot;HVO_RS17370&quot; &quot;HVO_RS17375&quot; &quot;HVO_RS17380&quot; &quot;HVO_RS17530&quot; &quot;HVO_RS20305&quot; &quot;HVO_RS17855&quot;
## [131] &quot;HVO_RS17925&quot; &quot;HVO_RS17930&quot; &quot;HVO_RS17935&quot; &quot;HVO_RS17940&quot; &quot;HVO_RS17950&quot; &quot;HVO_RS17980&quot; &quot;HVO_RS18000&quot; &quot;HVO_RS18005&quot; &quot;HVO_RS18015&quot; &quot;HVO_RS18020&quot;
## [141] &quot;HVO_RS18025&quot; &quot;HVO_RS18030&quot; &quot;HVO_RS18105&quot; &quot;HVO_RS18110&quot; &quot;HVO_RS18115&quot; &quot;HVO_RS18120&quot; &quot;HVO_RS18125&quot; &quot;HVO_RS18130&quot; &quot;HVO_RS18135&quot; &quot;HVO_RS18145&quot;
## [151] &quot;HVO_RS18150&quot; &quot;HVO_RS18155&quot; &quot;HVO_RS18160&quot; &quot;HVO_RS18225&quot; &quot;HVO_RS20310&quot; &quot;HVO_RS11955&quot; &quot;HVO_RS11720&quot; &quot;HVO_RS11705&quot; &quot;HVO_RS11700&quot; &quot;HVO_RS11455&quot;
## [161] &quot;HVO_RS11405&quot; &quot;HVO_RS11250&quot; &quot;HVO_RS11245&quot; &quot;HVO_RS11065&quot; &quot;HVO_RS10710&quot; &quot;HVO_RS10340&quot; &quot;HVO_RS10305&quot; &quot;HVO_RS10300&quot; &quot;HVO_RS10190&quot; &quot;HVO_RS10185&quot;
## [171] &quot;HVO_RS10170&quot; &quot;HVO_RS09940&quot; &quot;HVO_RS09845&quot; &quot;HVO_RS09565&quot; &quot;HVO_RS09430&quot; &quot;HVO_RS09415&quot; &quot;HVO_RS09410&quot; &quot;HVO_RS09405&quot; &quot;HVO_RS09390&quot; &quot;HVO_RS09360&quot;
## [181] &quot;HVO_RS09085&quot; &quot;HVO_RS09065&quot; &quot;HVO_RS09060&quot; &quot;HVO_RS08955&quot; &quot;HVO_RS08935&quot; &quot;HVO_RS08920&quot; &quot;HVO_RS08875&quot; &quot;HVO_RS08820&quot; &quot;HVO_RS08815&quot; &quot;HVO_RS08670&quot;
## [191] &quot;HVO_RS08570&quot; &quot;HVO_RS08515&quot; &quot;HVO_RS08450&quot; &quot;HVO_RS08200&quot; &quot;HVO_RS08070&quot; &quot;HVO_RS08065&quot; &quot;HVO_RS08060&quot; &quot;HVO_RS08055&quot; &quot;HVO_RS08030&quot; &quot;HVO_RS07955&quot;
## [201] &quot;HVO_RS07845&quot; &quot;HVO_RS07395&quot; &quot;HVO_RS07045&quot; &quot;HVO_RS07000&quot; &quot;HVO_RS06950&quot; &quot;HVO_RS06875&quot; &quot;HVO_RS06735&quot; &quot;HVO_RS06705&quot; &quot;HVO_RS06440&quot; &quot;HVO_RS06435&quot;
## [211] &quot;HVO_RS06420&quot; &quot;HVO_RS06410&quot; &quot;HVO_RS06405&quot; &quot;HVO_RS06325&quot; &quot;HVO_RS06245&quot; &quot;HVO_RS06235&quot; &quot;HVO_RS06230&quot; &quot;HVO_RS06225&quot; &quot;HVO_RS06220&quot; &quot;HVO_RS06215&quot;
## [221] &quot;HVO_RS06210&quot; &quot;HVO_RS06200&quot; &quot;HVO_RS18570&quot; &quot;HVO_RS18660&quot; &quot;HVO_RS18685&quot; &quot;HVO_RS18880&quot; &quot;HVO_RS18885&quot; &quot;HVO_RS18960&quot; &quot;HVO_RS19080&quot; &quot;HVO_RS19125&quot;
## [231] &quot;HVO_RS19255&quot; &quot;HVO_RS04975&quot; &quot;HVO_RS04980&quot; &quot;HVO_RS04985&quot; &quot;HVO_RS04990&quot; &quot;HVO_RS04995&quot; &quot;HVO_RS05250&quot; &quot;HVO_RS05255&quot; &quot;HVO_RS05260&quot; &quot;HVO_RS05375&quot;
## [241] &quot;HVO_RS05455&quot; &quot;HVO_RS05560&quot; &quot;HVO_RS05595&quot; &quot;HVO_RS05655&quot; &quot;HVO_RS05875&quot; &quot;HVO_RS09050&quot; &quot;HVO_RS05545&quot; &quot;HVO_RS11375&quot; &quot;HVO_RS15735&quot; &quot;HVO_RS12020&quot;
## [251] &quot;HVO_RS18675&quot; &quot;HVO_RS15640&quot; &quot;HVO_RS06205&quot; &quot;HVO_RS17035&quot; &quot;HVO_RS12050&quot; &quot;HVO_RS13495&quot; &quot;HVO_RS16995&quot; &quot;HVO_RS13270&quot; &quot;HVO_RS15720&quot; &quot;HVO_RS16220&quot;
## [261] &quot;HVO_RS15560&quot; &quot;HVO_RS18335&quot; &quot;HVO_RS12405&quot; &quot;HVO_RS06850&quot; &quot;HVO_RS15590&quot; &quot;HVO_RS14660&quot; &quot;HVO_RS15535&quot; &quot;HVO_RS10205&quot; &quot;HVO_RS15305&quot; &quot;HVO_RS14655&quot;
## [271] &quot;HVO_RS15685&quot; &quot;HVO_RS16920&quot; &quot;HVO_RS15695&quot; &quot;HVO_RS20090&quot; &quot;HVO_RS20170&quot; &quot;HVO_RS20345&quot; &quot;HVO_RS20355&quot; &quot;HVO_RS20405&quot; &quot;HVO_RS19590&quot; &quot;HVO_RS19475&quot;
## [281] &quot;HVO_RS19295&quot; &quot;HVO_RS19335&quot; &quot;HVO_RS19430&quot; &quot;HVO_RS19625&quot; &quot;HVO_RS20330&quot; &quot;HVO_RS20385&quot; &quot;HVO_RS20350&quot; &quot;HVO_RS19480&quot; &quot;HVO_RS19330&quot;</code></pre>
<pre class="r"><code>list_hvo &lt;- rownames(hvo_counts)

supp_table &lt;- sub_counts[, c(4,2,13,15,37,38)]
supp_table &lt;- distinct(supp_table)
write.table(supp_table, file = &quot;tables/supp_table_cluster3.txt&quot;, quote = FALSE, sep = &quot;\t&quot;, row.names = FALSE)

## make gene lengths object with HVOs
hvo_gene_lengths &lt;- data.frame(&quot;locus_tag&quot; = hvo_map$locus_tag,
                               &quot;length&quot; = hvo_map$length) 
hvo_gene_lengths &lt;- unique(hvo_gene_lengths)
sub_length &lt;- subset(hvo_gene_lengths, locus_tag %in% rownames(hvo_counts))

dds_group12 = NOISeq::readData(data = hvo_counts, factors = myfactors, length = sub_length)
dir.create(path = &quot;noiseq/group3&quot;)</code></pre>
<pre><code>## Warning in dir.create(path = &quot;noiseq/group3&quot;): &#39;noiseq/group3&#39; already exists</code></pre>
<pre class="r"><code>time &lt;- c(&quot;p0&quot;,&quot;00&quot;,&quot;02&quot;,&quot;04&quot;,&quot;08&quot;,&quot;24&quot;)
time_de = c(&quot;shaking&quot;, &quot;t00&quot;, &quot;t02&quot;, &quot;t04&quot;, &quot;t08&quot;, &quot;t24&quot;)

# Run differential expression for data with biological replicates and construct volcano plots
for(s1 in time_de){
  for(s2 in time_de){
    if(s1!=s2){
      if(s1&lt;s2){
        print(c(s1,s2))
        tmm &lt;- noiseqbio(dds_group12, norm = &quot;tmm&quot;, k = 0.5, factor = &quot;time_de&quot;, conditions = c(s1,s2), 
                             r = 50, filter = 1, nclust = 15, a0per= 0.9, random.seed = 12345, plot = TRUE, lc = 1)
        save(tmm, file = paste0(&quot;noiseq/group3/mate_&quot;,s1,&quot;vs&quot;,s2,&quot;_tmm.RData&quot;))
        tmm_r &lt;- as.data.frame(tmm@results, row.names = NULL)
        write.table(tmm@results, paste0(&quot;noiseq/group3/mate_&quot;,s1,&quot;vs&quot;,s2,&quot;_tmm.txt&quot;), 
                    sep = &quot;\t&quot;, row.names = TRUE, col.names = NA, quote = FALSE)
        tmm_r &lt;- na.omit(tmm_r)
        tmm_r$P &lt;- with(tmm_r, 1-prob)
        tmm_r$locus_tag &lt;- row.names(tmm_r)
        write.table(tmm_r, paste0(&quot;noiseq/group3/mate_&quot;,s1,&quot;vs&quot;,s2,&quot;_tmm_filtered.txt&quot;), 
                    sep = &quot;\t&quot;, row.names = TRUE, col.names = NA, quote = FALSE)
        tmm_rm &lt;- merge(tmm_r, hvo_map, by = &quot;locus_tag&quot;, all.x=TRUE)
        tmm_rm$EFFECTSIZE &lt;- tmm_rm$log2FC
        write.table(tmm_rm, paste0(&quot;noiseq/group3/mate_&quot;,s1,&quot;vs&quot;,s2,&quot;_tmm_filt_annot.txt&quot;),
                    sep = &quot;\t&quot;, row.names = F, quote = FALSE)
      }}}}</code></pre>
<pre><code>## [1] &quot;shaking&quot; &quot;t00&quot;    
## Computing Z values...
## Filtering out low count features...
## 272 features are to be kept for differential expression analysis with filtering method 1
## ...k-means clustering done
## Size of 15 clusters:
##  [1] 39 13 38 10  1 74 10 12  6 16  5 30  9  7  2
## Resampling cluster...[1] 1
## [1] 2
## [1] 3
## [1] 4
## [1] 5
## [1] 6
## [1] 7
## [1] 8
## [1] 9
## [1] 10
## [1] 11
## [1] 12
## [1] 13
## [1] 14
## [1] 15
## Computing Z for noise...
## Computing probability of differential expression...</code></pre>
<p><img src="hvo_mating_files/figure-html/de%20on%20clusters-1.png" width="672" /></p>
<pre><code>## p0 = 0.578142027352049
## Probability
##    Min. 1st Qu.  Median    Mean 3rd Qu.    Max.    NA&#39;s 
##  0.0000  0.2232  0.3257  0.4256  0.6832  0.9500      17 
## [1] &quot;shaking&quot; &quot;t02&quot;    
## Computing Z values...
## Filtering out low count features...
## 272 features are to be kept for differential expression analysis with filtering method 1
## ...k-means clustering done
## Size of 15 clusters:
##  [1] 39 23 60  7  1 44  4 14  9 21  2 27  9  8  4
## Resampling cluster...[1] 1
## [1] 2
## [1] 3
## [1] 4
## [1] 5
## [1] 6
## [1] 7
## [1] 8
## [1] 9
## [1] 10
## [1] 11
## [1] 12
## [1] 13
## [1] 14
## [1] 15
## Computing Z for noise...
## Computing probability of differential expression...</code></pre>
<p><img src="hvo_mating_files/figure-html/de%20on%20clusters-2.png" width="672" /></p>
<pre><code>## p0 = 0.56279611021385
## Probability
##    Min. 1st Qu.  Median    Mean 3rd Qu.    Max.    NA&#39;s 
##  0.0000  0.1862  0.3982  0.4351  0.6420  0.9164      17 
## [1] &quot;shaking&quot; &quot;t04&quot;    
## Computing Z values...
## Filtering out low count features...
## 272 features are to be kept for differential expression analysis with filtering method 1
## ...k-means clustering done
## Size of 15 clusters:
##  [1] 23 25 86  9  1 40  3  8  3 37  3 12 14  5  3
## Resampling cluster...[1] 1
## [1] 2
## [1] 3
## [1] 4
## [1] 5
## [1] 6
## [1] 7
## [1] 8
## [1] 9
## [1] 10
## [1] 11
## [1] 12
## [1] 13
## [1] 14
## [1] 15
## Computing Z for noise...
## Computing probability of differential expression...</code></pre>
<p><img src="hvo_mating_files/figure-html/de%20on%20clusters-3.png" width="672" /></p>
<pre><code>## p0 = 0.43487164077483
## Probability
##    Min. 1st Qu.  Median    Mean 3rd Qu.    Max.    NA&#39;s 
## 0.00000 0.08161 0.61062 0.53608 0.96163 0.98645      17 
## [1] &quot;shaking&quot; &quot;t08&quot;    
## Computing Z values...
## Filtering out low count features...
## 272 features are to be kept for differential expression analysis with filtering method 1
## ...k-means clustering done
## Size of 15 clusters:
##  [1]  27  16 114   8   1  42   5   8   5  24   2  10   4   2   4
## Resampling cluster...[1] 1
## [1] 2
## [1] 3
## [1] 4
## [1] 5
## [1] 6
## [1] 7
## [1] 8
## [1] 9
## [1] 10
## [1] 11
## [1] 12
## [1] 13
## [1] 14
## [1] 15
## Computing Z for noise...
## Computing probability of differential expression...</code></pre>
<p><img src="hvo_mating_files/figure-html/de%20on%20clusters-4.png" width="672" /></p>
<pre><code>## p0 = 0.352137219940421
## Probability
##    Min. 1st Qu.  Median    Mean 3rd Qu.    Max.    NA&#39;s 
##  0.0000  0.3267  0.7299  0.6332  0.9522  0.9951      17 
## [1] &quot;shaking&quot; &quot;t24&quot;    
## Computing Z values...
## Filtering out low count features...
## 272 features are to be kept for differential expression analysis with filtering method 1
## ...k-means clustering done
## Size of 15 clusters:
##  [1] 25  4 10 38  5  4 15 17  6 44 88  7  2  5  2
## Resampling cluster...[1] 1
## [1] 2
## [1] 3
## [1] 4
## [1] 5
## [1] 6
## [1] 7
## [1] 8
## [1] 9
## [1] 10
## [1] 11
## [1] 12
## [1] 13
## [1] 14
## [1] 15
## Computing Z for noise...
## Computing probability of differential expression...</code></pre>
<p><img src="hvo_mating_files/figure-html/de%20on%20clusters-5.png" width="672" /></p>
<pre><code>## p0 = 0.56390819556253
## Probability
##    Min. 1st Qu.  Median    Mean 3rd Qu.    Max.    NA&#39;s 
## 0.00000 0.09576 0.27852 0.41448 0.80539 0.96245      17 
## [1] &quot;t00&quot; &quot;t02&quot;
## Computing Z values...
## Filtering out low count features...
## 272 features are to be kept for differential expression analysis with filtering method 1
## ...k-means clustering done
## Size of 15 clusters:
##  [1] 47 30 19 11  3 44 15 13 23  7  2 38 10  8  2
## Resampling cluster...[1] 1
## [1] 2
## [1] 3
## [1] 4
## [1] 5
## [1] 6
## [1] 7
## [1] 8
## [1] 9
## [1] 10
## [1] 11
## [1] 12
## [1] 13
## [1] 14
## [1] 15
## Computing Z for noise...
## Computing probability of differential expression...
## [1] &quot;WARNING: f0(0)/f(0) &lt; 1 =&gt; FP with Z~0 will be detected.&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/de%20on%20clusters-6.png" width="672" /></p>
<pre><code>## p0 = 0.698484932236417
## Probability
##    Min. 1st Qu.  Median    Mean 3rd Qu.    Max.    NA&#39;s 
##  0.0000  0.2150  0.3580  0.3323  0.4541  0.4810      17 
## [1] &quot;t00&quot; &quot;t04&quot;
## Computing Z values...
## Filtering out low count features...
## 272 features are to be kept for differential expression analysis with filtering method 1
## ...k-means clustering done
## Size of 15 clusters:
##  [1] 53 19 28  6  4 67  7 10 11  6  2 41 12  4  2
## Resampling cluster...[1] 1
## [1] 2
## [1] 3
## [1] 4
## [1] 5
## [1] 6
## [1] 7
## [1] 8
## [1] 9
## [1] 10
## [1] 11
## [1] 12
## [1] 13
## [1] 14
## [1] 15
## Computing Z for noise...
## Computing probability of differential expression...</code></pre>
<p><img src="hvo_mating_files/figure-html/de%20on%20clusters-7.png" width="672" /></p>
<pre><code>## p0 = 0.305643088177354
## Probability
##    Min. 1st Qu.  Median    Mean 3rd Qu.    Max.    NA&#39;s 
##  0.0000  0.4830  0.8123  0.6863  0.9553  1.0000      17 
## [1] &quot;t00&quot; &quot;t08&quot;
## Computing Z values...
## Filtering out low count features...
## 272 features are to be kept for differential expression analysis with filtering method 1
## ...k-means clustering done
## Size of 15 clusters:
##  [1] 10  7 60  3  1 76  4  8 34  6  3 30 22  4  4
## Resampling cluster...[1] 1
## [1] 2
## [1] 3
## [1] 4
## [1] 5
## [1] 6
## [1] 7
## [1] 8
## [1] 9
## [1] 10
## [1] 11
## [1] 12
## [1] 13
## [1] 14
## [1] 15
## Computing Z for noise...
## Computing probability of differential expression...</code></pre>
<p><img src="hvo_mating_files/figure-html/de%20on%20clusters-8.png" width="672" /></p>
<pre><code>## p0 = 0.259865914282089
## Probability
##    Min. 1st Qu.  Median    Mean 3rd Qu.    Max.    NA&#39;s 
##  0.0000  0.6044  0.9120  0.7426  0.9649  1.0000      17 
## [1] &quot;t00&quot; &quot;t24&quot;
## Computing Z values...
## Filtering out low count features...
## 272 features are to be kept for differential expression analysis with filtering method 1
## ...k-means clustering done
## Size of 15 clusters:
##  [1] 10 50 68  2 19  9 10  6  5 29  4 25  3  9 23
## Resampling cluster...[1] 1
## [1] 2
## [1] 3
## [1] 4
## [1] 5
## [1] 6
## [1] 7
## [1] 8
## [1] 9
## [1] 10
## [1] 11
## [1] 12
## [1] 13
## [1] 14
## [1] 15
## Computing Z for noise...
## Computing probability of differential expression...</code></pre>
<p><img src="hvo_mating_files/figure-html/de%20on%20clusters-9.png" width="672" /></p>
<pre><code>## p0 = 0.329754247391958
## Probability
##    Min. 1st Qu.  Median    Mean 3rd Qu.    Max.    NA&#39;s 
##  0.0000  0.3042  0.8469  0.6389  0.9540  1.0000      17 
## [1] &quot;t02&quot; &quot;t04&quot;
## Computing Z values...
## Filtering out low count features...
## 272 features are to be kept for differential expression analysis with filtering method 1
## ...k-means clustering done
## Size of 15 clusters:
##  [1] 65 20 40  6  4 48  6 13  7  9  2 41  5  4  2
## Resampling cluster...[1] 1
## [1] 2
## [1] 3
## [1] 4
## [1] 5
## [1] 6
## [1] 7
## [1] 8
## [1] 9
## [1] 10
## [1] 11
## [1] 12
## [1] 13
## [1] 14
## [1] 15
## Computing Z for noise...
## Computing probability of differential expression...</code></pre>
<p><img src="hvo_mating_files/figure-html/de%20on%20clusters-10.png" width="672" /></p>
<pre><code>## p0 = 0.297035141396481
## Probability
##    Min. 1st Qu.  Median    Mean 3rd Qu.    Max.    NA&#39;s 
##  0.0000  0.4733  0.8763  0.6937  0.9499  1.0000      17 
## [1] &quot;t02&quot; &quot;t08&quot;
## Computing Z values...
## Filtering out low count features...
## 272 features are to be kept for differential expression analysis with filtering method 1
## ...k-means clustering done
## Size of 15 clusters:
##  [1] 21 49  9  3 40  3  6 64  9 14  3 10  3  3 35
## Resampling cluster...[1] 1
## [1] 2
## [1] 3
## [1] 4
## [1] 5
## [1] 6
## [1] 7
## [1] 8
## [1] 9
## [1] 10
## [1] 11
## [1] 12
## [1] 13
## [1] 14
## [1] 15
## Computing Z for noise...
## Computing probability of differential expression...</code></pre>
<p><img src="hvo_mating_files/figure-html/de%20on%20clusters-11.png" width="672" /></p>
<pre><code>## p0 = 0.285131023143652
## Probability
##    Min. 1st Qu.  Median    Mean 3rd Qu.    Max.    NA&#39;s 
##  0.0000  0.5389  0.8807  0.7234  0.9577  1.0000      17 
## [1] &quot;t02&quot; &quot;t24&quot;
## Computing Z values...
## Filtering out low count features...
## 272 features are to be kept for differential expression analysis with filtering method 1
## ...k-means clustering done
## Size of 15 clusters:
##  [1] 72 14 27  6  2 57  8  6 10 15  1 24 18  8  4
## Resampling cluster...[1] 1
## [1] 2
## [1] 3
## [1] 4
## [1] 5
## [1] 6
## [1] 7
## [1] 8
## [1] 9
## [1] 10
## [1] 11
## [1] 12
## [1] 13
## [1] 14
## [1] 15
## Computing Z for noise...
## Computing probability of differential expression...</code></pre>
<p><img src="hvo_mating_files/figure-html/de%20on%20clusters-12.png" width="672" /></p>
<pre><code>## p0 = 0.343172908603688
## Probability
##    Min. 1st Qu.  Median    Mean 3rd Qu.    Max.    NA&#39;s 
##  0.0000  0.3252  0.8040  0.6388  0.9567  1.0000      17 
## [1] &quot;t04&quot; &quot;t08&quot;
## Computing Z values...
## Filtering out low count features...
## 289 features are to be kept for differential expression analysis with filtering method 1
## ...k-means clustering done
## Size of 15 clusters:
##  [1]  14   5 110   9  15   4   1   8  14   3   4   7  40  12  43
## Resampling cluster...[1] 1
## [1] 2
## [1] 3
## [1] 4
## [1] 5
## [1] 6
## [1] 7
## [1] 8
## [1] 9
## [1] 10
## [1] 11
## [1] 12
## [1] 13
## [1] 14
## [1] 15
## Computing Z for noise...
## Computing probability of differential expression...</code></pre>
<p><img src="hvo_mating_files/figure-html/de%20on%20clusters-13.png" width="672" /></p>
<pre><code>## p0 = 0.502026963896028
## Probability
##    Min. 1st Qu.  Median    Mean 3rd Qu.    Max. 
##  0.0000  0.1107  0.4549  0.4680  0.8191  1.0000 
## [1] &quot;t04&quot; &quot;t24&quot;
## Computing Z values...
## Filtering out low count features...
## 289 features are to be kept for differential expression analysis with filtering method 1
## ...k-means clustering done
## Size of 15 clusters:
##  [1]   4   8   3   5   3   4 123  13   5   2   7  31  20  20  41
## Resampling cluster...[1] 1
## [1] 2
## [1] 3
## [1] 4
## [1] 5
## [1] 6
## [1] 7
## [1] 8
## [1] 9
## [1] 10
## [1] 11
## [1] 12
## [1] 13
## [1] 14
## [1] 15
## Computing Z for noise...
## Computing probability of differential expression...</code></pre>
<p><img src="hvo_mating_files/figure-html/de%20on%20clusters-14.png" width="672" /></p>
<pre><code>## p0 = 0.20715345397881
## Probability
##    Min. 1st Qu.  Median    Mean 3rd Qu.    Max. 
##  0.0000  0.6191  0.9360  0.7710  0.9905  1.0000 
## [1] &quot;t08&quot; &quot;t24&quot;
## Computing Z values...
## Filtering out low count features...
## 289 features are to be kept for differential expression analysis with filtering method 1
## ...k-means clustering done
## Size of 15 clusters:
##  [1]   2   5   5   6  14   3   3   3 153  41  19   9  18   1   7
## Resampling cluster...[1] 1
## [1] 2
## [1] 3
## [1] 4
## [1] 5
## [1] 6
## [1] 7
## [1] 8
## [1] 9
## [1] 10
## [1] 11
## [1] 12
## [1] 13
## [1] 14
## [1] 15
## Computing Z for noise...
## Computing probability of differential expression...</code></pre>
<p><img src="hvo_mating_files/figure-html/de%20on%20clusters-15.png" width="672" /></p>
<pre><code>## p0 = 0.191129371997977
## Probability
##    Min. 1st Qu.  Median    Mean 3rd Qu.    Max. 
##  0.0000  0.6931  0.9298  0.7863  0.9948  1.0000</code></pre>
<pre class="r"><code>## rename cluster number to run on additional clusters (e.g. change &quot;group3&quot; to &quot;group12&quot;)</code></pre>
<pre class="r"><code>for(s1 in time_de){
  for(s2 in time_de){
    if(s1!=s2){
      if(s1&lt;s2){
        print(c(s1,s2))
        tmm_r &lt;- read.delim(file = paste0(&quot;noiseq/group3/mate_&quot;,s1,&quot;vs&quot;,s2,&quot;_tmm_filt_annot.txt&quot;), 
                            header=TRUE, row.names = NULL, com=&#39;&#39;, check.names=F)
        plot(tmm_r$log2FC, -log10(1-tmm_r$prob), pch=20, xlim=c(-7,7), ylim=c(0,16), main = paste(s1,&quot;vs&quot;,s2), 
          xlab = &quot;log2FC&quot;, ylab = &quot;-log10(p-value)&quot;,
            col = ifelse(abs(tmm_r$log2FC)&gt;5 &amp; tmm_r$prob&gt;0.999,&quot;red&quot;,
              ifelse(abs(tmm_r$log2FC)&gt;5,&quot;orange&quot;,
                ifelse(tmm_r$prob&gt;0.99,&quot;green&quot;,&quot;black&quot;))))
      }}}}</code></pre>
<pre><code>## [1] &quot;shaking&quot; &quot;t00&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/group3%20figures-1.png" width="672" /></p>
<pre><code>## [1] &quot;shaking&quot; &quot;t02&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/group3%20figures-2.png" width="672" /></p>
<pre><code>## [1] &quot;shaking&quot; &quot;t04&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/group3%20figures-3.png" width="672" /></p>
<pre><code>## [1] &quot;shaking&quot; &quot;t08&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/group3%20figures-4.png" width="672" /></p>
<pre><code>## [1] &quot;shaking&quot; &quot;t24&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/group3%20figures-5.png" width="672" /></p>
<pre><code>## [1] &quot;t00&quot; &quot;t02&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/group3%20figures-6.png" width="672" /></p>
<pre><code>## [1] &quot;t00&quot; &quot;t04&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/group3%20figures-7.png" width="672" /></p>
<pre><code>## [1] &quot;t00&quot; &quot;t08&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/group3%20figures-8.png" width="672" /></p>
<pre><code>## [1] &quot;t00&quot; &quot;t24&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/group3%20figures-9.png" width="672" /></p>
<pre><code>## [1] &quot;t02&quot; &quot;t04&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/group3%20figures-10.png" width="672" /></p>
<pre><code>## [1] &quot;t02&quot; &quot;t08&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/group3%20figures-11.png" width="672" /></p>
<pre><code>## [1] &quot;t02&quot; &quot;t24&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/group3%20figures-12.png" width="672" /></p>
<pre><code>## [1] &quot;t04&quot; &quot;t08&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/group3%20figures-13.png" width="672" /></p>
<pre><code>## [1] &quot;t04&quot; &quot;t24&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/group3%20figures-14.png" width="672" /></p>
<pre><code>## [1] &quot;t08&quot; &quot;t24&quot;</code></pre>
<p><img src="hvo_mating_files/figure-html/group3%20figures-15.png" width="672" /></p>
<pre class="r"><code>for(s1 in time_de){
  for(s2 in time_de){
    if(s1!=s2){
      if(s1&lt;s2){
        print(paste0(&quot;Expression at time point &quot;,s1,&quot; is higher than time point &quot;,s2,
                     &quot; right of zero (i.e. positive) and lower than &quot;,s2, &quot; to the left of zero (i.e. negative).&quot;))
        tmm_r &lt;- read.delim(paste0(&quot;noiseq/group3/mate_&quot;,s1,&quot;vs&quot;,s2,&quot;_tmm_filt_annot.txt&quot;), 
                            header=TRUE, row.names = NULL, com=&#39;&#39;, check.names=F)
        volc_obj &lt;- volcanor(tmm_r, p = &quot;P&quot;, effect_size = &quot;EFFECTSIZE&quot;, snp = &quot;locus_tag&quot;, 
                             gene = &quot;protein&quot;, annotation1 = &quot;locus_tag&quot;, annotation2 = &quot;arCOG_name&quot;)
        volc_plot &lt;- volcanoly(volc_obj, effect_size_line = c(-1,1), effect_size_line_color = &quot;orange&quot;, 
                  genomewideline = -log10(1e-2), genomewideline_color = &quot;green&quot;, title = paste(s1,&quot;vs&quot;,s2), 
                  highlight = tmm_r$locus_tag, highlight_color = &quot;black&quot;)
        print(volc_plot)
      }}}}</code></pre>
<pre><code>## [1] &quot;Expression at time point shaking is higher than time point t00 right of zero (i.e. positive) and lower than t00 to the left of zero (i.e. negative).&quot;
## [1] &quot;Expression at time point shaking is higher than time point t02 right of zero (i.e. positive) and lower than t02 to the left of zero (i.e. negative).&quot;
## [1] &quot;Expression at time point shaking is higher than time point t04 right of zero (i.e. positive) and lower than t04 to the left of zero (i.e. negative).&quot;
## [1] &quot;Expression at time point shaking is higher than time point t08 right of zero (i.e. positive) and lower than t08 to the left of zero (i.e. negative).&quot;
## [1] &quot;Expression at time point shaking is higher than time point t24 right of zero (i.e. positive) and lower than t24 to the left of zero (i.e. negative).&quot;
## [1] &quot;Expression at time point t00 is higher than time point t02 right of zero (i.e. positive) and lower than t02 to the left of zero (i.e. negative).&quot;
## [1] &quot;Expression at time point t00 is higher than time point t04 right of zero (i.e. positive) and lower than t04 to the left of zero (i.e. negative).&quot;
## [1] &quot;Expression at time point t00 is higher than time point t08 right of zero (i.e. positive) and lower than t08 to the left of zero (i.e. negative).&quot;
## [1] &quot;Expression at time point t00 is higher than time point t24 right of zero (i.e. positive) and lower than t24 to the left of zero (i.e. negative).&quot;
## [1] &quot;Expression at time point t02 is higher than time point t04 right of zero (i.e. positive) and lower than t04 to the left of zero (i.e. negative).&quot;
## [1] &quot;Expression at time point t02 is higher than time point t08 right of zero (i.e. positive) and lower than t08 to the left of zero (i.e. negative).&quot;
## [1] &quot;Expression at time point t02 is higher than time point t24 right of zero (i.e. positive) and lower than t24 to the left of zero (i.e. negative).&quot;
## [1] &quot;Expression at time point t04 is higher than time point t08 right of zero (i.e. positive) and lower than t08 to the left of zero (i.e. negative).&quot;
## [1] &quot;Expression at time point t04 is higher than time point t24 right of zero (i.e. positive) and lower than t24 to the left of zero (i.e. negative).&quot;
## [1] &quot;Expression at time point t08 is higher than time point t24 right of zero (i.e. positive) and lower than t24 to the left of zero (i.e. negative).&quot;</code></pre>
<pre class="r"><code>group &lt;- cluster_all$df %&gt;%
          filter(cluster == 3)

group &lt;- data.frame(lapply(group, function(x) {
  gsub(&quot;lcl.&quot;, &quot;lcl|&quot;, x)
}))
sub_counts &lt;- subset(annot_counts, ncbi_id %in% group$genes)

hvo_counts &lt;- sub_counts[, c(4,45:56)]
rownames(hvo_counts) &lt;- NULL
hvo_counts &lt;- distinct(hvo_counts)
rownames(hvo_counts) &lt;- hvo_counts$locus_tag
  hvo_counts$locus_tag &lt;- NULL
  hvo_counts &lt;- data.matrix(hvo_counts)

print(sub_counts$old_locus_tag)</code></pre>
<pre><code>## NULL</code></pre>
<pre class="r"><code>print(rownames(hvo_counts))</code></pre>
<pre><code>##   [1] &quot;HVO_RS00705&quot; &quot;HVO_RS00635&quot; &quot;HVO_RS00570&quot; &quot;HVO_RS00350&quot; &quot;HVO_RS00090&quot; &quot;HVO_RS01720&quot; &quot;HVO_RS01710&quot; &quot;HVO_RS01330&quot; &quot;HVO_RS00950&quot; &quot;HVO_RS00945&quot;
##  [11] &quot;HVO_RS00895&quot; &quot;HVO_RS00770&quot; &quot;HVO_RS00755&quot; &quot;HVO_RS19935&quot; &quot;HVO_RS03435&quot; &quot;HVO_RS04090&quot; &quot;HVO_RS04280&quot; &quot;HVO_RS04545&quot; &quot;HVO_RS04675&quot; &quot;HVO_RS02480&quot;
##  [21] &quot;HVO_RS02940&quot; &quot;HVO_RS03305&quot; &quot;HVO_RS01920&quot; &quot;HVO_RS02130&quot; &quot;HVO_RS19745&quot; &quot;HVO_RS02045&quot; &quot;HVO_RS02040&quot; &quot;HVO_RS01975&quot; &quot;HVO_RS01970&quot; &quot;HVO_RS01960&quot;
##  [31] &quot;HVO_RS01955&quot; &quot;HVO_RS03500&quot; &quot;HVO_RS02535&quot; &quot;HVO_RS03835&quot; &quot;HVO_RS03085&quot; &quot;HVO_RS03550&quot; &quot;HVO_RS02965&quot; &quot;HVO_RS19945&quot; &quot;HVO_RS02380&quot; &quot;HVO_RS03005&quot;
##  [41] &quot;HVO_RS20080&quot; &quot;HVO_RS02165&quot; &quot;HVO_RS03490&quot; &quot;HVO_RS02455&quot; &quot;HVO_RS03695&quot; &quot;HVO_RS19720&quot; &quot;HVO_RS19805&quot; &quot;HVO_RS20145&quot; &quot;HVO_RS20160&quot; &quot;HVO_RS20165&quot;
##  [51] &quot;HVO_RS12285&quot; &quot;HVO_RS12345&quot; &quot;HVO_RS12475&quot; &quot;HVO_RS12560&quot; &quot;HVO_RS12625&quot; &quot;HVO_RS12645&quot; &quot;HVO_RS12715&quot; &quot;HVO_RS12795&quot; &quot;HVO_RS12855&quot; &quot;HVO_RS12955&quot;
##  [61] &quot;HVO_RS13045&quot; &quot;HVO_RS13100&quot; &quot;HVO_RS13115&quot; &quot;HVO_RS13545&quot; &quot;HVO_RS13795&quot; &quot;HVO_RS13805&quot; &quot;HVO_RS13810&quot; &quot;HVO_RS14080&quot; &quot;HVO_RS14085&quot; &quot;HVO_RS14205&quot;
##  [71] &quot;HVO_RS14460&quot; &quot;HVO_RS14600&quot; &quot;HVO_RS14635&quot; &quot;HVO_RS14640&quot; &quot;HVO_RS14935&quot; &quot;HVO_RS14985&quot; &quot;HVO_RS15030&quot; &quot;HVO_RS15300&quot; &quot;HVO_RS15530&quot; &quot;HVO_RS15550&quot;
##  [81] &quot;HVO_RS15585&quot; &quot;HVO_RS15625&quot; &quot;HVO_RS15645&quot; &quot;HVO_RS15650&quot; &quot;HVO_RS15690&quot; &quot;HVO_RS15710&quot; &quot;HVO_RS15770&quot; &quot;HVO_RS15840&quot; &quot;HVO_RS15990&quot; &quot;HVO_RS16010&quot;
##  [91] &quot;HVO_RS16125&quot; &quot;HVO_RS16415&quot; &quot;HVO_RS16505&quot; &quot;HVO_RS16635&quot; &quot;HVO_RS16710&quot; &quot;HVO_RS16735&quot; &quot;HVO_RS16740&quot; &quot;HVO_RS16790&quot; &quot;HVO_RS16820&quot; &quot;HVO_RS16925&quot;
## [101] &quot;HVO_RS16965&quot; &quot;HVO_RS16970&quot; &quot;HVO_RS16975&quot; &quot;HVO_RS16980&quot; &quot;HVO_RS16985&quot; &quot;HVO_RS16990&quot; &quot;HVO_RS17000&quot; &quot;HVO_RS17005&quot; &quot;HVO_RS17010&quot; &quot;HVO_RS17015&quot;
## [111] &quot;HVO_RS17020&quot; &quot;HVO_RS17025&quot; &quot;HVO_RS17030&quot; &quot;HVO_RS17040&quot; &quot;HVO_RS17045&quot; &quot;HVO_RS17050&quot; &quot;HVO_RS17055&quot; &quot;HVO_RS17060&quot; &quot;HVO_RS17065&quot; &quot;HVO_RS17070&quot;
## [121] &quot;HVO_RS17075&quot; &quot;HVO_RS17080&quot; &quot;HVO_RS17085&quot; &quot;HVO_RS17200&quot; &quot;HVO_RS17370&quot; &quot;HVO_RS17375&quot; &quot;HVO_RS17380&quot; &quot;HVO_RS17530&quot; &quot;HVO_RS20305&quot; &quot;HVO_RS17855&quot;
## [131] &quot;HVO_RS17925&quot; &quot;HVO_RS17930&quot; &quot;HVO_RS17935&quot; &quot;HVO_RS17940&quot; &quot;HVO_RS17950&quot; &quot;HVO_RS17980&quot; &quot;HVO_RS18000&quot; &quot;HVO_RS18005&quot; &quot;HVO_RS18015&quot; &quot;HVO_RS18020&quot;
## [141] &quot;HVO_RS18025&quot; &quot;HVO_RS18030&quot; &quot;HVO_RS18105&quot; &quot;HVO_RS18110&quot; &quot;HVO_RS18115&quot; &quot;HVO_RS18120&quot; &quot;HVO_RS18125&quot; &quot;HVO_RS18130&quot; &quot;HVO_RS18135&quot; &quot;HVO_RS18145&quot;
## [151] &quot;HVO_RS18150&quot; &quot;HVO_RS18155&quot; &quot;HVO_RS18160&quot; &quot;HVO_RS18225&quot; &quot;HVO_RS20310&quot; &quot;HVO_RS11955&quot; &quot;HVO_RS11720&quot; &quot;HVO_RS11705&quot; &quot;HVO_RS11700&quot; &quot;HVO_RS11455&quot;
## [161] &quot;HVO_RS11405&quot; &quot;HVO_RS11250&quot; &quot;HVO_RS11245&quot; &quot;HVO_RS11065&quot; &quot;HVO_RS10710&quot; &quot;HVO_RS10340&quot; &quot;HVO_RS10305&quot; &quot;HVO_RS10300&quot; &quot;HVO_RS10190&quot; &quot;HVO_RS10185&quot;
## [171] &quot;HVO_RS10170&quot; &quot;HVO_RS09940&quot; &quot;HVO_RS09845&quot; &quot;HVO_RS09565&quot; &quot;HVO_RS09430&quot; &quot;HVO_RS09415&quot; &quot;HVO_RS09410&quot; &quot;HVO_RS09405&quot; &quot;HVO_RS09390&quot; &quot;HVO_RS09360&quot;
## [181] &quot;HVO_RS09085&quot; &quot;HVO_RS09065&quot; &quot;HVO_RS09060&quot; &quot;HVO_RS08955&quot; &quot;HVO_RS08935&quot; &quot;HVO_RS08920&quot; &quot;HVO_RS08875&quot; &quot;HVO_RS08820&quot; &quot;HVO_RS08815&quot; &quot;HVO_RS08670&quot;
## [191] &quot;HVO_RS08570&quot; &quot;HVO_RS08515&quot; &quot;HVO_RS08450&quot; &quot;HVO_RS08200&quot; &quot;HVO_RS08070&quot; &quot;HVO_RS08065&quot; &quot;HVO_RS08060&quot; &quot;HVO_RS08055&quot; &quot;HVO_RS08030&quot; &quot;HVO_RS07955&quot;
## [201] &quot;HVO_RS07845&quot; &quot;HVO_RS07395&quot; &quot;HVO_RS07045&quot; &quot;HVO_RS07000&quot; &quot;HVO_RS06950&quot; &quot;HVO_RS06875&quot; &quot;HVO_RS06735&quot; &quot;HVO_RS06705&quot; &quot;HVO_RS06440&quot; &quot;HVO_RS06435&quot;
## [211] &quot;HVO_RS06420&quot; &quot;HVO_RS06410&quot; &quot;HVO_RS06405&quot; &quot;HVO_RS06325&quot; &quot;HVO_RS06245&quot; &quot;HVO_RS06235&quot; &quot;HVO_RS06230&quot; &quot;HVO_RS06225&quot; &quot;HVO_RS06220&quot; &quot;HVO_RS06215&quot;
## [221] &quot;HVO_RS06210&quot; &quot;HVO_RS06200&quot; &quot;HVO_RS18570&quot; &quot;HVO_RS18660&quot; &quot;HVO_RS18685&quot; &quot;HVO_RS18880&quot; &quot;HVO_RS18885&quot; &quot;HVO_RS18960&quot; &quot;HVO_RS19080&quot; &quot;HVO_RS19125&quot;
## [231] &quot;HVO_RS19255&quot; &quot;HVO_RS04975&quot; &quot;HVO_RS04980&quot; &quot;HVO_RS04985&quot; &quot;HVO_RS04990&quot; &quot;HVO_RS04995&quot; &quot;HVO_RS05250&quot; &quot;HVO_RS05255&quot; &quot;HVO_RS05260&quot; &quot;HVO_RS05375&quot;
## [241] &quot;HVO_RS05455&quot; &quot;HVO_RS05560&quot; &quot;HVO_RS05595&quot; &quot;HVO_RS05655&quot; &quot;HVO_RS05875&quot; &quot;HVO_RS09050&quot; &quot;HVO_RS05545&quot; &quot;HVO_RS11375&quot; &quot;HVO_RS15735&quot; &quot;HVO_RS12020&quot;
## [251] &quot;HVO_RS18675&quot; &quot;HVO_RS15640&quot; &quot;HVO_RS06205&quot; &quot;HVO_RS17035&quot; &quot;HVO_RS12050&quot; &quot;HVO_RS13495&quot; &quot;HVO_RS16995&quot; &quot;HVO_RS13270&quot; &quot;HVO_RS15720&quot; &quot;HVO_RS16220&quot;
## [261] &quot;HVO_RS15560&quot; &quot;HVO_RS18335&quot; &quot;HVO_RS12405&quot; &quot;HVO_RS06850&quot; &quot;HVO_RS15590&quot; &quot;HVO_RS14660&quot; &quot;HVO_RS15535&quot; &quot;HVO_RS10205&quot; &quot;HVO_RS15305&quot; &quot;HVO_RS14655&quot;
## [271] &quot;HVO_RS15685&quot; &quot;HVO_RS16920&quot; &quot;HVO_RS15695&quot; &quot;HVO_RS20090&quot; &quot;HVO_RS20170&quot; &quot;HVO_RS20345&quot; &quot;HVO_RS20355&quot; &quot;HVO_RS20405&quot; &quot;HVO_RS19590&quot; &quot;HVO_RS19475&quot;
## [281] &quot;HVO_RS19295&quot; &quot;HVO_RS19335&quot; &quot;HVO_RS19430&quot; &quot;HVO_RS19625&quot; &quot;HVO_RS20330&quot; &quot;HVO_RS20385&quot; &quot;HVO_RS20350&quot; &quot;HVO_RS19480&quot; &quot;HVO_RS19330&quot;</code></pre>
<pre class="r"><code>list_hvo &lt;- rownames(hvo_counts)

supp_table &lt;- sub_counts[, c(4,2,13,15,37,38)]
supp_table &lt;- distinct(supp_table)
write.table(supp_table, file = &quot;tables/tableS3_cluster3.txt&quot;, quote = FALSE, sep = &quot;\t&quot;, row.names = FALSE)

## full excel spreadsheet with additional clusters prepared by combining the above table for each cluster</code></pre>
<pre class="r"><code>## find operons in the cluster data
# filter hvo_map by cluster list -&gt; find start and stops within 100 nt?
# find consecutive by 5 RS within clustered genes
cdf &lt;- NULL
cdf$genes &lt;- cluster_all$raw$genes
cdf$cluster &lt;- cluster_all$raw$cluster
cdf &lt;- as.data.frame(cdf)
cdf &lt;- unique(cdf)
cdf &lt;- data.frame(lapply(cdf, function(x) {
  gsub(&quot;lcl.&quot;, &quot;lcl|&quot;, x)
}))

sub_map &lt;- NULL
sub_map$ncbi_id &lt;- hvo_map$ncbi_id
sub_map$locus_tag &lt;- hvo_map$locus_tag
sub_map$old_locus_tag &lt;- hvo_map$old_locus_tag
sub_map$start &lt;- hvo_map$start
sub_map$stop &lt;- hvo_map$stop
sub_map$chr &lt;- hvo_map$chr
sub_map &lt;- as.data.frame(sub_map)

cdf &lt;- merge(cdf, sub_map, by.x = &quot;genes&quot;, by.y = &quot;ncbi_id&quot;, all.x = TRUE)

cdf &lt;- separate(cdf, locus_tag, into = c(&quot;na&quot;, &quot;rs&quot;), sep = 6, remove = FALSE)
cdf &lt;- unique(cdf)
cdf$na &lt;- NULL
cdf$rs &lt;- as.numeric(cdf$rs)
mat &lt;- NULL
# filter by cluster, then for each value in cdf$rs, take abs value of the difference and put into a matrix
dir.create(&quot;tables/cluster_operons&quot;)</code></pre>
<pre><code>## Warning in dir.create(&quot;tables/cluster_operons&quot;): &#39;tables/cluster_operons&#39; already exists</code></pre>
<pre class="r"><code>for (x in 1:22) {
  cdfsub &lt;- cdf %&gt;%
          filter(cluster == x)
  cdfsub &lt;- cdfsub[order(cdfsub$rs),]
  cdfsub$op1 &lt;- c(0, abs(diff(cdfsub$rs)) == 5)
  print(cdfsub)
  write.table(cdfsub, file = paste0(&quot;tables/cluster_operons/cluster_&quot;,x,&quot;_operons.txt&quot;), sep = &quot;\t&quot;, row.names = FALSE, quote = FALSE)
  # mat &lt;- matrix(nrow = length(cdfsub$genes), ncol = length(cdfsub$genes))
  # rownames(mat) &lt;- cdfsub$genes
  # colnames(mat) &lt;- cdfsub$genes
  # mat
}</code></pre>
<pre><code>##                                      genes cluster   locus_tag    rs   start    stop        chr op1
## 1   lcl|NC_013964.1_cds_WP_004041051.1_190       1 HVO_RS00505   505  117697  117999       pHV3   0
## 2   lcl|NC_013966.1_cds_WP_004040996.1_467       1 HVO_RS01890  1890    5824    6483       pHV4   0
## 5   lcl|NC_013966.1_cds_WP_004043398.1_505       1 HVO_RS02055  2055   37089   37340       pHV4   0
## 4   lcl|NC_013966.1_cds_WP_004043365.1_535       1 HVO_RS02195  2195   63533   64375       pHV4   0
## 3   lcl|NC_013966.1_cds_WP_004043126.1_660       1 HVO_RS02760  2760  208856  210925       pHV4   0
## 6   lcl|NC_013966.1_cds_WP_013035073.1_831       1 HVO_RS03545  3545  391212  392903       pHV4   0
## 8   lcl|NC_013966.1_cds_WP_013035173.1_862       1 HVO_RS03685  3685  419256  419897       pHV4   0
## 18 lcl|NC_013967.1_cds_WP_004044633.1_1360       1 HVO_RS06100  6100  260022  260435 Chromosome   0
## 17 lcl|NC_013967.1_cds_WP_004044531.1_1455       1 HVO_RS06590  6590  347026  347382 Chromosome   0
## 16 lcl|NC_013967.1_cds_WP_004044352.1_1630       1 HVO_RS07495  7495  516141  517343 Chromosome   0
## 15 lcl|NC_013967.1_cds_WP_004043859.1_2103       1 HVO_RS09880  9880  984665  985642 Chromosome   0
## 14 lcl|NC_013967.1_cds_WP_004043463.1_2483       1 HVO_RS11805 11805 1343584 1343796 Chromosome   0
## 9  lcl|NC_013967.1_cds_WP_004041405.1_2534       1 HVO_RS12035 12035 1393000 1393731 Chromosome   0
## 10 lcl|NC_013967.1_cds_WP_004041450.1_2576       1 HVO_RS12245 12245 1438743 1441139 Chromosome   0
## 11 lcl|NC_013967.1_cds_WP_004041552.1_2664       1 HVO_RS12685 12685 1525352 1525714 Chromosome   0
## 12 lcl|NC_013967.1_cds_WP_004041843.1_2941       1 HVO_RS14120 14120 1813650 1813895 Chromosome   0
## 22 lcl|NC_013967.1_cds_WP_049914791.1_2999       1 HVO_RS14405 14405 1865495 1865797 Chromosome   0
## 13 lcl|NC_013967.1_cds_WP_004041935.1_3029       1 HVO_RS14560 14560 1895852 1896775 Chromosome   0
## 19 lcl|NC_013967.1_cds_WP_004044866.1_3932       1 HVO_RS19130 19130 2814791 2814994 Chromosome   0
## 20 lcl|NC_013967.1_cds_WP_004044895.1_3943       1 HVO_RS19180 19180 2822533 2822901 Chromosome   0
## 21 lcl|NC_013967.1_cds_WP_004044952.1_3964       1 HVO_RS19285 19285 2846015 2846656 Chromosome   0
## 23   lcl|NC_013968.1_cds_WP_013035685.1_52       1 HVO_RS19505 19505   55340   55567       pHV1   0
## 7   lcl|NC_013966.1_cds_WP_013035126.1_881       1 HVO_RS20030 20030  438155  438697       pHV4   0
## 25   lcl|NC_013968.1_cds_WP_013035690.1_28       1 HVO_RS20370 20370   19887   20411       pHV1   0
## 26   lcl|NC_013968.1_cds_WP_013035694.1_29       1 HVO_RS20375 20375   20437   20628       pHV1   1
## 24   lcl|NC_013968.1_cds_WP_013035686.1_53       1 HVO_RS20410 20410   55899   56111       pHV1   0
##                                      genes cluster   locus_tag    rs   start    stop        chr op1
## 16  lcl|NC_013964.1_cds_WP_004041113.1_128       2 HVO_RS00195   195   46367   48205       pHV3   0
## 15  lcl|NC_013964.1_cds_WP_004041112.1_129       2 HVO_RS00200   200   48214   49587       pHV3   1
## 14  lcl|NC_013964.1_cds_WP_004041110.1_131       2 HVO_RS00210   210   50222   52003       pHV3   0
## 13  lcl|NC_013964.1_cds_WP_004041109.1_132       2 HVO_RS00215   215   52000   53565       pHV3   1
## 12  lcl|NC_013964.1_cds_WP_004041108.1_133       2 HVO_RS00220   220   53562   54941       pHV3   1
## 11  lcl|NC_013964.1_cds_WP_004041107.1_134       2 HVO_RS00225   225   55399   56601       pHV3   1
## 10  lcl|NC_013964.1_cds_WP_004041105.1_136       2 HVO_RS00235   235   57495   58175       pHV3   0
## 9   lcl|NC_013964.1_cds_WP_004041104.1_137       2 HVO_RS00240   240   58168   62022       pHV3   1
## 8   lcl|NC_013964.1_cds_WP_004041100.1_141       2 HVO_RS00260   260   65852   67072       pHV3   0
## 7   lcl|NC_013964.1_cds_WP_004041097.1_144       2 HVO_RS00275   275   67999   69033       pHV3   0
## 6   lcl|NC_013964.1_cds_WP_004041096.1_145       2 HVO_RS00280   280   69037   69921       pHV3   1
## 5   lcl|NC_013964.1_cds_WP_004041095.1_146       2 HVO_RS00285   285   69918   70898       pHV3   1
## 4   lcl|NC_013964.1_cds_WP_004041094.1_147       2 HVO_RS00290   290   70891   71730       pHV3   1
## 3   lcl|NC_013964.1_cds_WP_004041089.1_152       2 HVO_RS00315   315   74179   74520       pHV3   0
## 2   lcl|NC_013964.1_cds_WP_004041060.1_181       2 HVO_RS00460   460  107105  107566       pHV3   0
## 1   lcl|NC_013964.1_cds_WP_004041026.1_215       2 HVO_RS00630   630  144548  144955       pHV3   0
## 18  lcl|NC_013964.1_cds_WP_081442862.1_234       2 HVO_RS00725   725  167680  168720       pHV3   0
## 17  lcl|NC_013964.1_cds_WP_004041294.1_328       2 HVO_RS01185  1185  287729  288964       pHV3   0
## 21  lcl|NC_013966.1_cds_WP_004043134.1_667       2 HVO_RS02795  2795  218613  219671       pHV4   0
## 22  lcl|NC_013966.1_cds_WP_004043153.1_687       2 HVO_RS02890  2890  237346  237771       pHV4   0
## 23  lcl|NC_013966.1_cds_WP_004043230.1_767       2 HVO_RS03265  3265  327461  327676       pHV4   0
## 25  lcl|NC_013966.1_cds_WP_013035099.1_877       2 HVO_RS03750  3750  432411  434114       pHV4   0
## 19  lcl|NC_013966.1_cds_WP_004040858.1_957       2 HVO_RS04105  4105  513415  514359       pHV4   0
## 20  lcl|NC_013966.1_cds_WP_004040892.1_991       2 HVO_RS04270  4270  546161  546592       pHV4   0
## 27 lcl|NC_013966.1_cds_WP_013035145.1_1004       2 HVO_RS04340  4340  561274  562509       pHV4   0
## 26 lcl|NC_013966.1_cds_WP_013035138.1_1029       2 HVO_RS04465  4465  589597  590685       pHV4   0
## 51 lcl|NC_013967.1_cds_WP_004044393.1_1589       2 HVO_RS07290  7290  468630  469064 Chromosome   0
## 50 lcl|NC_013967.1_cds_WP_004044233.1_1744       2 HVO_RS08075  8075  629659  629919 Chromosome   0
## 49 lcl|NC_013967.1_cds_WP_004044220.1_1757       2 HVO_RS08145  8145  643397  644536 Chromosome   0
## 48 lcl|NC_013967.1_cds_WP_004044201.1_1774       2 HVO_RS08230  8230  657583  658038 Chromosome   0
## 47 lcl|NC_013967.1_cds_WP_004044137.1_1835       2 HVO_RS08535  8535  719373  719642 Chromosome   0
## 46 lcl|NC_013967.1_cds_WP_004044044.1_1925       2 HVO_RS08985  8985  811838  813826 Chromosome   0
## 45 lcl|NC_013967.1_cds_WP_004044043.1_1926       2 HVO_RS08990  8990  813856  814101 Chromosome   1
## 44 lcl|NC_013967.1_cds_WP_004044032.1_1937       2 HVO_RS09045  9045  824054  825817 Chromosome   0
## 43 lcl|NC_013967.1_cds_WP_004043815.1_2145       2 HVO_RS10090 10090 1022943 1023128 Chromosome   0
## 42 lcl|NC_013967.1_cds_WP_004043596.1_2356       2 HVO_RS11165 11165 1221860 1222195 Chromosome   0
## 41 lcl|NC_013967.1_cds_WP_004043497.1_2450       2 HVO_RS11640 11640 1309411 1309830 Chromosome   0
## 40 lcl|NC_013967.1_cds_WP_004043489.1_2458       2 HVO_RS11680 11680 1317778 1318671 Chromosome   0
## 39 lcl|NC_013967.1_cds_WP_004043471.1_2475       2 HVO_RS11765 11765 1335454 1336671 Chromosome   0
## 28 lcl|NC_013967.1_cds_WP_004041505.1_2618       2 HVO_RS12455 12455 1482327 1483013 Chromosome   0
## 29 lcl|NC_013967.1_cds_WP_004041642.1_2753       2 HVO_RS13165 13165 1627769 1628884 Chromosome   0
## 56 lcl|NC_013967.1_cds_WP_049914765.1_2754       2 HVO_RS13170 13170 1628917 1629765 Chromosome   1
## 30 lcl|NC_013967.1_cds_WP_004041826.1_2925       2 HVO_RS14035 14035 1797346 1797639 Chromosome   0
## 31 lcl|NC_013967.1_cds_WP_004041872.1_2969       2 HVO_RS14260 14260 1837331 1838560 Chromosome   0
## 32 lcl|NC_013967.1_cds_WP_004041899.1_2994       2 HVO_RS14380 14380 1860594 1861604 Chromosome   0
## 33 lcl|NC_013967.1_cds_WP_004041925.1_3018       2 HVO_RS14505 14505 1883147 1884325 Chromosome   0
## 34 lcl|NC_013967.1_cds_WP_004041951.1_3046       2 HVO_RS14630 14630 1914245 1915219 Chromosome   0
## 35 lcl|NC_013967.1_cds_WP_004041966.1_3063       2 HVO_RS14715 14715 1942628 1943305 Chromosome   0
## 36 lcl|NC_013967.1_cds_WP_004041978.1_3074       2 HVO_RS14770 14770 1958400 1959647 Chromosome   0
## 37 lcl|NC_013967.1_cds_WP_004042020.1_3116       2 HVO_RS14975 14975 2002317 2002931 Chromosome   0
## 54 lcl|NC_013967.1_cds_WP_008578059.1_3666       2 HVO_RS17745 17745 2549210 2549455 Chromosome   0
## 38 lcl|NC_013967.1_cds_WP_004043033.1_3752       2 HVO_RS18185 18185 2622229 2623191 Chromosome   0
## 52 lcl|NC_013967.1_cds_WP_004044754.1_3879       2 HVO_RS18835 18835 2755914 2756096 Chromosome   0
## 55 lcl|NC_013967.1_cds_WP_013035500.1_3962       2 HVO_RS19275 19275 2844575 2845567 Chromosome   0
## 53 lcl|NC_013967.1_cds_WP_004044951.1_3963       2 HVO_RS19280 19280 2845616 2846008 Chromosome   1
## 57   lcl|NC_013968.1_cds_WP_013035710.1_41       2 HVO_RS19455 19455   42448   44433       pHV1   0
## 58   lcl|NC_013968.1_cds_WP_049941526.1_69       2 HVO_RS19585 19585   74734   76173       pHV1   0
## 24  lcl|NC_013966.1_cds_WP_013035095.1_804       2 HVO_RS19930 19930  363648  364424       pHV4   0
##                                       genes cluster   locus_tag    rs   start    stop        chr op1
## 5    lcl|NC_013964.1_cds_WP_004041134.1_106       3 HVO_RS00090    90   20051   20704       pHV3   0
## 4    lcl|NC_013964.1_cds_WP_004041082.1_159       3 HVO_RS00350   350   81961   82491       pHV3   0
## 3    lcl|NC_013964.1_cds_WP_004041038.1_203       3 HVO_RS00570   570  131561  133510       pHV3   0
## 2    lcl|NC_013964.1_cds_WP_004041025.1_216       3 HVO_RS00635   635  145516  146379       pHV3   0
## 1    lcl|NC_013964.1_cds_WP_004041011.1_230       3 HVO_RS00705   705  162589  163284       pHV3   0
## 13   lcl|NC_013964.1_cds_WP_004041383.1_240       3 HVO_RS00755   755  174818  175231       pHV3   0
## 12   lcl|NC_013964.1_cds_WP_004041380.1_243       3 HVO_RS00770   770  181542  182090       pHV3   0
## 11   lcl|NC_013964.1_cds_WP_004041353.1_268       3 HVO_RS00895   895  211615  212589       pHV3   0
## 10   lcl|NC_013964.1_cds_WP_004041342.1_279       3 HVO_RS00945   945  223140  223451       pHV3   0
## 9    lcl|NC_013964.1_cds_WP_004041341.1_280       3 HVO_RS00950   950  223453  223824       pHV3   1
## 8    lcl|NC_013964.1_cds_WP_004041265.1_357       3 HVO_RS01330  1330  325629  327335       pHV3   0
## 7    lcl|NC_013964.1_cds_WP_004041187.1_434       3 HVO_RS01710  1710  408141  410444       pHV3   0
## 6    lcl|NC_013964.1_cds_WP_004041182.1_437       3 HVO_RS01720  1720  411652  412176       pHV3   0
## 23   lcl|NC_013966.1_cds_WP_004043309.1_474       3 HVO_RS01920  1920   12870   13319       pHV4   0
## 31   lcl|NC_013966.1_cds_WP_004043418.1_484       3 HVO_RS01955  1955   22306   23010       pHV4   0
## 30   lcl|NC_013966.1_cds_WP_004043417.1_485       3 HVO_RS01960  1960   23099   23494       pHV4   1
## 29   lcl|NC_013966.1_cds_WP_004043415.1_487       3 HVO_RS01970  1970   24692   24859       pHV4   0
## 28   lcl|NC_013966.1_cds_WP_004043414.1_488       3 HVO_RS01975  1975   25023   25706       pHV4   1
## 27   lcl|NC_013966.1_cds_WP_004043402.1_500       3 HVO_RS02040  2040   34050   34475       pHV4   0
## 26   lcl|NC_013966.1_cds_WP_004043401.1_501       3 HVO_RS02045  2045   34478   34780       pHV4   1
## 24   lcl|NC_013966.1_cds_WP_004043378.1_521       3 HVO_RS02130  2130   48734   49555       pHV4   0
## 42   lcl|NC_013966.1_cds_WP_013035182.1_529       3 HVO_RS02165  2165   55398   56897       pHV4   0
## 39   lcl|NC_013966.1_cds_WP_013035157.1_574       3 HVO_RS02380  2380  110914  112080       pHV4   0
## 44   lcl|NC_013966.1_cds_WP_013035207.1_593       3 HVO_RS02455  2455  130759  131637       pHV4   0
## 20   lcl|NC_013966.1_cds_WP_004043070.1_601       3 HVO_RS02480  2480  136233  136844       pHV4   0
## 33   lcl|NC_013966.1_cds_WP_013035042.1_612       3 HVO_RS02535  2535  149297  150466       pHV4   0
## 21   lcl|NC_013966.1_cds_WP_004043162.1_698       3 HVO_RS02940  2940  247481  248008       pHV4   0
## 37   lcl|NC_013966.1_cds_WP_013035110.1_703       3 HVO_RS02965  2965  252951  253325       pHV4   0
## 40   lcl|NC_013966.1_cds_WP_013035158.1_713       3 HVO_RS03005  3005  264290  265456       pHV4   0
## 35   lcl|NC_013966.1_cds_WP_013035094.1_730       3 HVO_RS03085  3085  287605  287922       pHV4   0
## 22   lcl|NC_013966.1_cds_WP_004043238.1_777       3 HVO_RS03305  3305  334054  334635       pHV4   0
## 15                  lcl|NC_013966.1_cds_806       3 HVO_RS03435  3435  365276  365710       pHV4   0
## 43   lcl|NC_013966.1_cds_WP_013035191.1_819       3 HVO_RS03490  3490  378291  379457       pHV4   0
## 32   lcl|NC_013966.1_cds_WP_013035021.1_821       3 HVO_RS03500  3500  379843  380139       pHV4   0
## 36   lcl|NC_013966.1_cds_WP_013035096.1_832       3 HVO_RS03550  3550  392914  393933       pHV4   0
## 45   lcl|NC_013966.1_cds_WP_013035218.1_864       3 HVO_RS03695  3695  420828  421511       pHV4   0
## 34   lcl|NC_013966.1_cds_WP_013035066.1_901       3 HVO_RS03835  3835  456445  457611       pHV4   0
## 16   lcl|NC_013966.1_cds_WP_004040855.1_953       3 HVO_RS04090  4090  510372  511244       pHV4   0
## 17   lcl|NC_013966.1_cds_WP_004040894.1_993       3 HVO_RS04280  4280  546987  547586       pHV4   0
## 18  lcl|NC_013966.1_cds_WP_004040958.1_1046       3 HVO_RS04545  4545  604291  604902       pHV4   0
## 19  lcl|NC_013966.1_cds_WP_004040984.1_1073       3 HVO_RS04675  4675  629390  629758       pHV4   0
## 232 lcl|NC_013967.1_cds_WP_004045081.1_1135       3 HVO_RS04975  4975   55732   56178 Chromosome   0
## 233 lcl|NC_013967.1_cds_WP_004045083.1_1136       3 HVO_RS04980  4980   56181   57515 Chromosome   1
## 234 lcl|NC_013967.1_cds_WP_004045086.1_1137       3 HVO_RS04985  4985   57512   58591 Chromosome   1
## 235 lcl|NC_013967.1_cds_WP_004045088.1_1138       3 HVO_RS04990  4990   58588   60027 Chromosome   1
## 236 lcl|NC_013967.1_cds_WP_004045089.1_1139       3 HVO_RS04995  4995   60034   61083 Chromosome   1
## 237 lcl|NC_013967.1_cds_WP_004045214.1_1190       3 HVO_RS05250  5250  110903  111055 Chromosome   0
## 238 lcl|NC_013967.1_cds_WP_004045216.1_1191       3 HVO_RS05255  5255  111059  111337 Chromosome   1
## 239 lcl|NC_013967.1_cds_WP_004045218.1_1192       3 HVO_RS05260  5260  111341  112006 Chromosome   1
## 240 lcl|NC_013967.1_cds_WP_004045251.1_1214       3 HVO_RS05375  5375  128123  129109 Chromosome   0
## 241 lcl|NC_013967.1_cds_WP_004045267.1_1230       3 HVO_RS05455  5455  142507  142956 Chromosome   0
## 247 lcl|NC_013967.1_cds_WP_004065604.1_1248       3 HVO_RS05545  5545  155649  155852 Chromosome   0
## 242 lcl|NC_013967.1_cds_WP_004045289.1_1251       3 HVO_RS05560  5560  157928  158671 Chromosome   0
## 243 lcl|NC_013967.1_cds_WP_004045298.1_1258       3 HVO_RS05595  5595  165330  165506 Chromosome   0
## 244 lcl|NC_013967.1_cds_WP_004045310.1_1270       3 HVO_RS05655  5655  176934  177101 Chromosome   0
## 245 lcl|NC_013967.1_cds_WP_004045355.1_1314       3 HVO_RS05875  5875  217441  217899 Chromosome   0
## 222 lcl|NC_013967.1_cds_WP_004044613.1_1380       3 HVO_RS06200  6200  278458  278790 Chromosome   0
## 253 lcl|NC_013967.1_cds_WP_013035358.1_1381       3 HVO_RS06205  6205  278777  280978 Chromosome   1
## 221 lcl|NC_013967.1_cds_WP_004044611.1_1382       3 HVO_RS06210  6210  281153  281410 Chromosome   1
## 220 lcl|NC_013967.1_cds_WP_004044610.1_1383       3 HVO_RS06215  6215  281432  282016 Chromosome   1
## 219 lcl|NC_013967.1_cds_WP_004044609.1_1384       3 HVO_RS06220  6220  282013  283059 Chromosome   1
## 218 lcl|NC_013967.1_cds_WP_004044608.1_1385       3 HVO_RS06225  6225  283056  283376 Chromosome   1
## 217 lcl|NC_013967.1_cds_WP_004044607.1_1386       3 HVO_RS06230  6230  283383  285143 Chromosome   1
## 216 lcl|NC_013967.1_cds_WP_004044606.1_1387       3 HVO_RS06235  6235  285148  286554 Chromosome   1
## 215 lcl|NC_013967.1_cds_WP_004044604.1_1389       3 HVO_RS06245  6245  288859  289566 Chromosome   0
## 214 lcl|NC_013967.1_cds_WP_004044584.1_1405       3 HVO_RS06325  6325  306250  306504 Chromosome   0
## 213 lcl|NC_013967.1_cds_WP_004044570.1_1419       3 HVO_RS06405  6405  321385  321813 Chromosome   0
## 212 lcl|NC_013967.1_cds_WP_004044569.1_1420       3 HVO_RS06410  6410  321816  322430 Chromosome   1
## 211 lcl|NC_013967.1_cds_WP_004044567.1_1422       3 HVO_RS06420  6420  323485  325668 Chromosome   0
## 210 lcl|NC_013967.1_cds_WP_004044564.1_1425       3 HVO_RS06435  6435  327676  328941 Chromosome   0
## 209 lcl|NC_013967.1_cds_WP_004044563.1_1426       3 HVO_RS06440  6440  328944  329252 Chromosome   1
## 208 lcl|NC_013967.1_cds_WP_004044506.1_1479       3 HVO_RS06705  6705  373221  374717 Chromosome   0
## 207 lcl|NC_013967.1_cds_WP_004044500.1_1485       3 HVO_RS06735  6735  380564  382378 Chromosome   0
## 264 lcl|NC_013967.1_cds_WP_013035608.1_1508       3 HVO_RS06850  6850  396573  397619 Chromosome   0
## 206 lcl|NC_013967.1_cds_WP_004044471.1_1513       3 HVO_RS06875  6875  400730  403606 Chromosome   0
## 205 lcl|NC_013967.1_cds_WP_004044456.1_1527       3 HVO_RS06950  6950  415459  416598 Chromosome   0
## 204 lcl|NC_013967.1_cds_WP_004044446.1_1537       3 HVO_RS07000  7000  424661  425317 Chromosome   0
## 203 lcl|NC_013967.1_cds_WP_004044437.1_1546       3 HVO_RS07045  7045  432592  433122 Chromosome   0
## 202 lcl|NC_013967.1_cds_WP_004044372.1_1610       3 HVO_RS07395  7395  495226  495816 Chromosome   0
## 201 lcl|NC_013967.1_cds_WP_004044282.1_1698       3 HVO_RS07845  7845  584352  584621 Chromosome   0
## 200 lcl|NC_013967.1_cds_WP_004044258.1_1720       3 HVO_RS07955  7955  604398  605702 Chromosome   0
## 199 lcl|NC_013967.1_cds_WP_004044243.1_1735       3 HVO_RS08030  8030  622950  623810 Chromosome   0
## 198 lcl|NC_013967.1_cds_WP_004044237.1_1740       3 HVO_RS08055  8055  627962  628153 Chromosome   0
## 197 lcl|NC_013967.1_cds_WP_004044236.1_1741       3 HVO_RS08060  8060  628158  628958 Chromosome   1
## 196 lcl|NC_013967.1_cds_WP_004044235.1_1742       3 HVO_RS08065  8065  628977  629171 Chromosome   1
## 195 lcl|NC_013967.1_cds_WP_004044234.1_1743       3 HVO_RS08070  8070  629175  629456 Chromosome   1
## 194 lcl|NC_013967.1_cds_WP_004044208.1_1768       3 HVO_RS08200  8200  652485  653018 Chromosome   0
## 193 lcl|NC_013967.1_cds_WP_004044155.1_1818       3 HVO_RS08450  8450  701789  702334 Chromosome   0
## 192 lcl|NC_013967.1_cds_WP_004044141.1_1831       3 HVO_RS08515  8515  714561  715568 Chromosome   0
## 191 lcl|NC_013967.1_cds_WP_004044129.1_1842       3 HVO_RS08570  8570  726446  726733 Chromosome   0
## 190 lcl|NC_013967.1_cds_WP_004044109.1_1862       3 HVO_RS08670  8670  746707  746964 Chromosome   0
## 189 lcl|NC_013967.1_cds_WP_004044078.1_1891       3 HVO_RS08815  8815  774651  775556 Chromosome   0
## 188 lcl|NC_013967.1_cds_WP_004044077.1_1892       3 HVO_RS08820  8820  775569  776999 Chromosome   1
## 187 lcl|NC_013967.1_cds_WP_004044067.1_1903       3 HVO_RS08875  8875  789658  791124 Chromosome   0
## 186 lcl|NC_013967.1_cds_WP_004044057.1_1912       3 HVO_RS08920  8920  797576  798499 Chromosome   0
## 185 lcl|NC_013967.1_cds_WP_004044054.1_1915       3 HVO_RS08935  8935  800808  800999 Chromosome   0
## 184 lcl|NC_013967.1_cds_WP_004044050.1_1919       3 HVO_RS08955  8955  803377  805137 Chromosome   0
## 246 lcl|NC_013967.1_cds_WP_004064260.1_1938       3 HVO_RS09050  9050  825916  826275 Chromosome   0
## 183 lcl|NC_013967.1_cds_WP_004044029.1_1940       3 HVO_RS09060  9060  826789  826992 Chromosome   0
## 182 lcl|NC_013967.1_cds_WP_004044028.1_1941       3 HVO_RS09065  9065  827150  828259 Chromosome   1
## 181 lcl|NC_013967.1_cds_WP_004044024.1_1945       3 HVO_RS09085  9085  831303  832214 Chromosome   0
## 180 lcl|NC_013967.1_cds_WP_004043970.1_2000       3 HVO_RS09360  9360  883706  884860 Chromosome   0
## 179 lcl|NC_013967.1_cds_WP_004043964.1_2006       3 HVO_RS09390  9390  889665  890366 Chromosome   0
## 178 lcl|NC_013967.1_cds_WP_004043961.1_2009       3 HVO_RS09405  9405  893082  893543 Chromosome   0
## 177 lcl|NC_013967.1_cds_WP_004043960.1_2010       3 HVO_RS09410  9410  893649  893909 Chromosome   1
## 176 lcl|NC_013967.1_cds_WP_004043959.1_2011       3 HVO_RS09415  9415  893906  894274 Chromosome   1
## 175 lcl|NC_013967.1_cds_WP_004043956.1_2014       3 HVO_RS09430  9430  896613  898142 Chromosome   0
## 174 lcl|NC_013967.1_cds_WP_004043928.1_2041       3 HVO_RS09565  9565  923045  923803 Chromosome   0
## 173 lcl|NC_013967.1_cds_WP_004043866.1_2096       3 HVO_RS09845  9845  979089  979655 Chromosome   0
## 172 lcl|NC_013967.1_cds_WP_004043847.1_2115       3 HVO_RS09940  9940  997547  998305 Chromosome   0
## 171 lcl|NC_013967.1_cds_WP_004043799.1_2161       3 HVO_RS10170 10170 1040246 1041094 Chromosome   0
## 170 lcl|NC_013967.1_cds_WP_004043796.1_2164       3 HVO_RS10185 10185 1043654 1043977 Chromosome   0
## 169 lcl|NC_013967.1_cds_WP_004043795.1_2165       3 HVO_RS10190 10190 1044118 1044393 Chromosome   1
## 268 lcl|NC_013967.1_cds_WP_013035666.1_2168       3 HVO_RS10205 10205 1045827 1046489 Chromosome   0
## 168 lcl|NC_013967.1_cds_WP_004043776.1_2185       3 HVO_RS10300 10300 1063656 1064564 Chromosome   0
## 167 lcl|NC_013967.1_cds_WP_004043775.1_2186       3 HVO_RS10305 10305 1064561 1064809 Chromosome   1
## 166 lcl|NC_013967.1_cds_WP_004043768.1_2193       3 HVO_RS10340 10340 1070008 1070550 Chromosome   0
## 165 lcl|NC_013967.1_cds_WP_004043692.1_2267       3 HVO_RS10710 10710 1139413 1139841 Chromosome   0
## 164 lcl|NC_013967.1_cds_WP_004043616.1_2337       3 HVO_RS11065 11065 1204459 1204770 Chromosome   0
## 163 lcl|NC_013967.1_cds_WP_004043581.1_2372       3 HVO_RS11245 11245 1238438 1238830 Chromosome   0
## 162 lcl|NC_013967.1_cds_WP_004043580.1_2373       3 HVO_RS11250 11250 1238925 1239131 Chromosome   1
## 248 lcl|NC_013967.1_cds_WP_008574128.1_2398       3 HVO_RS11375 11375 1260973 1261293 Chromosome   0
## 161 lcl|NC_013967.1_cds_WP_004043548.1_2404       3 HVO_RS11405 11405 1264940 1265395 Chromosome   0
## 160 lcl|NC_013967.1_cds_WP_004043537.1_2414       3 HVO_RS11455 11455 1276175 1277308 Chromosome   0
## 159 lcl|NC_013967.1_cds_WP_004043485.1_2462       3 HVO_RS11700 11700 1321660 1322130 Chromosome   0
##  [ reached &#39;max&#39; / getOption(&quot;max.print&quot;) -- omitted 164 rows ]
##                                      genes cluster   locus_tag    rs   start    stop        chr op1
## 10 lcl|NC_013967.1_cds_WP_004045288.1_1250       4 HVO_RS05555  5555  157202  157795 Chromosome   0
## 8  lcl|NC_013967.1_cds_WP_004044241.1_1737       4 HVO_RS08040  8040  624993  625706 Chromosome   0
## 7  lcl|NC_013967.1_cds_WP_004044199.1_1776       4 HVO_RS08240  8240  658792  659073 Chromosome   0
## 6  lcl|NC_013967.1_cds_WP_004043792.1_2169       4 HVO_RS10210 10210 1046482 1046724 Chromosome   0
## 5  lcl|NC_013967.1_cds_WP_004043771.1_2190       4 HVO_RS10325 10325 1067603 1068031 Chromosome   0
## 1  lcl|NC_013967.1_cds_WP_004041617.1_2727       4 HVO_RS13040 13040 1603637 1604212 Chromosome   0
## 2  lcl|NC_013967.1_cds_WP_004041740.1_2844       4 HVO_RS13620 13620 1716206 1716661 Chromosome   0
## 3  lcl|NC_013967.1_cds_WP_004041912.1_3007       4 HVO_RS14445 14445 1871717 1872109 Chromosome   0
## 4  lcl|NC_013967.1_cds_WP_004041947.1_3043       4 HVO_RS14615 14615 1911379 1911816 Chromosome   0
## 9  lcl|NC_013967.1_cds_WP_004044900.1_3946       4 HVO_RS19195 19195 2825137 2825526 Chromosome   0
## 11   lcl|NC_013968.1_cds_WP_013035713.1_21       4 HVO_RS19375 19375   16507   16875       pHV1   0
##                                      genes cluster   locus_tag    rs   start    stop        chr op1
## 1   lcl|NC_013964.1_cds_WP_049914695.1_228       5 HVO_RS00695   695  159005  160810       pHV3   0
## 2  lcl|NC_013966.1_cds_WP_004040960.1_1048       5 HVO_RS04555  4555  606528  606842       pHV4   0
## 25 lcl|NC_013967.1_cds_WP_004044521.1_1465       5 HVO_RS06640  6640  357270  357947 Chromosome   0
## 24 lcl|NC_013967.1_cds_WP_004044449.1_1534       5 HVO_RS06985  6985  422618  422818 Chromosome   0
## 23 lcl|NC_013967.1_cds_WP_004044441.1_1542       5 HVO_RS07025  7025  428646  429851 Chromosome   0
## 22 lcl|NC_013967.1_cds_WP_004043822.1_2139       5 HVO_RS10060 10060 1019071 1019271 Chromosome   0
## 21 lcl|NC_013967.1_cds_WP_004043695.1_2264       5 HVO_RS10695 10695 1138037 1138414 Chromosome   0
## 20 lcl|NC_013967.1_cds_WP_004043467.1_2479       5 HVO_RS11785 11785 1338855 1339379 Chromosome   0
## 19 lcl|NC_013967.1_cds_WP_004043443.1_2503       5 HVO_RS11905 11905 1362157 1362600 Chromosome   0
## 5  lcl|NC_013967.1_cds_WP_004041412.1_2540       5 HVO_RS12065 12065 1401868 1402422 Chromosome   0
## 6  lcl|NC_013967.1_cds_WP_004041448.1_2574       5 HVO_RS12235 12235 1437398 1437616 Chromosome   0
## 7  lcl|NC_013967.1_cds_WP_004041553.1_2665       5 HVO_RS12690 12690 1525931 1526119 Chromosome   0
## 8  lcl|NC_013967.1_cds_WP_004041759.1_2861       5 HVO_RS13705 13705 1732943 1733122 Chromosome   0
## 9  lcl|NC_013967.1_cds_WP_004041808.1_2907       5 HVO_RS13945 13945 1777642 1778232 Chromosome   0
## 10 lcl|NC_013967.1_cds_WP_004041856.1_2954       5 HVO_RS14185 14185 1823943 1824173 Chromosome   0
## 11 lcl|NC_013967.1_cds_WP_004042014.1_3109       5 HVO_RS14940 14940 1996962 1997831 Chromosome   0
## 12 lcl|NC_013967.1_cds_WP_004042018.1_3114       5 HVO_RS14965 14965 2001515 2001814 Chromosome   0
## 13 lcl|NC_013967.1_cds_WP_004042039.1_3134       5 HVO_RS15065 15065 2019911 2020816 Chromosome   0
## 14 lcl|NC_013967.1_cds_WP_004042069.1_3156       5 HVO_RS15175 15175 2049729 2050007 Chromosome   0
## 15 lcl|NC_013967.1_cds_WP_004042253.1_3338       5 HVO_RS16080 16080 2229188 2230375 Chromosome   0
## 16 lcl|NC_013967.1_cds_WP_004042512.1_3493       5 HVO_RS16875 16875 2389504 2390523 Chromosome   0
## 17 lcl|NC_013967.1_cds_WP_004042728.1_3590       5 HVO_RS17365 17365 2473880 2474437 Chromosome   0
## 18 lcl|NC_013967.1_cds_WP_004043010.1_3728       5 HVO_RS18055 18055 2601441 2602187 Chromosome   0
## 26 lcl|NC_013967.1_cds_WP_004044720.1_3860       5 HVO_RS18740 18740 2740705 2741232 Chromosome   0
## 27 lcl|NC_013967.1_cds_WP_004044870.1_3934       5 HVO_RS19140 19140 2816245 2816748 Chromosome   0
## 3                 lcl|NC_013967.1_cds_2430       5 HVO_RS20135 20135 1291354 1291661 Chromosome   0
## 4                 lcl|NC_013967.1_cds_3937       5 HVO_RS20320 20320 2818169 2818487 Chromosome   0
##                                     genes cluster   locus_tag    rs   start    stop        chr op1
## 1 lcl|NC_013966.1_cds_WP_004040945.1_1033       6 HVO_RS04480  4480  594041  594484       pHV4   0
## 8 lcl|NC_013967.1_cds_WP_013035304.1_1370       6 HVO_RS06150  6150  267117  268112 Chromosome   0
## 5 lcl|NC_013967.1_cds_WP_004044229.1_1748       6 HVO_RS08100  8100  633967  634392 Chromosome   0
## 9 lcl|NC_013967.1_cds_WP_013035338.1_2940       6 HVO_RS14115 14115 1811649 1813451 Chromosome   0
## 2 lcl|NC_013967.1_cds_WP_004042081.1_3167       6 HVO_RS15230 15230 2058283 2058534 Chromosome   0
## 3 lcl|NC_013967.1_cds_WP_004042242.1_3328       6 HVO_RS16030 16030 2218384 2218572 Chromosome   0
## 4 lcl|NC_013967.1_cds_WP_004042264.1_3349       6 HVO_RS16135 16135 2239456 2240493 Chromosome   0
## 6 lcl|NC_013967.1_cds_WP_004044777.1_3890       6 HVO_RS18900 18900 2765393 2766283 Chromosome   0
## 7 lcl|NC_013967.1_cds_WP_004044843.1_3923       6 HVO_RS19085 19085 2807600 2807809 Chromosome   0
##                                       genes cluster   locus_tag    rs   start    stop        chr op1
## 7    lcl|NC_013964.1_cds_WP_004041340.1_281       7 HVO_RS00955   955  224177  225361       pHV3   0
## 6    lcl|NC_013964.1_cds_WP_004041262.1_360       7 HVO_RS01345  1345  328409  329380       pHV3   0
## 5    lcl|NC_013964.1_cds_WP_004041251.1_371       7 HVO_RS01395  1395  337368  337565       pHV3   0
## 4    lcl|NC_013964.1_cds_WP_004041174.1_442       7 HVO_RS01745  1745  417818  418459       pHV3   0
## 3    lcl|NC_013964.1_cds_WP_004041173.1_443       7 HVO_RS01750  1750  418592  418831       pHV3   1
## 2    lcl|NC_013964.1_cds_WP_004041157.1_457       7 HVO_RS01820  1820  433048  433440       pHV3   0
## 1    lcl|NC_013964.1_cds_WP_004041153.1_461       7 HVO_RS01840  1840  436747  437313       pHV3   0
## 37   lcl|NC_013966.1_cds_WP_049941486.1_471       7 HVO_RS01905  1905    9938   10657       pHV4   0
## 29   lcl|NC_013966.1_cds_WP_013035031.1_477       7 HVO_RS01930  1930   15297   16295       pHV4   0
## 21   lcl|NC_013966.1_cds_WP_004043067.1_599       7 HVO_RS02470  2470  135432  135617       pHV4   0
## 22   lcl|NC_013966.1_cds_WP_004043082.1_614       7 HVO_RS02545  2545  151328  151816       pHV4   0
## 34   lcl|NC_013966.1_cds_WP_013035165.1_617       7 HVO_RS02560  2560  153959  155953       pHV4   0
## 23   lcl|NC_013966.1_cds_WP_004043095.1_628       7 HVO_RS02615  2615  169386  170126       pHV4   0
## 24   lcl|NC_013966.1_cds_WP_004043108.1_641       7 HVO_RS02680  2680  184889  185605       pHV4   0
## 25   lcl|NC_013966.1_cds_WP_004043112.1_647       7 HVO_RS02700  2700  188487  188930       pHV4   0
## 35   lcl|NC_013966.1_cds_WP_013035187.1_664       7 HVO_RS02780  2780  215804  216355       pHV4   0
## 26   lcl|NC_013966.1_cds_WP_004043160.1_696       7 HVO_RS02930  2930  246816  247001       pHV4   0
## 27   lcl|NC_013966.1_cds_WP_004043258.1_798       7 HVO_RS03405  3405  357800  359047       pHV4   0
## 31   lcl|NC_013966.1_cds_WP_013035046.1_816       7 HVO_RS03480  3480  375384  376550       pHV4   0
## 8    lcl|NC_013966.1_cds_WP_004040803.1_894       7 HVO_RS03810  3810  450828  451184       pHV4   0
## 9    lcl|NC_013966.1_cds_WP_004040808.1_902       7 HVO_RS03840  3840  457632  457904       pHV4   0
## 10   lcl|NC_013966.1_cds_WP_004040822.1_920       7 HVO_RS03925  3925  476319  476540       pHV4   0
## 11   lcl|NC_013966.1_cds_WP_004040823.1_921       7 HVO_RS03930  3930  476751  477503       pHV4   1
## 12   lcl|NC_013966.1_cds_WP_004040848.1_946       7 HVO_RS04055  4055  504016  504654       pHV4   0
## 13   lcl|NC_013966.1_cds_WP_004040849.1_947       7 HVO_RS04060  4060  504796  505668       pHV4   1
## 14   lcl|NC_013966.1_cds_WP_004040853.1_951       7 HVO_RS04080  4080  508652  509119       pHV4   0
## 15   lcl|NC_013966.1_cds_WP_004040854.1_952       7 HVO_RS04085  4085  509255  509860       pHV4   1
## 16   lcl|NC_013966.1_cds_WP_004040864.1_962       7 HVO_RS04130  4130  517796  518179       pHV4   0
## 32   lcl|NC_013966.1_cds_WP_013035071.1_984       7 HVO_RS04235  4235  536700  537950       pHV4   0
## 17  lcl|NC_013966.1_cds_WP_004040915.1_1003       7 HVO_RS04330  4330  559953  561179       pHV4   0
## 18  lcl|NC_013966.1_cds_WP_004040943.1_1031       7 HVO_RS04470  4470  591104  592339       pHV4   0
## 36  lcl|NC_013966.1_cds_WP_013035211.1_1049       7 HVO_RS04560  4560  607019  607660       pHV4   0
## 19  lcl|NC_013966.1_cds_WP_004040976.1_1064       7 HVO_RS04635  4635  620140  620598       pHV4   0
## 33  lcl|NC_013966.1_cds_WP_013035098.1_1065       7 HVO_RS04640  4640  620616  621026       pHV4   1
## 20  lcl|NC_013966.1_cds_WP_004040978.1_1066       7 HVO_RS04645  4645  621243  623930       pHV4   1
## 30  lcl|NC_013966.1_cds_WP_013035036.1_1079       7 HVO_RS04705  4705  634653  634937       pHV4   0
## 229 lcl|NC_013967.1_cds_WP_004044961.1_1086       7 HVO_RS04735  4735    6854    7282 Chromosome   0
## 230 lcl|NC_013967.1_cds_WP_004044967.1_1089       7 HVO_RS04750  4750   10248   10907 Chromosome   0
## 231 lcl|NC_013967.1_cds_WP_004045016.1_1106       7 HVO_RS04840  4840   27832   28101 Chromosome   0
## 232 lcl|NC_013967.1_cds_WP_004045032.1_1112       7 HVO_RS04865  4865   32744   33238 Chromosome   0
## 233 lcl|NC_013967.1_cds_WP_004045034.1_1113       7 HVO_RS04870  4870   33689   34879 Chromosome   1
## 234 lcl|NC_013967.1_cds_WP_004045116.1_1149       7 HVO_RS05045  5045   73059   73280 Chromosome   0
## 235 lcl|NC_013967.1_cds_WP_004045139.1_1160       7 HVO_RS05100  5100   83382   83753 Chromosome   0
## 263 lcl|NC_013967.1_cds_WP_049914910.1_1206       7 HVO_RS05335  5335  121969  122181 Chromosome   0
## 236 lcl|NC_013967.1_cds_WP_004045246.1_1209       7 HVO_RS05350  5350  124992  125534 Chromosome   0
## 237 lcl|NC_013967.1_cds_WP_004045280.1_1242       7 HVO_RS05515  5515  151249  151560 Chromosome   0
## 238 lcl|NC_013967.1_cds_WP_004045285.1_1247       7 HVO_RS05540  5540  154782  155429 Chromosome   0
## 239 lcl|NC_013967.1_cds_WP_004045314.1_1274       7 HVO_RS05675  5675  180124  180504 Chromosome   0
## 264 lcl|NC_013967.1_cds_WP_049914914.1_1278       7 HVO_RS05695  5695  182364  184730 Chromosome   0
## 240 lcl|NC_013967.1_cds_WP_004045328.1_1288       7 HVO_RS05745  5745  193386  194348 Chromosome   0
## 241 lcl|NC_013967.1_cds_WP_004045341.1_1301       7 HVO_RS05810  5810  205994  206515 Chromosome   0
## 217 lcl|NC_013967.1_cds_WP_004044638.1_1356       7 HVO_RS06075  6075  256281  256670 Chromosome   0
## 265 lcl|NC_013967.1_cds_WP_049941501.1_1443       7 HVO_RS06530  6530  338106  338621 Chromosome   0
## 216 lcl|NC_013967.1_cds_WP_004044534.1_1452       7 HVO_RS06575  6575  345139  345441 Chromosome   0
## 215 lcl|NC_013967.1_cds_WP_004044525.1_1461       7 HVO_RS06620  6620  354646  354945 Chromosome   0
## 214 lcl|NC_013967.1_cds_WP_004044482.1_1503       7 HVO_RS06825  6825  393182  393352 Chromosome   0
## 213 lcl|NC_013967.1_cds_WP_004044473.1_1511       7 HVO_RS06865  6865  399548  399955 Chromosome   0
## 212 lcl|NC_013967.1_cds_WP_004044470.1_1514       7 HVO_RS06880  6880  403730  403948 Chromosome   0
## 211 lcl|NC_013967.1_cds_WP_004044465.1_1518       7 HVO_RS06900  6900  407615  407797 Chromosome   0
## 210 lcl|NC_013967.1_cds_WP_004044462.1_1521       7 HVO_RS06920  6920  409132  409338 Chromosome   0
## 209 lcl|NC_013967.1_cds_WP_004044461.1_1522       7 HVO_RS06925  6925  409378  410376 Chromosome   1
## 208 lcl|NC_013967.1_cds_WP_004044460.1_1523       7 HVO_RS06930  6930  410369  411793 Chromosome   1
## 207 lcl|NC_013967.1_cds_WP_004044443.1_1540       7 HVO_RS07015  7015  427742  428230 Chromosome   0
## 206 lcl|NC_013967.1_cds_WP_004044434.1_1549       7 HVO_RS07060  7060  434848  435303 Chromosome   0
## 205 lcl|NC_013967.1_cds_WP_004044432.1_1551       7 HVO_RS07070  7070  435901  436059 Chromosome   0
## 204 lcl|NC_013967.1_cds_WP_004044415.1_1566       7 HVO_RS07160  7160  446839  447285 Chromosome   0
## 203 lcl|NC_013967.1_cds_WP_004044404.1_1577       7 HVO_RS07220  7220  454913  455344 Chromosome   0
## 202 lcl|NC_013967.1_cds_WP_004044391.1_1591       7 HVO_RS07300  7300  470089  470577 Chromosome   0
## 201 lcl|NC_013967.1_cds_WP_004044390.1_1592       7 HVO_RS07305  7305  470736  473552 Chromosome   1
## 200 lcl|NC_013967.1_cds_WP_004044389.1_1593       7 HVO_RS07310  7310  473683  474102 Chromosome   1
## 38                 lcl|NC_013967.1_cds_1606       7 HVO_RS07375  7375  492257  493255 Chromosome   0
## 199 lcl|NC_013967.1_cds_WP_004044375.1_1607       7 HVO_RS07380  7380  493539  494351 Chromosome   1
## 198 lcl|NC_013967.1_cds_WP_004044356.1_1626       7 HVO_RS07475  7475  513578  513793 Chromosome   0
## 245 lcl|NC_013967.1_cds_WP_013035353.1_1631       7 HVO_RS07500  7500  517346  517531 Chromosome   0
## 197 lcl|NC_013967.1_cds_WP_004044335.1_1646       7 HVO_RS07580  7580  530034  530474 Chromosome   0
## 196 lcl|NC_013967.1_cds_WP_004044332.1_1649       7 HVO_RS07595  7595  532608  533282 Chromosome   0
## 195 lcl|NC_013967.1_cds_WP_004044313.1_1668       7 HVO_RS07690  7690  552851  554632 Chromosome   0
## 244 lcl|NC_013967.1_cds_WP_013035296.1_1680       7 HVO_RS07750  7750  567596  569050 Chromosome   0
## 194 lcl|NC_013967.1_cds_WP_004044291.1_1690       7 HVO_RS07805  7805  578988  579335 Chromosome   0
## 193 lcl|NC_013967.1_cds_WP_004044281.1_1699       7 HVO_RS07850  7850  584724  585749 Chromosome   0
## 192 lcl|NC_013967.1_cds_WP_004044267.1_1712       7 HVO_RS07915  7915  596686  597696 Chromosome   0
## 191 lcl|NC_013967.1_cds_WP_004044266.1_1713       7 HVO_RS07920  7920  597693  598727 Chromosome   1
## 242 lcl|NC_013967.1_cds_WP_013035285.1_1731       7 HVO_RS08010  8010  617184  617507 Chromosome   0
## 190 lcl|NC_013967.1_cds_WP_004044217.1_1760       7 HVO_RS08160  8160  645525  645827 Chromosome   0
## 257 lcl|NC_013967.1_cds_WP_013035638.1_1772       7 HVO_RS08220  8220  655014  655994 Chromosome   0
## 189 lcl|NC_013967.1_cds_WP_004044195.1_1780       7 HVO_RS08260  8260  662596  663036 Chromosome   0
## 188 lcl|NC_013967.1_cds_WP_004044192.1_1783       7 HVO_RS08275  8275  665970  667052 Chromosome   0
## 187 lcl|NC_013967.1_cds_WP_004044171.1_1803       7 HVO_RS08375  8375  687866  688354 Chromosome   0
## 186 lcl|NC_013967.1_cds_WP_004044163.1_1810       7 HVO_RS08410  8410  694155  695057 Chromosome   0
## 185 lcl|NC_013967.1_cds_WP_004044154.1_1819       7 HVO_RS08455  8455  702504  704588 Chromosome   0
## 184 lcl|NC_013967.1_cds_WP_004044135.1_1837       7 HVO_RS08545  8545  720565  721839 Chromosome   0
## 183 lcl|NC_013967.1_cds_WP_004044134.1_1838       7 HVO_RS08550  8550  721979  722581 Chromosome   1
## 182 lcl|NC_013967.1_cds_WP_004044131.1_1840       7 HVO_RS08560  8560  724281  724529 Chromosome   0
## 252 lcl|NC_013967.1_cds_WP_013035511.1_1849       7 HVO_RS08605  8605  734301  734900 Chromosome   0
## 181 lcl|NC_013967.1_cds_WP_004044104.1_1867       7 HVO_RS08695  8695  751246  752172 Chromosome   0
## 180 lcl|NC_013967.1_cds_WP_004044075.1_1894       7 HVO_RS08830  8830  778355  778846 Chromosome   0
## 179 lcl|NC_013967.1_cds_WP_004044066.1_1904       7 HVO_RS08880  8880  791288  791674 Chromosome   0
## 178 lcl|NC_013967.1_cds_WP_004044061.1_1908       7 HVO_RS08900  8900  795023  795406 Chromosome   0
## 177 lcl|NC_013967.1_cds_WP_004044046.1_1923       7 HVO_RS08975  8975  807803  809728 Chromosome   0
## 176 lcl|NC_013967.1_cds_WP_004044034.1_1935       7 HVO_RS09035  9035  822927  823331 Chromosome   0
## 261 lcl|NC_013967.1_cds_WP_049914886.1_1947       7 HVO_RS09095  9095  833161  833736 Chromosome   0
## 175 lcl|NC_013967.1_cds_WP_004044015.1_1958       7 HVO_RS09150  9150  842405  842875 Chromosome   0
## 174 lcl|NC_013967.1_cds_WP_004044014.1_1959       7 HVO_RS09155  9155  843073  843501 Chromosome   1
## 173 lcl|NC_013967.1_cds_WP_004044013.1_1960       7 HVO_RS09160  9160  843545  843985 Chromosome   1
## 172 lcl|NC_013967.1_cds_WP_004044012.1_1961       7 HVO_RS09165  9165  844064  844780 Chromosome   1
## 171 lcl|NC_013967.1_cds_WP_004044010.1_1963       7 HVO_RS09175  9175  847700  849742 Chromosome   0
## 170 lcl|NC_013967.1_cds_WP_004044009.1_1964       7 HVO_RS09180  9180  849735  850499 Chromosome   1
## 169 lcl|NC_013967.1_cds_WP_004044008.1_1965       7 HVO_RS09185  9185  850537  850977 Chromosome   1
## 168 lcl|NC_013967.1_cds_WP_004044003.1_1970       7 HVO_RS09210  9210  855224  855442 Chromosome   0
## 167 lcl|NC_013967.1_cds_WP_004044002.1_1971       7 HVO_RS09215  9215  855608  855835 Chromosome   1
## 166 lcl|NC_013967.1_cds_WP_004044001.1_1972       7 HVO_RS09220  9220  855836  856375 Chromosome   1
## 165 lcl|NC_013967.1_cds_WP_004043995.1_1978       7 HVO_RS09250  9250  859880  860539 Chromosome   0
## 164 lcl|NC_013967.1_cds_WP_004043979.1_1993       7 HVO_RS09325  9325  872721  873692 Chromosome   0
## 163 lcl|NC_013967.1_cds_WP_004043954.1_2016       7 HVO_RS09440  9440  899702  900082 Chromosome   0
## 162 lcl|NC_013967.1_cds_WP_004043951.1_2019       7 HVO_RS09455  9455  902466  902693 Chromosome   0
## 161 lcl|NC_013967.1_cds_WP_004043945.1_2024       7 HVO_RS09480  9480  907962  908150 Chromosome   0
## 160 lcl|NC_013967.1_cds_WP_004043944.1_2025       7 HVO_RS09485  9485  908202  909338 Chromosome   1
## 159 lcl|NC_013967.1_cds_WP_004043931.1_2038       7 HVO_RS09550  9550  920489  920722 Chromosome   0
## 260 lcl|NC_013967.1_cds_WP_049914880.1_2060       7 HVO_RS09665  9665  943502  945196 Chromosome   0
## 158 lcl|NC_013967.1_cds_WP_004043908.1_2061       7 HVO_RS09670  9670  945296  945793 Chromosome   1
## 157 lcl|NC_013967.1_cds_WP_004043906.1_2063       7 HVO_RS09680  9680  946398  946871 Chromosome   0
## 156 lcl|NC_013967.1_cds_WP_004043856.1_2106       7 HVO_RS09895  9895  986942  987574 Chromosome   0
## 155 lcl|NC_013967.1_cds_WP_004043851.1_2111       7 HVO_RS09920  9920  993047  993472 Chromosome   0
## 253 lcl|NC_013967.1_cds_WP_013035526.1_2143       7 HVO_RS10080 10080 1021329 1021658 Chromosome   0
## 154 lcl|NC_013967.1_cds_WP_004043816.1_2144       7 HVO_RS10085 10085 1021655 1022863 Chromosome   1
##  [ reached &#39;max&#39; / getOption(&quot;max.print&quot;) -- omitted 142 rows ]
##                                       genes cluster   locus_tag    rs   start    stop        chr op1
## 2    lcl|NC_013964.1_cds_WP_004041008.1_233       8 HVO_RS00720   720  165739  167340       pHV3   0
## 1    lcl|NC_013964.1_cds_WP_004041006.1_235       8 HVO_RS00730   730  169317  169745       pHV3   0
## 11   lcl|NC_013966.1_cds_WP_013035025.1_479       8 HVO_RS01935  1935   17092   18090       pHV4   0
## 10   lcl|NC_013966.1_cds_WP_004062178.1_576       8 HVO_RS02385  2385  113348  113815       pHV4   0
## 12   lcl|NC_013966.1_cds_WP_013035079.1_577       8 HVO_RS02390  2390  113796  114104       pHV4   1
## 9    lcl|NC_013966.1_cds_WP_004043293.1_844       8 HVO_RS03610  3610  404316  404828       pHV4   0
## 13   lcl|NC_013966.1_cds_WP_049941497.1_996       8 HVO_RS04295  4295  549629  550117       pHV4   0
## 4   lcl|NC_013966.1_cds_WP_004040946.1_1034       8 HVO_RS04485  4485  594503  594871       pHV4   0
## 5   lcl|NC_013966.1_cds_WP_004040953.1_1041       8 HVO_RS04520  4520  600914  601564       pHV4   0
## 6   lcl|NC_013966.1_cds_WP_004040969.1_1057       8 HVO_RS04600  4600  613711  613917       pHV4   0
## 7   lcl|NC_013966.1_cds_WP_004040986.1_1075       8 HVO_RS04685  4685  630851  631291       pHV4   0
## 128 lcl|NC_013967.1_cds_WP_004044965.1_1088       8 HVO_RS04745  4745    8873   10219 Chromosome   0
## 129 lcl|NC_013967.1_cds_WP_004044970.1_1090       8 HVO_RS04755  4755   10956   11528 Chromosome   0
## 130 lcl|NC_013967.1_cds_WP_004045112.1_1147       8 HVO_RS05035  5035   72161   72538 Chromosome   0
## 131 lcl|NC_013967.1_cds_WP_004045161.1_1167       8 HVO_RS05135  5135   90474   90758 Chromosome   0
## 132 lcl|NC_013967.1_cds_WP_004045195.1_1180       8 HVO_RS05200  5200  103089  104234 Chromosome   0
## 133 lcl|NC_013967.1_cds_WP_004045197.1_1181       8 HVO_RS05205  5205  104245  104658 Chromosome   1
## 134 lcl|NC_013967.1_cds_WP_004045241.1_1204       8 HVO_RS05320  5320  120802  121179 Chromosome   0
## 135 lcl|NC_013967.1_cds_WP_004045277.1_1239       8 HVO_RS05500  5500  148866  149378 Chromosome   0
## 136 lcl|NC_013967.1_cds_WP_004045278.1_1240       8 HVO_RS05505  5505  149375  149722 Chromosome   1
## 137 lcl|NC_013967.1_cds_WP_004045308.1_1268       8 HVO_RS05645  5645  174892  176016 Chromosome   0
## 138 lcl|NC_013967.1_cds_WP_004045324.1_1284       8 HVO_RS05725  5725  191384  191674 Chromosome   0
## 139 lcl|NC_013967.1_cds_WP_004045377.1_1332       8 HVO_RS05965  5965  233436  233702 Chromosome   0
## 123 lcl|NC_013967.1_cds_WP_004044632.1_1361       8 HVO_RS06105  6105  260600  261190 Chromosome   0
## 122 lcl|NC_013967.1_cds_WP_004044631.1_1362       8 HVO_RS06110  6110  261191  262126 Chromosome   1
## 121 lcl|NC_013967.1_cds_WP_004044625.1_1368       8 HVO_RS06140  6140  265809  266291 Chromosome   0
## 120 lcl|NC_013967.1_cds_WP_004044568.1_1421       8 HVO_RS06415  6415  322539  323294 Chromosome   0
## 119 lcl|NC_013967.1_cds_WP_004044543.1_1444       8 HVO_RS06535  6535  339578  339793 Chromosome   0
## 118 lcl|NC_013967.1_cds_WP_004044522.1_1464       8 HVO_RS06635  6635  356734  357192 Chromosome   0
## 117 lcl|NC_013967.1_cds_WP_004044495.1_1490       8 HVO_RS06760  6760  385608  386129 Chromosome   0
## 116 lcl|NC_013967.1_cds_WP_004044481.1_1504       8 HVO_RS06830  6830  393349  393960 Chromosome   0
## 145 lcl|NC_013967.1_cds_WP_013035300.1_1515       8 HVO_RS06885  6885  404006  405001 Chromosome   0
## 115 lcl|NC_013967.1_cds_WP_004044440.1_1543       8 HVO_RS07030  7030  429927  430979 Chromosome   0
## 114 lcl|NC_013967.1_cds_WP_004044433.1_1550       8 HVO_RS07065  7065  435432  435830 Chromosome   0
## 113 lcl|NC_013967.1_cds_WP_004044428.1_1554       8 HVO_RS07085  7085  437774  438166 Chromosome   0
## 141 lcl|NC_013967.1_cds_WP_004057909.1_1557       8 HVO_RS07105  7105  440257  440451 Chromosome   0
## 140 lcl|NC_013967.1_cds_WP_004057907.1_1558       8 HVO_RS07110  7110  440611  440805 Chromosome   1
## 112 lcl|NC_013967.1_cds_WP_004044423.1_1559       8 HVO_RS07120  7120  441371  442039 Chromosome   0
## 111 lcl|NC_013967.1_cds_WP_004044422.1_1560       8 HVO_RS07125  7125  442085  442483 Chromosome   1
## 110 lcl|NC_013967.1_cds_WP_004044399.1_1584       8 HVO_RS07265  7265  463243  464286 Chromosome   0
## 109 lcl|NC_013967.1_cds_WP_004044395.1_1588       8 HVO_RS07285  7285  467702  468232 Chromosome   0
## 108 lcl|NC_013967.1_cds_WP_004044369.1_1613       8 HVO_RS07410  7410  498059  499351 Chromosome   0
## 107 lcl|NC_013967.1_cds_WP_004044368.1_1614       8 HVO_RS07415  7415  499582  500721 Chromosome   1
## 106 lcl|NC_013967.1_cds_WP_004044337.1_1644       8 HVO_RS07570  7570  528489  528956 Chromosome   0
## 105 lcl|NC_013967.1_cds_WP_004044336.1_1645       8 HVO_RS07575  7575  529064  529795 Chromosome   1
## 104 lcl|NC_013967.1_cds_WP_004044207.1_1769       8 HVO_RS08205  8205  653354  653713 Chromosome   0
## 103 lcl|NC_013967.1_cds_WP_004044203.1_1773       8 HVO_RS08225  8225  656079  657341 Chromosome   0
## 151 lcl|NC_013967.1_cds_WP_049941502.1_1800       8 HVO_RS08360  8360  684671  685159 Chromosome   0
## 102 lcl|NC_013967.1_cds_WP_004044167.1_1806       8 HVO_RS08390  8390  690965  691660 Chromosome   0
## 101 lcl|NC_013967.1_cds_WP_004044164.1_1809       8 HVO_RS08405  8405  693320  694084 Chromosome   0
## 100 lcl|NC_013967.1_cds_WP_004044145.1_1827       8 HVO_RS08495  8495  710480  710893 Chromosome   0
## 99  lcl|NC_013967.1_cds_WP_004044136.1_1836       8 HVO_RS08540  8540  719859  720482 Chromosome   0
## 98  lcl|NC_013967.1_cds_WP_004044093.1_1877       8 HVO_RS08745  8745  759259  759648 Chromosome   0
## 97  lcl|NC_013967.1_cds_WP_004044092.1_1878       8 HVO_RS08750  8750  759648  760286 Chromosome   1
## 96  lcl|NC_013967.1_cds_WP_004044080.1_1889       8 HVO_RS08805  8805  770134  770319 Chromosome   0
## 95  lcl|NC_013967.1_cds_WP_004044072.1_1898       8 HVO_RS08850  8850  781408  782166 Chromosome   0
## 94  lcl|NC_013967.1_cds_WP_004044068.1_1902       8 HVO_RS08870  8870  784965  789500 Chromosome   0
## 148 lcl|NC_013967.1_cds_WP_013035477.1_1917       8 HVO_RS08945  8945  802107  802304 Chromosome   0
## 93  lcl|NC_013967.1_cds_WP_004044021.1_1948       8 HVO_RS09100  9100  833824  834033 Chromosome   0
## 92  lcl|NC_013967.1_cds_WP_004043998.1_1975       8 HVO_RS09235  9235  858517  858738 Chromosome   0
## 91  lcl|NC_013967.1_cds_WP_004043971.1_1999       8 HVO_RS09355  9355  882969  883490 Chromosome   0
## 90  lcl|NC_013967.1_cds_WP_004043966.1_2004       8 HVO_RS09380  9380  888441  889052 Chromosome   0
## 89  lcl|NC_013967.1_cds_WP_004043940.1_2029       8 HVO_RS09505  9505  913053  913850 Chromosome   0
## 88  lcl|NC_013967.1_cds_WP_004043939.1_2030       8 HVO_RS09510  9510  913947  914231 Chromosome   1
## 87  lcl|NC_013967.1_cds_WP_004043915.1_2054       8 HVO_RS09630  9630  936539  936811 Chromosome   0
## 144 lcl|NC_013967.1_cds_WP_006600487.1_2076       8 HVO_RS09745  9745  959346  960344 Chromosome   0
## 86  lcl|NC_013967.1_cds_WP_004043858.1_2104       8 HVO_RS09885  9885  985780  986346 Chromosome   0
## 85  lcl|NC_013967.1_cds_WP_004043857.1_2105       8 HVO_RS09890  9890  986514  986858 Chromosome   1
## 84  lcl|NC_013967.1_cds_WP_004043855.1_2107       8 HVO_RS09900  9900  987571  988644 Chromosome   0
## 83  lcl|NC_013967.1_cds_WP_004043843.1_2119       8 HVO_RS09960  9960 1000245 1000799 Chromosome   0
## 82  lcl|NC_013967.1_cds_WP_004043823.1_2138       8 HVO_RS10055 10055 1018346 1019044 Chromosome   0
## 81  lcl|NC_013967.1_cds_WP_004043821.1_2140       8 HVO_RS10065 10065 1019271 1019618 Chromosome   0
## 80  lcl|NC_013967.1_cds_WP_004043818.1_2142       8 HVO_RS10075 10075 1020707 1021228 Chromosome   0
## 79  lcl|NC_013967.1_cds_WP_004043804.1_2156       8 HVO_RS10145 10145 1033423 1033728 Chromosome   0
## 146 lcl|NC_013967.1_cds_WP_013035444.1_2162       8 HVO_RS10175 10175 1041315 1042391 Chromosome   0
## 14                 lcl|NC_013967.1_cds_2173       8 HVO_RS10235 10235 1049584 1050838 Chromosome   0
## 78  lcl|NC_013967.1_cds_WP_004043761.1_2199       8 HVO_RS10370 10370 1074831 1075205 Chromosome   0
## 77  lcl|NC_013967.1_cds_WP_004043760.1_2200       8 HVO_RS10375 10375 1075523 1076554 Chromosome   1
## 76  lcl|NC_013967.1_cds_WP_004043755.1_2205       8 HVO_RS10400 10400 1079366 1079590 Chromosome   0
## 75  lcl|NC_013967.1_cds_WP_004043751.1_2209       8 HVO_RS10420 10420 1082548 1083267 Chromosome   0
## 74  lcl|NC_013967.1_cds_WP_004043688.1_2271       8 HVO_RS10730 10730 1141388 1141789 Chromosome   0
## 73  lcl|NC_013967.1_cds_WP_004043667.1_2287       8 HVO_RS10810 10810 1159897 1160487 Chromosome   0
## 72  lcl|NC_013967.1_cds_WP_004043666.1_2288       8 HVO_RS10815 10815 1160641 1161729 Chromosome   1
## 147 lcl|NC_013967.1_cds_WP_013035467.1_2328       8 HVO_RS11020 11020 1197175 1198020 Chromosome   0
## 71  lcl|NC_013967.1_cds_WP_004043613.1_2340       8 HVO_RS11080 11080 1206212 1206448 Chromosome   0
## 70  lcl|NC_013967.1_cds_WP_004043594.1_2358       8 HVO_RS11175 11175 1222761 1223477 Chromosome   0
## 69  lcl|NC_013967.1_cds_WP_004043579.1_2374       8 HVO_RS11255 11255 1239241 1239621 Chromosome   0
## 68  lcl|NC_013967.1_cds_WP_004043561.1_2391       8 HVO_RS11340 11340 1255474 1255854 Chromosome   0
## 67  lcl|NC_013967.1_cds_WP_004043553.1_2399       8 HVO_RS11380 11380 1261387 1261644 Chromosome   0
## 66  lcl|NC_013967.1_cds_WP_004043519.1_2431       8 HVO_RS11540 11540 1291746 1293083 Chromosome   0
## 65  lcl|NC_013967.1_cds_WP_004043509.1_2439       8 HVO_RS11580 11580 1299309 1300286 Chromosome   0
## 64  lcl|NC_013967.1_cds_WP_004043495.1_2452       8 HVO_RS11650 11650 1311049 1311372 Chromosome   0
## 63  lcl|NC_013967.1_cds_WP_004043476.1_2470       8 HVO_RS11740 11740 1330658 1331374 Chromosome   0
## 62  lcl|NC_013967.1_cds_WP_004043454.1_2492       8 HVO_RS11850 11850 1351477 1351968 Chromosome   0
## 61  lcl|NC_013967.1_cds_WP_004043447.1_2499       8 HVO_RS11885 11885 1358000 1359238 Chromosome   0
## 143 lcl|NC_013967.1_cds_WP_004063281.1_2524       8 HVO_RS12010 12010 1381022 1381348 Chromosome   0
## 17  lcl|NC_013967.1_cds_WP_004041415.1_2543       8 HVO_RS12080 12080 1403626 1404828 Chromosome   0
## 150 lcl|NC_013967.1_cds_WP_013035654.1_2544       8 HVO_RS12085 12085 1405185 1406945 Chromosome   1
## 18  lcl|NC_013967.1_cds_WP_004041434.1_2562       8 HVO_RS12175 12175 1425648 1425884 Chromosome   0
## 19  lcl|NC_013967.1_cds_WP_004041457.1_2583       8 HVO_RS12280 12280 1448985 1449518 Chromosome   0
## 20  lcl|NC_013967.1_cds_WP_004041465.1_2589       8 HVO_RS12310 12310 1454019 1454288 Chromosome   0
## 21  lcl|NC_013967.1_cds_WP_004041528.1_2640       8 HVO_RS12565 12565 1502585 1503061 Chromosome   0
## 22  lcl|NC_013967.1_cds_WP_004041538.1_2650       8 HVO_RS12615 12615 1510223 1510738 Chromosome   0
## 15                 lcl|NC_013967.1_cds_2667       8 HVO_RS12705 12705 1526777 1526979 Chromosome   0
## 23  lcl|NC_013967.1_cds_WP_004041577.1_2688       8 HVO_RS12810 12810 1545551 1545802 Chromosome   0
## 24  lcl|NC_013967.1_cds_WP_004041598.1_2709       8 HVO_RS12920 12920 1573455 1574471 Chromosome   0
## 142 lcl|NC_013967.1_cds_WP_004063115.1_2724       8 HVO_RS13000 13000 1594707 1595912 Chromosome   0
## 25  lcl|NC_013967.1_cds_WP_004041634.1_2745       8 HVO_RS13125 13125 1620826 1623411 Chromosome   0
## 26  lcl|NC_013967.1_cds_WP_004041635.1_2746       8 HVO_RS13130 13130 1623553 1624152 Chromosome   1
## 27  lcl|NC_013967.1_cds_WP_004041712.1_2812       8 HVO_RS13460 13460 1684989 1686278 Chromosome   0
## 28  lcl|NC_013967.1_cds_WP_004041739.1_2843       8 HVO_RS13615 13615 1715738 1716100 Chromosome   0
## 29  lcl|NC_013967.1_cds_WP_004041769.1_2871       8 HVO_RS13765 13765 1741604 1742632 Chromosome   0
## 30  lcl|NC_013967.1_cds_WP_004041802.1_2901       8 HVO_RS13915 13915 1772389 1772886 Chromosome   0
## 31  lcl|NC_013967.1_cds_WP_004041873.1_2970       8 HVO_RS14265 14265 1838678 1838872 Chromosome   0
## 32  lcl|NC_013967.1_cds_WP_004041922.1_3016       8 HVO_RS14495 14495 1882056 1882409 Chromosome   0
## 33  lcl|NC_013967.1_cds_WP_004041950.1_3045       8 HVO_RS14625 14625 1913566 1913850 Chromosome   0
## 34  lcl|NC_013967.1_cds_WP_004041964.1_3061       8 HVO_RS14705 14705 1938116 1941178 Chromosome   0
## 35  lcl|NC_013967.1_cds_WP_004042038.1_3133       8 HVO_RS15060 15060 2018805 2019839 Chromosome   0
## 36  lcl|NC_013967.1_cds_WP_004042042.1_3137       8 HVO_RS15080 15080 2023654 2023944 Chromosome   0
## 37  lcl|NC_013967.1_cds_WP_004042043.1_3138       8 HVO_RS15085 15085 2024395 2024574 Chromosome   1
## 38  lcl|NC_013967.1_cds_WP_004042064.1_3151       8 HVO_RS15150 15150 2042729 2044912 Chromosome   0
## 39  lcl|NC_013967.1_cds_WP_004042068.1_3155       8 HVO_RS15170 15170 2049252 2049452 Chromosome   0
## 149 lcl|NC_013967.1_cds_WP_013035592.1_3183       8 HVO_RS15310 15310 2073204 2074109 Chromosome   0
## 40  lcl|NC_013967.1_cds_WP_004042111.1_3197       8 HVO_RS15380 15380 2086870 2087388 Chromosome   0
## 41  lcl|NC_013967.1_cds_WP_004042123.1_3209       8 HVO_RS15440 15440 2097367 2097768 Chromosome   0
##  [ reached &#39;max&#39; / getOption(&quot;max.print&quot;) -- omitted 32 rows ]
##                                     genes cluster   locus_tag    rs   start    stop        chr op1
## 6 lcl|NC_013967.1_cds_WP_004043662.1_2293       9 HVO_RS10840 10840 1165483 1165683 Chromosome   0
## 5 lcl|NC_013967.1_cds_WP_004043533.1_2418       9 HVO_RS11475 11475 1279645 1280088 Chromosome   0
## 2 lcl|NC_013967.1_cds_WP_004042125.1_3211       9 HVO_RS15450 15450 2098267 2098566 Chromosome   0
## 3 lcl|NC_013967.1_cds_WP_004042149.1_3236       9 HVO_RS15575 15575 2119911 2122316 Chromosome   0
## 4 lcl|NC_013967.1_cds_WP_004042680.1_3567       9 HVO_RS17250 17250 2448882 2449583 Chromosome   0
## 7 lcl|NC_013967.1_cds_WP_004044740.1_3872       9 HVO_RS18800 18800 2751153 2751443 Chromosome   0
## 9   lcl|NC_013968.1_cds_WP_013035728.1_27       9 HVO_RS19400 19400   18908   19882       pHV1   0
## 8   lcl|NC_013968.1_cds_WP_013035727.1_30       9 HVO_RS19405 19405   20884   21090       pHV1   1
## 1  lcl|NC_013966.1_cds_WP_013035007.1_760       9 HVO_RS19885 19885  322805  323104       pHV4   0
##                                      genes cluster   locus_tag    rs   start    stop        chr op1
## 1   lcl|NC_013964.1_cds_WP_004041037.1_204      10 HVO_RS00575   575  133644  134408       pHV3   0
## 2   lcl|NC_013964.1_cds_WP_004041248.1_375      10 HVO_RS01410  1410  339767  340444       pHV3   0
## 8   lcl|NC_013966.1_cds_WP_013035014.1_659      10 HVO_RS02755  2755  207954  208772       pHV4   0
## 6   lcl|NC_013966.1_cds_WP_004043255.1_795      10 HVO_RS03390  3390  354529  355344       pHV4   0
## 3   lcl|NC_013966.1_cds_WP_004040797.1_888      10 HVO_RS03785  3785  445596  445964       pHV4   0
## 4  lcl|NC_013966.1_cds_WP_004040948.1_1036      10 HVO_RS04495  4495  595489  596250       pHV4   0
## 5  lcl|NC_013966.1_cds_WP_004040950.1_1038      10 HVO_RS04505  4505  596686  597246       pHV4   0
## 10                lcl|NC_013967.1_cds_1165      10 HVO_RS05125  5125   88714   89226 Chromosome   0
## 43 lcl|NC_013967.1_cds_WP_004045170.1_1169      10 HVO_RS05145  5145   92298   92993 Chromosome   0
## 44 lcl|NC_013967.1_cds_WP_004045172.1_1170      10 HVO_RS05150  5150   93097   94029 Chromosome   1
## 45 lcl|NC_013967.1_cds_WP_004045179.1_1173      10 HVO_RS05165  5165   95441   97351 Chromosome   0
## 46 lcl|NC_013967.1_cds_WP_004045182.1_1174      10 HVO_RS05170  5170   97756   97950 Chromosome   1
## 47 lcl|NC_013967.1_cds_WP_004045323.1_1283      10 HVO_RS05720  5720  190112  191275 Chromosome   0
## 39 lcl|NC_013967.1_cds_WP_004044400.1_1583      10 HVO_RS07260  7260  462054  463211 Chromosome   0
## 38 lcl|NC_013967.1_cds_WP_004044398.1_1585      10 HVO_RS07270  7270  464283  465227 Chromosome   0
## 37 lcl|NC_013967.1_cds_WP_004044397.1_1586      10 HVO_RS07275  7275  465348  466499 Chromosome   1
## 36 lcl|NC_013967.1_cds_WP_004044168.1_1805      10 HVO_RS08385  8385  690496  690900 Chromosome   0
## 35 lcl|NC_013967.1_cds_WP_004044165.1_1808      10 HVO_RS08400  8400  692660  692932 Chromosome   0
## 34 lcl|NC_013967.1_cds_WP_004044100.1_1870      10 HVO_RS08710  8710  754108  755415 Chromosome   0
## 49 lcl|NC_013967.1_cds_WP_013035590.1_1979      10 HVO_RS09255  9255  860652  861653 Chromosome   0
## 33 lcl|NC_013967.1_cds_WP_004043710.1_2249      10 HVO_RS10620 10620 1123534 1124115 Chromosome   0
## 32 lcl|NC_013967.1_cds_WP_004043636.1_2317      10 HVO_RS10965 10965 1184666 1186432 Chromosome   0
## 31 lcl|NC_013967.1_cds_WP_004043588.1_2364      10 HVO_RS11205 11205 1228716 1229099 Chromosome   0
## 30 lcl|NC_013967.1_cds_WP_004043570.1_2382      10 HVO_RS11295 11295 1245842 1246504 Chromosome   0
## 13 lcl|NC_013967.1_cds_WP_004041440.1_2567      10 HVO_RS12200 12200 1430347 1431078 Chromosome   0
## 14 lcl|NC_013967.1_cds_WP_004041476.1_2594      10 HVO_RS12335 12335 1459300 1459620 Chromosome   0
## 15 lcl|NC_013967.1_cds_WP_004041578.1_2689      10 HVO_RS12815 12815 1545803 1546033 Chromosome   0
## 16 lcl|NC_013967.1_cds_WP_004041610.1_2720      10 HVO_RS12975 12975 1589274 1589552 Chromosome   0
## 17 lcl|NC_013967.1_cds_WP_004041616.1_2726      10 HVO_RS13010 13010 1596955 1597515 Chromosome   0
## 18 lcl|NC_013967.1_cds_WP_004041751.1_2854      10 HVO_RS13670 13670 1725397 1725873 Chromosome   0
## 19 lcl|NC_013967.1_cds_WP_004041766.1_2868      10 HVO_RS13750 13750 1739055 1739708 Chromosome   0
## 20 lcl|NC_013967.1_cds_WP_004041850.1_2948      10 HVO_RS14155 14155 1818629 1819327 Chromosome   0
## 21 lcl|NC_013967.1_cds_WP_004041949.1_3044      10 HVO_RS14620 14620 1912946 1913362 Chromosome   0
## 22 lcl|NC_013967.1_cds_WP_004042482.1_3478      10 HVO_RS16795 16795 2373145 2373558 Chromosome   0
## 23 lcl|NC_013967.1_cds_WP_004042483.1_3479      10 HVO_RS16800 16800 2373656 2374738 Chromosome   1
## 24 lcl|NC_013967.1_cds_WP_004042741.1_3596      10 HVO_RS17395 17395 2479527 2480363 Chromosome   0
## 25 lcl|NC_013967.1_cds_WP_004042762.1_3607      10 HVO_RS17450 17450 2490473 2490871 Chromosome   0
## 26 lcl|NC_013967.1_cds_WP_004042859.1_3653      10 HVO_RS17680 17680 2534459 2535520 Chromosome   0
## 27 lcl|NC_013967.1_cds_WP_004042883.1_3663      10 HVO_RS17730 17730 2545169 2546164 Chromosome   0
## 28 lcl|NC_013967.1_cds_WP_004042895.1_3669      10 HVO_RS17760 17760 2551488 2551994 Chromosome   0
## 29 lcl|NC_013967.1_cds_WP_004042946.1_3693      10 HVO_RS17880 17880 2573663 2574142 Chromosome   0
## 40 lcl|NC_013967.1_cds_WP_004044667.1_3809      10 HVO_RS18480 18480 2688332 2690404 Chromosome   0
## 41 lcl|NC_013967.1_cds_WP_004044669.1_3811      10 HVO_RS18490 18490 2691343 2691792 Chromosome   0
## 48 lcl|NC_013967.1_cds_WP_013035541.1_3874      10 HVO_RS18810 18810 2752331 2752909 Chromosome   0
## 42 lcl|NC_013967.1_cds_WP_004044838.1_3920      10 HVO_RS19070 19070 2806048 2806419 Chromosome   0
## 9   lcl|NC_013966.1_cds_WP_081442865.1_478      10 HVO_RS19730 19730   16538   16759       pHV4   0
## 7   lcl|NC_013966.1_cds_WP_013035006.1_799      10 HVO_RS19910 19910  359044  359226       pHV4   0
## 11                lcl|NC_013967.1_cds_2990      10 HVO_RS20225 20225 1857883 1858141 Chromosome   0
## 12                lcl|NC_013967.1_cds_3193      10 HVO_RS20255 20255 2081627 2081925 Chromosome   0
##                                     genes cluster   locus_tag    rs   start    stop        chr op1
## 1  lcl|NC_013966.1_cds_WP_013035130.1_927      11 HVO_RS03960  3960  481492  482094       pHV4   0
## 3 lcl|NC_013967.1_cds_WP_004044285.1_1695      11 HVO_RS07830  7830  583485  583868 Chromosome   0
## 2 lcl|NC_013967.1_cds_WP_004043566.1_2386      11 HVO_RS11315 11315 1249714 1250865 Chromosome   0
## 4 lcl|NC_013967.1_cds_WP_004044657.1_3799      11 HVO_RS18430 18430 2675352 2675705 Chromosome   0
##                                      genes cluster   locus_tag    rs   start    stop        chr op1
## 1   lcl|NC_013964.1_cds_WP_004041074.1_167      12 HVO_RS00390   390   90597   91628       pHV3   0
## 6   lcl|NC_013964.1_cds_WP_004041335.1_286      12 HVO_RS00980   980  230208  230630       pHV3   0
## 5   lcl|NC_013964.1_cds_WP_004041311.1_311      12 HVO_RS01105  1105  264338  264748       pHV3   0
## 4   lcl|NC_013964.1_cds_WP_004041272.1_350      12 HVO_RS01295  1295  317826  319133       pHV3   0
## 3   lcl|NC_013964.1_cds_WP_004041162.1_453      12 HVO_RS01800  1800  429092  429286       pHV3   0
## 2   lcl|NC_013964.1_cds_WP_004041160.1_454      12 HVO_RS01805  1805  429920  430657       pHV3   1
## 30  lcl|NC_013966.1_cds_WP_013035119.1_468      12 HVO_RS01895  1895    6551    7903       pHV4   0
## 24  lcl|NC_013966.1_cds_WP_004043399.1_503      12 HVO_RS02050  2050   35783   36763       pHV4   0
## 23  lcl|NC_013966.1_cds_WP_004043397.1_506      12 HVO_RS02060  2060   37548   38588       pHV4   0
## 22  lcl|NC_013966.1_cds_WP_004043393.1_510      12 HVO_RS02080  2080   41599   42171       pHV4   0
## 21  lcl|NC_013966.1_cds_WP_004043348.1_558      12 HVO_RS02300  2300   96278   97018       pHV4   0
## 25  lcl|NC_013966.1_cds_WP_004062180.1_580      12 HVO_RS02405  2405  116872  117156       pHV4   0
## 31  lcl|NC_013966.1_cds_WP_013035153.1_581      12 HVO_RS02410  2410  117274  117957       pHV4   1
## 15  lcl|NC_013966.1_cds_WP_004043113.1_648      12 HVO_RS02705  2705  189115  189552       pHV4   0
## 16  lcl|NC_013966.1_cds_WP_004043127.1_661      12 HVO_RS02765  2765  210946  211968       pHV4   0
## 27  lcl|NC_013966.1_cds_WP_013035011.1_706      12 HVO_RS02975  2975  254889  256274       pHV4   0
## 28  lcl|NC_013966.1_cds_WP_013035061.1_707      12 HVO_RS02980  2980  256788  257111       pHV4   1
## 17  lcl|NC_013966.1_cds_WP_004043181.1_716      12 HVO_RS03020  3020  270416  271183       pHV4   0
## 18  lcl|NC_013966.1_cds_WP_004043186.1_721      12 HVO_RS03045  3045  277185  278621       pHV4   0
## 29  lcl|NC_013966.1_cds_WP_013035085.1_722      12 HVO_RS03050  3050  278689  279132       pHV4   1
## 33  lcl|NC_013966.1_cds_WP_049914848.1_801      12 HVO_RS03415  3415  361252  361443       pHV4   0
## 34  lcl|NC_013966.1_cds_WP_049914854.1_820      12 HVO_RS03495  3495  379569  379853       pHV4   0
## 19  lcl|NC_013966.1_cds_WP_004043273.1_822      12 HVO_RS03505  3505  380521  381015       pHV4   0
## 20  lcl|NC_013966.1_cds_WP_004043277.1_827      12 HVO_RS03525  3525  385336  386439       pHV4   0
## 26  lcl|NC_013966.1_cds_WP_004062180.1_865      12 HVO_RS03700  3700  421628  421912       pHV4   0
## 11  lcl|NC_013966.1_cds_WP_004040826.1_924      12 HVO_RS03945  3945  479387  479971       pHV4   0
## 12  lcl|NC_013966.1_cds_WP_004040832.1_929      12 HVO_RS03970  3970  483383  484573       pHV4   0
## 13  lcl|NC_013966.1_cds_WP_004040888.1_987      12 HVO_RS04250  4250  541791  543008       pHV4   0
## 14 lcl|NC_013966.1_cds_WP_004040985.1_1074      12 HVO_RS04680  4680  630181  630612       pHV4   0
## 62 lcl|NC_013967.1_cds_WP_004045008.1_1103      12 HVO_RS04825  4825   24438   25298 Chromosome   0
## 63 lcl|NC_013967.1_cds_WP_004045051.1_1123      12 HVO_RS04920  4920   44389   45432 Chromosome   0
## 64 lcl|NC_013967.1_cds_WP_004045055.1_1124      12 HVO_RS04925  4925   45429   46304 Chromosome   1
## 65 lcl|NC_013967.1_cds_WP_004045057.1_1125      12 HVO_RS04930  4930   46306   46485 Chromosome   1
## 66 lcl|NC_013967.1_cds_WP_004045062.1_1128      12 HVO_RS04940  4940   48274   49491 Chromosome   0
## 67 lcl|NC_013967.1_cds_WP_004045306.1_1266      12 HVO_RS05635  5635  174066  174335 Chromosome   0
## 68 lcl|NC_013967.1_cds_WP_004045307.1_1267      12 HVO_RS05640  5640  174329  174736 Chromosome   1
## 61 lcl|NC_013967.1_cds_WP_004044594.1_1397      12 HVO_RS06285  6285  298435  299373 Chromosome   0
## 60 lcl|NC_013967.1_cds_WP_004044541.1_1446      12 HVO_RS06545  6545  340199  340432 Chromosome   0
## 59 lcl|NC_013967.1_cds_WP_004044487.1_1498      12 HVO_RS06800  6800  391096  391260 Chromosome   0
## 58 lcl|NC_013967.1_cds_WP_004044464.1_1519      12 HVO_RS06910  6910  408453  408659 Chromosome   0
## 57 lcl|NC_013967.1_cds_WP_004043886.1_2079      12 HVO_RS09760  9760  962244  962924 Chromosome   0
## 56 lcl|NC_013967.1_cds_WP_004043803.1_2157      12 HVO_RS10150 10150 1033994 1036267 Chromosome   0
## 55 lcl|NC_013967.1_cds_WP_004043648.1_2306      12 HVO_RS10905 10905 1175604 1176335 Chromosome   0
## 54 lcl|NC_013967.1_cds_WP_004043606.1_2346      12 HVO_RS11115 11115 1211427 1211816 Chromosome   0
## 38 lcl|NC_013967.1_cds_WP_004041407.1_2536      12 HVO_RS12045 12045 1394676 1395617 Chromosome   0
## 39 lcl|NC_013967.1_cds_WP_004041539.1_2651      12 HVO_RS12620 12620 1510853 1511410 Chromosome   0
## 70 lcl|NC_013967.1_cds_WP_013035645.1_2675      12 HVO_RS12745 12745 1532081 1532731 Chromosome   0
## 40 lcl|NC_013967.1_cds_WP_004041622.1_2733      12 HVO_RS13065 13065 1610100 1610468 Chromosome   0
## 41 lcl|NC_013967.1_cds_WP_004041624.1_2735      12 HVO_RS13075 13075 1611242 1611874 Chromosome   0
## 71 lcl|NC_013967.1_cds_WP_049914773.1_2826      12 HVO_RS13530 13530 1698524 1699651 Chromosome   0
## 42 lcl|NC_013967.1_cds_WP_004041730.1_2834      12 HVO_RS13570 13570 1706585 1708384 Chromosome   0
## 43 lcl|NC_013967.1_cds_WP_004041799.1_2898      12 HVO_RS13900 13900 1768729 1770297 Chromosome   0
## 44 lcl|NC_013967.1_cds_WP_004041905.1_3000      12 HVO_RS14410 14410 1865741 1866544 Chromosome   0
## 45 lcl|NC_013967.1_cds_WP_004041934.1_3028      12 HVO_RS14555 14555 1894729 1895808 Chromosome   0
## 47 lcl|NC_013967.1_cds_WP_004042153.1_3240      12 HVO_RS15595 15595 2124744 2125901 Chromosome   0
## 48 lcl|NC_013967.1_cds_WP_004042159.1_3245      12 HVO_RS15620 15620 2131833 2132333 Chromosome   0
## 49 lcl|NC_013967.1_cds_WP_004042298.1_3382      12 HVO_RS16300 16300 2276106 2276522 Chromosome   0
## 50 lcl|NC_013967.1_cds_WP_004042381.1_3436      12 HVO_RS16580 16580 2332664 2333533 Chromosome   0
## 51 lcl|NC_013967.1_cds_WP_004042383.1_3437      12 HVO_RS16585 16585 2333530 2334675 Chromosome   1
## 52 lcl|NC_013967.1_cds_WP_004042699.1_3575      12 HVO_RS17290 17290 2456395 2457615 Chromosome   0
## 53 lcl|NC_013967.1_cds_WP_004043049.1_3768      12 HVO_RS18275 18275 2637088 2637306 Chromosome   0
## 78   lcl|NC_013968.1_cds_WP_049941515.1_17      12 HVO_RS19360 19360   12637   13536       pHV1   0
## 74   lcl|NC_013968.1_cds_WP_013035699.1_37      12 HVO_RS19435 19435   32946   34112       pHV1   0
## 75   lcl|NC_013968.1_cds_WP_013035708.1_43      12 HVO_RS19465 19465   45606   46760       pHV1   0
## 76   lcl|NC_013968.1_cds_WP_013035723.1_44      12 HVO_RS19470 19470   47163   47717       pHV1   1
## 77   lcl|NC_013968.1_cds_WP_013035729.1_61      12 HVO_RS19545 19545   63753   64943       pHV1   0
## 7                  lcl|NC_013966.1_cds_481      12 HVO_RS19735 19735   20569   20822       pHV4   0
## 8                  lcl|NC_013966.1_cds_710      12 HVO_RS19870 19870  260771  261226       pHV4   0
## 32  lcl|NC_013966.1_cds_WP_013035175.1_718      12 HVO_RS19875 19875  272548  273888       pHV4   1
## 9                  lcl|NC_013966.1_cds_853      12 HVO_RS19970 19970  410435  411435       pHV4   0
## 10                 lcl|NC_013966.1_cds_885      12 HVO_RS20040 20040  442928  443446       pHV4   0
## 35  lcl|NC_013966.1_cds_WP_081442874.1_954      12 HVO_RS20065 20065  511352  511837       pHV4   0
## 36 lcl|NC_013967.1_cds_WP_004041400.1_2529      12 HVO_RS20150 20150 1386375 1387490 Chromosome   0
## 37 lcl|NC_013967.1_cds_WP_004041401.1_2530      12 HVO_RS20155 20155 1387487 1388401 Chromosome   1
## 46 lcl|NC_013967.1_cds_WP_004041939.1_3035      12 HVO_RS20240 20240 1903778 1905196 Chromosome   0
## 69 lcl|NC_013967.1_cds_WP_013035494.1_3244      12 HVO_RS20265 20265 2130371 2131603 Chromosome   0
## 73   lcl|NC_013968.1_cds_WP_013035683.1_39      12 HVO_RS20390 20390   35917   38424       pHV1   0
## 72                  lcl|NC_013968.1_cds_72      12 HVO_RS20415 20415   79142   79477       pHV1   0
##                                      genes cluster   locus_tag    rs   start    stop        chr op1
## 4   lcl|NC_013964.1_cds_WP_004041099.1_142      13 HVO_RS00265   265   67069   67746       pHV3   0
## 3   lcl|NC_013964.1_cds_WP_004041092.1_149      13 HVO_RS00300   300   72431   72985       pHV3   0
## 2   lcl|NC_013964.1_cds_WP_004041067.1_174      13 HVO_RS00425   425  100122  100340       pHV3   0
## 1   lcl|NC_013964.1_cds_WP_004041061.1_180      13 HVO_RS00455   455  106769  106990       pHV3   0
## 5   lcl|NC_013964.1_cds_WP_004041212.1_410      13 HVO_RS01585  1585  384012  384230       pHV3   0
## 6   lcl|NC_013966.1_cds_WP_013035200.1_554      13 HVO_RS02280  2280   91575   91754       pHV4   0
## 13 lcl|NC_013967.1_cds_WP_004044350.1_1632      13 HVO_RS07505  7505  517744  518328 Chromosome   0
## 12 lcl|NC_013967.1_cds_WP_004044328.1_1653      13 HVO_RS07615  7615  536843  537532 Chromosome   0
## 14 lcl|NC_013967.1_cds_WP_013035267.1_2246      13 HVO_RS10605 10605 1118978 1121443 Chromosome   0
## 8  lcl|NC_013967.1_cds_WP_004041910.1_3005      13 HVO_RS14435 14435 1870247 1871404 Chromosome   0
## 9  lcl|NC_013967.1_cds_WP_004042010.1_3105      13 HVO_RS14920 14920 1990314 1992065 Chromosome   0
## 10 lcl|NC_013967.1_cds_WP_004042011.1_3106      13 HVO_RS14925 14925 1992248 1993510 Chromosome   1
## 11 lcl|NC_013967.1_cds_WP_004042958.1_3700      13 HVO_RS17915 17915 2580464 2580655 Chromosome   0
## 17    lcl|NC_013968.1_cds_WP_013035701.1_8      13 HVO_RS19320 19320    5743    6006       pHV1   0
## 18   lcl|NC_013968.1_cds_WP_013035734.1_40      13 HVO_RS19450 19450   38529   42353       pHV1   0
## 16   lcl|NC_013968.1_cds_WP_004045462.1_67      13 HVO_RS19575 19575   71227   72192       pHV1   0
## 7  lcl|NC_013967.1_cds_WP_004041723.1_2825      13 HVO_RS20200 20200 1698203 1698400 Chromosome   0
## 15 lcl|NC_013967.1_cds_WP_081442892.1_3412      13 HVO_RS20285 20285 2308465 2308644 Chromosome   0
##                                     genes cluster   locus_tag    rs   start    stop        chr op1
## 1  lcl|NC_013966.1_cds_WP_004043151.1_685      14 HVO_RS02880  2880  236463  236645       pHV4   0
## 9 lcl|NC_013967.1_cds_WP_004065787.1_1144      14 HVO_RS05020  5020   69249   69536 Chromosome   0
## 8 lcl|NC_013967.1_cds_WP_004043918.1_2051      14 HVO_RS09615  9615  933893  934270 Chromosome   0
## 7 lcl|NC_013967.1_cds_WP_004043820.1_2141      14 HVO_RS10070 10070 1019803 1020444 Chromosome   0
## 2 lcl|NC_013967.1_cds_WP_004041703.1_2808      14 HVO_RS13440 13440 1682896 1683264 Chromosome   0
## 3 lcl|NC_013967.1_cds_WP_004042100.1_3186      14 HVO_RS15325 15325 2075199 2075519 Chromosome   0
## 4 lcl|NC_013967.1_cds_WP_004042117.1_3203      14 HVO_RS15410 15410 2092375 2093244 Chromosome   0
## 5 lcl|NC_013967.1_cds_WP_004042122.1_3208      14 HVO_RS15435 15435 2096887 2097303 Chromosome   0
## 6 lcl|NC_013967.1_cds_WP_004042869.1_3656      14 HVO_RS17695 17695 2537673 2537876 Chromosome   0
##                                      genes cluster   locus_tag    rs   start    stop        chr op1
## 12 lcl|NC_013967.1_cds_WP_004045247.1_1210      15 HVO_RS05355  5355  125628  126500 Chromosome   0
## 15 lcl|NC_013967.1_cds_WP_013035596.1_1394      15 HVO_RS06270  6270  296025  296738 Chromosome   0
## 11 lcl|NC_013967.1_cds_WP_004044491.1_1494      15 HVO_RS06780  6780  387864  388532 Chromosome   0
## 10 lcl|NC_013967.1_cds_WP_004043997.1_1976      15 HVO_RS09240  9240  858873  859304 Chromosome   0
## 9  lcl|NC_013967.1_cds_WP_004043996.1_1977      15 HVO_RS09245  9245  859328  859729 Chromosome   1
## 8  lcl|NC_013967.1_cds_WP_004043898.1_2069      15 HVO_RS09710  9710  951364  951924 Chromosome   0
## 7  lcl|NC_013967.1_cds_WP_004043700.1_2259      15 HVO_RS10670 10670 1133594 1134265 Chromosome   0
## 6  lcl|NC_013967.1_cds_WP_004043598.1_2354      15 HVO_RS11155 11155 1219836 1220945 Chromosome   0
## 5  lcl|NC_013967.1_cds_WP_004043457.1_2489      15 HVO_RS11835 11835 1348391 1349344 Chromosome   0
## 1  lcl|NC_013967.1_cds_WP_004041566.1_2677      15 HVO_RS12755 12755 1534149 1535147 Chromosome   0
## 2  lcl|NC_013967.1_cds_WP_004041921.1_3015      15 HVO_RS14490 14490 1881529 1881720 Chromosome   0
## 14 lcl|NC_013967.1_cds_WP_013035228.1_3318      15 HVO_RS15980 15980 2209251 2209853 Chromosome   0
## 3  lcl|NC_013967.1_cds_WP_004042279.1_3364      15 HVO_RS16210 16210 2261527 2261898 Chromosome   0
## 4  lcl|NC_013967.1_cds_WP_004042655.1_3554      15 HVO_RS17185 17185 2437133 2437477 Chromosome   0
## 13 lcl|NC_013967.1_cds_WP_004058105.1_1468      15 HVO_RS20115 20115  361088  361420 Chromosome   0
##                                       genes cluster   locus_tag    rs   start    stop        chr op1
## 9    lcl|NC_013964.1_cds_WP_004041114.1_127      16 HVO_RS00190   190   44953   45708       pHV3   0
## 8    lcl|NC_013964.1_cds_WP_004041091.1_150      16 HVO_RS00305   305   73155   73340       pHV3   0
## 7    lcl|NC_013964.1_cds_WP_004041090.1_151      16 HVO_RS00310   310   73343   74179       pHV3   1
## 6    lcl|NC_013964.1_cds_WP_004041088.1_153      16 HVO_RS00320   320   74645   75139       pHV3   0
## 5    lcl|NC_013964.1_cds_WP_004041079.1_162      16 HVO_RS00365   365   84904   86448       pHV3   0
## 4    lcl|NC_013964.1_cds_WP_004041078.1_163      16 HVO_RS00370   370   86441   87532       pHV3   1
## 3    lcl|NC_013964.1_cds_WP_004041073.1_168      16 HVO_RS00395   395   91625   93346       pHV3   0
## 2    lcl|NC_013964.1_cds_WP_004041071.1_170      16 HVO_RS00405   405   95047   95463       pHV3   0
## 1    lcl|NC_013964.1_cds_WP_004041012.1_229      16 HVO_RS00700   700  161336  162310       pHV3   0
## 14   lcl|NC_013964.1_cds_WP_004041376.1_245      16 HVO_RS00780   780  185436  186071       pHV3   0
## 13   lcl|NC_013964.1_cds_WP_004041336.1_285      16 HVO_RS00975   975  227816  229012       pHV3   0
## 15   lcl|NC_013964.1_cds_WP_049914744.1_298      16 HVO_RS01040  1040  245522  246322       pHV3   0
## 12   lcl|NC_013964.1_cds_WP_004041301.1_322      16 HVO_RS01155  1155  279171  279377       pHV3   0
## 11   lcl|NC_013964.1_cds_WP_004041242.1_381      16 HVO_RS01440  1440  345549  345734       pHV3   0
## 10   lcl|NC_013964.1_cds_WP_004041159.1_455      16 HVO_RS01810  1810  430739  431749       pHV3   0
## 22   lcl|NC_013966.1_cds_WP_004040992.1_463      16 HVO_RS01875  1875    2008    2586       pHV4   0
## 53   lcl|NC_013966.1_cds_WP_049941487.1_483      16 HVO_RS01950  1950   21954   22307       pHV4   0
## 39   lcl|NC_013966.1_cds_WP_004043390.1_512      16 HVO_RS02090  2090   43402   43689       pHV4   0
## 38   lcl|NC_013966.1_cds_WP_004043357.1_548      16 HVO_RS02250  2250   83529   83906       pHV4   0
## 41   lcl|NC_013966.1_cds_WP_013035027.1_551      16 HVO_RS02265  2265   86868   87260       pHV4   0
## 37   lcl|NC_013966.1_cds_WP_004043353.1_552      16 HVO_RS02270  2270   87519   88388       pHV4   1
## 52   lcl|NC_013966.1_cds_WP_013035227.1_560      16 HVO_RS02310  2310   98970   99812       pHV4   0
## 54   lcl|NC_013966.1_cds_WP_049941494.1_582      16 HVO_RS02415  2415  118039  118692       pHV4   0
## 50   lcl|NC_013966.1_cds_WP_013035173.1_583      16 HVO_RS02420  2420  118888  119529       pHV4   1
## 26   lcl|NC_013966.1_cds_WP_004043063.1_590      16 HVO_RS02450  2450  127679  127990       pHV4   0
## 27   lcl|NC_013966.1_cds_WP_004043080.1_611      16 HVO_RS02530  2530  148020  149084       pHV4   0
## 28   lcl|NC_013966.1_cds_WP_004043086.1_618      16 HVO_RS02565  2565  156152  157918       pHV4   0
## 56   lcl|NC_013966.1_cds_WP_081442876.1_649      16 HVO_RS02710  2710  189549  189806       pHV4   0
## 29   lcl|NC_013966.1_cds_WP_004043189.1_724      16 HVO_RS03060  3060  280224  281027       pHV4   0
## 30   lcl|NC_013966.1_cds_WP_004043191.1_726      16 HVO_RS03070  3070  282252  283844       pHV4   0
## 31   lcl|NC_013966.1_cds_WP_004043204.1_740      16 HVO_RS03135  3135  298349  298723       pHV4   0
## 48   lcl|NC_013966.1_cds_WP_013035125.1_779      16 HVO_RS03315  3315  336439  338493       pHV4   0
## 46   lcl|NC_013966.1_cds_WP_013035117.1_783      16 HVO_RS03335  3335  341689  342843       pHV4   0
## 32   lcl|NC_013966.1_cds_WP_004043245.1_784      16 HVO_RS03340  3340  342911  343681       pHV4   1
## 33   lcl|NC_013966.1_cds_WP_004043267.1_812      16 HVO_RS03460  3460  371820  372134       pHV4   0
## 51   lcl|NC_013966.1_cds_WP_013035194.1_818      16 HVO_RS03485  3485  377675  377890       pHV4   0
## 34   lcl|NC_013966.1_cds_WP_004043278.1_828      16 HVO_RS03530  3530  386436  387218       pHV4   0
## 35   lcl|NC_013966.1_cds_WP_004043279.1_829      16 HVO_RS03535  3535  387218  388993       pHV4   1
## 44   lcl|NC_013966.1_cds_WP_013035057.1_830      16 HVO_RS03540  3540  388990  391035       pHV4   1
## 23   lcl|NC_013966.1_cds_WP_004041391.1_876      16 HVO_RS03745  3745  431457  432383       pHV4   0
## 47   lcl|NC_013966.1_cds_WP_013035123.1_878      16 HVO_RS03755  3755  434234  435847       pHV4   0
## 19   lcl|NC_013966.1_cds_WP_004040812.1_910      16 HVO_RS03875  3875  465053  465928       pHV4   0
## 20   lcl|NC_013966.1_cds_WP_004040813.1_911      16 HVO_RS03880  3880  465925  466335       pHV4   1
## 42   lcl|NC_013966.1_cds_WP_013035041.1_990      16 HVO_RS04265  4265  544688  545944       pHV4   0
## 21  lcl|NC_013966.1_cds_WP_004040957.1_1045      16 HVO_RS04540  4540  603818  604237       pHV4   0
## 114 lcl|NC_013967.1_cds_WP_004044979.1_1095      16 HVO_RS04785  4785   16358   16936 Chromosome   0
## 115 lcl|NC_013967.1_cds_WP_004045090.1_1140      16 HVO_RS05000  5000   61296   63134 Chromosome   0
## 116 lcl|NC_013967.1_cds_WP_004045114.1_1148      16 HVO_RS05040  5040   72583   72900 Chromosome   0
## 117 lcl|NC_013967.1_cds_WP_004045245.1_1208      16 HVO_RS05345  5345  123087  124739 Chromosome   0
## 118 lcl|NC_013967.1_cds_WP_004045293.1_1254      16 HVO_RS05575  5575  162031  162816 Chromosome   0
## 119 lcl|NC_013967.1_cds_WP_004045346.1_1306      16 HVO_RS05835  5835  209687  209917 Chromosome   0
## 120 lcl|NC_013967.1_cds_WP_004045357.1_1315      16 HVO_RS05880  5880  218337  219419 Chromosome   0
## 113 lcl|NC_013967.1_cds_WP_004044535.1_1451      16 HVO_RS06570  6570  343699  345018 Chromosome   0
## 112 lcl|NC_013967.1_cds_WP_004044467.1_1516      16 HVO_RS06890  6890  405300  406943 Chromosome   0
## 111 lcl|NC_013967.1_cds_WP_004044463.1_1520      16 HVO_RS06915  6915  408661  409047 Chromosome   0
## 124 lcl|NC_013967.1_cds_WP_013035627.1_1539      16 HVO_RS07010  7010  426605  427672 Chromosome   0
## 110 lcl|NC_013967.1_cds_WP_004044373.1_1609      16 HVO_RS07390  7390  494928  495119 Chromosome   0
## 109 lcl|NC_013967.1_cds_WP_004044253.1_1725      16 HVO_RS07980  7980  609107  611629 Chromosome   0
## 108 lcl|NC_013967.1_cds_WP_004044159.1_1814      16 HVO_RS08430  8430  697240  698802 Chromosome   0
## 107 lcl|NC_013967.1_cds_WP_004044124.1_1847      16 HVO_RS08595  8595  731572  733857 Chromosome   0
## 127 lcl|NC_013967.1_cds_WP_049941503.1_1995      16 HVO_RS09335  9335  874483  876177 Chromosome   0
## 106 lcl|NC_013967.1_cds_WP_004043865.1_2097      16 HVO_RS09850  9850  979756  980430 Chromosome   0
## 105 lcl|NC_013967.1_cds_WP_004043814.1_2146      16 HVO_RS10095 10095 1023195 1024244 Chromosome   0
## 104 lcl|NC_013967.1_cds_WP_004043779.1_2182      16 HVO_RS10285 10285 1061644 1061919 Chromosome   0
## 103 lcl|NC_013967.1_cds_WP_004043778.1_2183      16 HVO_RS10290 10290 1062179 1062679 Chromosome   1
## 102 lcl|NC_013967.1_cds_WP_004043777.1_2184      16 HVO_RS10295 10295 1062761 1063573 Chromosome   1
## 101 lcl|NC_013967.1_cds_WP_004043756.1_2204      16 HVO_RS10395 10395 1079116 1079298 Chromosome   0
## 100 lcl|NC_013967.1_cds_WP_004043650.1_2304      16 HVO_RS10895 10895 1174083 1174481 Chromosome   0
## 99  lcl|NC_013967.1_cds_WP_004043649.1_2305      16 HVO_RS10900 10900 1174580 1175296 Chromosome   1
## 98  lcl|NC_013967.1_cds_WP_004043633.1_2320      16 HVO_RS10980 10980 1188454 1189392 Chromosome   0
## 97  lcl|NC_013967.1_cds_WP_004043632.1_2321      16 HVO_RS10985 10985 1189396 1191291 Chromosome   1
## 96  lcl|NC_013967.1_cds_WP_004043626.1_2327      16 HVO_RS11015 11015 1196487 1197008 Chromosome   0
## 95  lcl|NC_013967.1_cds_WP_004043593.1_2359      16 HVO_RS11180 11180 1223505 1223834 Chromosome   0
## 94  lcl|NC_013967.1_cds_WP_004043510.1_2438      16 HVO_RS11575 11575 1298230 1299027 Chromosome   0
## 93  lcl|NC_013967.1_cds_WP_004043491.1_2456      16 HVO_RS11670 11670 1315527 1316387 Chromosome   0
## 92  lcl|NC_013967.1_cds_WP_004043462.1_2484      16 HVO_RS11810 11810 1343855 1344133 Chromosome   0
## 59  lcl|NC_013967.1_cds_WP_004041462.1_2587      16 HVO_RS12300 12300 1453098 1453520 Chromosome   0
## 60  lcl|NC_013967.1_cds_WP_004041492.1_2605      16 HVO_RS12390 12390 1470929 1471363 Chromosome   0
## 61  lcl|NC_013967.1_cds_WP_004041499.1_2612      16 HVO_RS12425 12425 1477729 1477974 Chromosome   0
## 62  lcl|NC_013967.1_cds_WP_004041569.1_2680      16 HVO_RS12770 12770 1536158 1536394 Chromosome   0
## 63  lcl|NC_013967.1_cds_WP_004041593.1_2704      16 HVO_RS12895 12895 1565815 1566972 Chromosome   0
## 64  lcl|NC_013967.1_cds_WP_004041594.1_2705      16 HVO_RS12900 12900 1567095 1568261 Chromosome   1
## 65  lcl|NC_013967.1_cds_WP_004041623.1_2734      16 HVO_RS13070 13070 1610465 1611235 Chromosome   0
## 66  lcl|NC_013967.1_cds_WP_004041626.1_2737      16 HVO_RS13085 13085 1612996 1613553 Chromosome   0
## 67  lcl|NC_013967.1_cds_WP_004041630.1_2741      16 HVO_RS13105 13105 1617657 1618628 Chromosome   0
## 68  lcl|NC_013967.1_cds_WP_004041668.1_2778      16 HVO_RS13290 13290 1652567 1654345 Chromosome   0
## 126 lcl|NC_013967.1_cds_WP_049914780.1_2857      16 HVO_RS13685 13685 1729602 1729850 Chromosome   0
## 69  lcl|NC_013967.1_cds_WP_004041844.1_2942      16 HVO_RS14125 14125 1813914 1814372 Chromosome   0
## 70  lcl|NC_013967.1_cds_WP_004041906.1_3001      16 HVO_RS14415 14415 1866807 1867007 Chromosome   0
## 71  lcl|NC_013967.1_cds_WP_004041927.1_3020      16 HVO_RS14515 14515 1885450 1886370 Chromosome   0
## 72  lcl|NC_013967.1_cds_WP_004041940.1_3036      16 HVO_RS14580 14580 1905450 1905920 Chromosome   0
## 73  lcl|NC_013967.1_cds_WP_004041941.1_3037      16 HVO_RS14585 14585 1905983 1907056 Chromosome   1
## 74  lcl|NC_013967.1_cds_WP_004041942.1_3038      16 HVO_RS14590 14590 1907059 1907238 Chromosome   1
## 75  lcl|NC_013967.1_cds_WP_004041965.1_3062      16 HVO_RS14710 14710 1941469 1942626 Chromosome   0
## 76  lcl|NC_013967.1_cds_WP_004041967.1_3064      16 HVO_RS14720 14720 1943302 1944444 Chromosome   0
## 77  lcl|NC_013967.1_cds_WP_004041987.1_3082      16 HVO_RS14810 14810 1967493 1967789 Chromosome   0
## 78  lcl|NC_013967.1_cds_WP_004042023.1_3119      16 HVO_RS14990 14990 2004170 2005249 Chromosome   0
## 79  lcl|NC_013967.1_cds_WP_004042025.1_3120      16 HVO_RS14995 14995 2005712 2006071 Chromosome   1
## 80  lcl|NC_013967.1_cds_WP_004042143.1_3230      16 HVO_RS15545 15545 2115430 2116818 Chromosome   0
## 81  lcl|NC_013967.1_cds_WP_004042150.1_3237      16 HVO_RS15580 15580 2122480 2122746 Chromosome   0
## 125 lcl|NC_013967.1_cds_WP_013035629.1_3242      16 HVO_RS15605 15605 2126758 2128980 Chromosome   0
## 82  lcl|NC_013967.1_cds_WP_004042169.1_3253      16 HVO_RS15660 15660 2144264 2144617 Chromosome   0
## 83  lcl|NC_013967.1_cds_WP_004042173.1_3257      16 HVO_RS15680 15680 2147711 2148052 Chromosome   0
## 128 lcl|NC_013967.1_cds_WP_049941509.1_3266      16 HVO_RS15725 15725 2159320 2160219 Chromosome   0
## 84  lcl|NC_013967.1_cds_WP_004042226.1_3311      16 HVO_RS15945 15945 2201120 2202028 Chromosome   0
## 85  lcl|NC_013967.1_cds_WP_004042244.1_3329      16 HVO_RS16035 16035 2219002 2220024 Chromosome   0
## 86  lcl|NC_013967.1_cds_WP_004042359.1_3425      16 HVO_RS16515 16515 2319307 2319735 Chromosome   0
## 87  lcl|NC_013967.1_cds_WP_004042360.1_3426      16 HVO_RS16520 16520 2320189 2323290 Chromosome   1
## 88  lcl|NC_013967.1_cds_WP_004042648.1_3550      16 HVO_RS17165 17165 2432748 2433905 Chromosome   0
## 89  lcl|NC_013967.1_cds_WP_004042688.1_3571      16 HVO_RS17270 17270 2452379 2452969 Chromosome   0
## 90  lcl|NC_013967.1_cds_WP_004042796.1_3627      16 HVO_RS17550 17550 2509804 2510916 Chromosome   0
## 91  lcl|NC_013967.1_cds_WP_004042886.1_3665      16 HVO_RS17740 17740 2546945 2549209 Chromosome   0
## 137   lcl|NC_013968.1_cds_WP_013035714.1_12      16 HVO_RS19340 19340    7721    9085       pHV1   0
## 136   lcl|NC_013968.1_cds_WP_013035707.1_18      16 HVO_RS19365 19365   13539   13925       pHV1   0
## 131   lcl|NC_013968.1_cds_WP_013035671.1_31      16 HVO_RS19410 19410   21542   22072       pHV1   0
## 135   lcl|NC_013968.1_cds_WP_013035705.1_42      16 HVO_RS19460 19460   44460   45461       pHV1   0
## 130   lcl|NC_013968.1_cds_WP_004043317.1_58      16 HVO_RS19530 19530   61032   61364       pHV1   0
## 132   lcl|NC_013968.1_cds_WP_013035678.1_63      16 HVO_RS19555 19555   66399   67373       pHV1   0
## 133   lcl|NC_013968.1_cds_WP_013035693.1_68      16 HVO_RS19580 19580   72312   74384       pHV1   0
## 139   lcl|NC_013968.1_cds_WP_013035730.1_71      16 HVO_RS19595 19595   77782   78945       pHV1   0
## 49   lcl|NC_013966.1_cds_WP_013035160.1_504      16 HVO_RS19750 19750   36821   37042       pHV4   0
## 16                  lcl|NC_013966.1_cds_522      16 HVO_RS19760 19760   50200   50469       pHV4   0
## 17                  lcl|NC_013966.1_cds_565      16 HVO_RS19775 19775  105533  106533       pHV4   0
## 45   lcl|NC_013966.1_cds_WP_013035103.1_591      16 HVO_RS19800 19800  127994  128602       pHV4   0
## 18                  lcl|NC_013966.1_cds_594      16 HVO_RS19810 19810  131940  132431       pHV4   0
##  [ reached &#39;max&#39; / getOption(&quot;max.print&quot;) -- omitted 14 rows ]
##                                      genes cluster   locus_tag    rs   start    stop        chr op1
## 1   lcl|NC_013964.1_cds_WP_004041138.1_102      17 HVO_RS00070    70   16592   16822       pHV3   0
## 4   lcl|NC_013964.1_cds_WP_013034939.1_270      17 HVO_RS00905   905  213673  215244       pHV3   0
## 3   lcl|NC_013964.1_cds_WP_004041339.1_282      17 HVO_RS00960   960  225456  225641       pHV3   0
## 2   lcl|NC_013964.1_cds_WP_004041154.1_460      17 HVO_RS01835  1835  435809  436474       pHV3   0
## 10  lcl|NC_013966.1_cds_WP_013035020.1_566      17 HVO_RS02345  2345  106831  107016       pHV4   0
## 5   lcl|NC_013966.1_cds_WP_004040859.1_958      17 HVO_RS04110  4110  514365  514685       pHV4   0
## 6   lcl|NC_013966.1_cds_WP_004040860.1_959      17 HVO_RS04115  4115  514688  515506       pHV4   1
## 7   lcl|NC_013966.1_cds_WP_004040861.1_960      17 HVO_RS04120  4120  515499  515942       pHV4   1
## 8   lcl|NC_013966.1_cds_WP_004040883.1_982      17 HVO_RS04225  4225  535259  535849       pHV4   0
## 9  lcl|NC_013966.1_cds_WP_004040967.1_1055      17 HVO_RS04590  4590  611603  612610       pHV4   0
## 21 lcl|NC_013967.1_cds_WP_004045302.1_1262      17 HVO_RS05615  5615  169702  170184 Chromosome   0
## 22 lcl|NC_013967.1_cds_WP_049914900.1_1590      17 HVO_RS07295  7295  469061  469873 Chromosome   0
## 20 lcl|NC_013967.1_cds_WP_004043974.1_1997      17 HVO_RS09345  9345  879448  880692 Chromosome   0
## 11 lcl|NC_013967.1_cds_WP_004041591.1_2702      17 HVO_RS12885 12885 1562799 1563230 Chromosome   0
## 12 lcl|NC_013967.1_cds_WP_004041637.1_2748      17 HVO_RS13140 13140 1624604 1624786 Chromosome   0
## 13 lcl|NC_013967.1_cds_WP_004042005.1_3100      17 HVO_RS14895 14895 1985891 1986100 Chromosome   0
## 14 lcl|NC_013967.1_cds_WP_004042008.1_3103      17 HVO_RS14910 14910 1988273 1989277 Chromosome   0
## 15 lcl|NC_013967.1_cds_WP_004042009.1_3104      17 HVO_RS14915 14915 1989279 1990238 Chromosome   1
## 16 lcl|NC_013967.1_cds_WP_004042021.1_3117      17 HVO_RS14980 14980 2003173 2003493 Chromosome   0
## 17 lcl|NC_013967.1_cds_WP_004042098.1_3184      17 HVO_RS15315 15315 2074256 2074501 Chromosome   0
## 18 lcl|NC_013967.1_cds_WP_004042753.1_3602      17 HVO_RS17425 17425 2485152 2485982 Chromosome   0
## 19 lcl|NC_013967.1_cds_WP_004042764.1_3609      17 HVO_RS17460 17460 2491350 2492390 Chromosome   0
## 23 lcl|NC_013967.1_cds_WP_081442893.1_3504      17 HVO_RS20295 20295 2400186 2400497 Chromosome   0
##                                     genes cluster   locus_tag    rs   start    stop        chr op1
## 1  lcl|NC_013964.1_cds_WP_004041039.1_202      18 HVO_RS00565   565  130859  131485       pHV3   0
## 5 lcl|NC_013967.1_cds_WP_004043989.1_1984      18 HVO_RS09280  9280  865710  866747 Chromosome   0
## 4 lcl|NC_013967.1_cds_WP_004043916.1_2053      18 HVO_RS09625  9625  935515  935928 Chromosome   0
## 3 lcl|NC_013967.1_cds_WP_004043665.1_2289      18 HVO_RS10820 10820 1161871 1163367 Chromosome   0
## 2 lcl|NC_013967.1_cds_WP_004042306.1_3390      18 HVO_RS16340 16340 2285471 2286058 Chromosome   0
##                                      genes cluster   locus_tag    rs   start    stop        chr op1
## 1   lcl|NC_013966.1_cds_WP_049941496.1_903      19 HVO_RS03845  3845  458053  458292       pHV4   0
## 24 lcl|NC_013967.1_cds_WP_004045222.1_1194      19 HVO_RS05270  5270  112264  112734 Chromosome   0
## 25 lcl|NC_013967.1_cds_WP_004045268.1_1231      19 HVO_RS05460  5460  142995  143201 Chromosome   0
## 23 lcl|NC_013967.1_cds_WP_004044577.1_1412      19 HVO_RS06370  6370  312095  312322 Chromosome   0
## 22 lcl|NC_013967.1_cds_WP_004044502.1_1483      19 HVO_RS06725  6725  379829  380122 Chromosome   0
## 21 lcl|NC_013967.1_cds_WP_004044459.1_1524      19 HVO_RS06935  6935  412138  412344 Chromosome   0
## 20 lcl|NC_013967.1_cds_WP_004044289.1_1692      19 HVO_RS07815  7815  580081  580632 Chromosome   0
## 19 lcl|NC_013967.1_cds_WP_004044074.1_1895      19 HVO_RS08835  8835  778851  779303 Chromosome   0
## 18 lcl|NC_013967.1_cds_WP_004044030.1_1939      19 HVO_RS09055  9055  826378  826743 Chromosome   0
## 17 lcl|NC_013967.1_cds_WP_004043965.1_2005      19 HVO_RS09385  9385  889252  889668 Chromosome   0
## 16 lcl|NC_013967.1_cds_WP_004043848.1_2114      19 HVO_RS09935  9935  996959  997312 Chromosome   0
## 15 lcl|NC_013967.1_cds_WP_004043790.1_2171      19 HVO_RS10220 10220 1047847 1048314 Chromosome   0
## 14 lcl|NC_013967.1_cds_WP_004043767.1_2194      19 HVO_RS10345 10345 1070635 1071165 Chromosome   0
## 13 lcl|NC_013967.1_cds_WP_004043733.1_2227      19 HVO_RS10510 10510 1099534 1099872 Chromosome   0
## 12 lcl|NC_013967.1_cds_WP_004043547.1_2405      19 HVO_RS11410 11410 1265555 1265884 Chromosome   0
## 11 lcl|NC_013967.1_cds_WP_004043512.1_2436      19 HVO_RS11565 11565 1296636 1296893 Chromosome   0
## 2  lcl|NC_013967.1_cds_WP_004041438.1_2566      19 HVO_RS12195 12195 1429308 1429667 Chromosome   0
## 3  lcl|NC_013967.1_cds_WP_004041451.1_2577      19 HVO_RS12250 12250 1441393 1443312 Chromosome   0
## 26 lcl|NC_013967.1_cds_WP_013035508.1_2848      19 HVO_RS13640 13640 1720493 1720738 Chromosome   0
## 4  lcl|NC_013967.1_cds_WP_004041782.1_2882      19 HVO_RS13820 13820 1749192 1750427 Chromosome   0
## 5  lcl|NC_013967.1_cds_WP_004041883.1_2980      19 HVO_RS14315 14315 1848449 1848724 Chromosome   0
## 6  lcl|NC_013967.1_cds_WP_004041902.1_2997      19 HVO_RS14395 14395 1864256 1864615 Chromosome   0
## 7  lcl|NC_013967.1_cds_WP_004042136.1_3222      19 HVO_RS15505 15505 2104893 2105708 Chromosome   0
## 8  lcl|NC_013967.1_cds_WP_004042227.1_3312      19 HVO_RS15950 15950 2202120 2202623 Chromosome   0
## 9  lcl|NC_013967.1_cds_WP_004042667.1_3561      19 HVO_RS17220 17220 2442972 2443604 Chromosome   0
## 10 lcl|NC_013967.1_cds_WP_004042991.1_3714      19 HVO_RS17985 17985 2590818 2591108 Chromosome   0
##                                      genes cluster   locus_tag    rs   start    stop        chr op1
## 1   lcl|NC_013964.1_cds_WP_004041111.1_130      20 HVO_RS00205   205   49584   50225       pHV3   0
## 3   lcl|NC_013966.1_cds_WP_013035208.1_547      20 HVO_RS02245  2245   82394   83161       pHV4   0
## 2   lcl|NC_013966.1_cds_WP_013035056.1_626      20 HVO_RS02605  2605  167050  168516       pHV4   0
## 10 lcl|NC_013967.1_cds_WP_004044307.1_1675      20 HVO_RS07725  7725  560896  562620 Chromosome   0
## 9  lcl|NC_013967.1_cds_WP_004044270.1_1709      20 HVO_RS07900  7900  593079  594002 Chromosome   0
## 8  lcl|NC_013967.1_cds_WP_004043868.1_2094      20 HVO_RS09835  9835  977936  978274 Chromosome   0
## 5  lcl|NC_013967.1_cds_WP_004041580.1_2691      20 HVO_RS12830 12830 1547818 1548111 Chromosome   0
## 6  lcl|NC_013967.1_cds_WP_004041729.1_2831      20 HVO_RS13550 13550 1704184 1704411 Chromosome   0
## 7  lcl|NC_013967.1_cds_WP_004041990.1_3085      20 HVO_RS14825 14825 1969381 1969806 Chromosome   0
## 11 lcl|NC_013967.1_cds_WP_013035563.1_3132      20 HVO_RS15055 15055 2017239 2018687 Chromosome   0
## 12    lcl|NC_013968.1_cds_WP_013035697.1_7      20 HVO_RS19315 19315    5497    5742       pHV1   0
## 4                 lcl|NC_013967.1_cds_2897      20 HVO_RS20210 20210 1768353 1768587 Chromosome   0
##                                     genes cluster   locus_tag    rs   start    stop        chr op1
## 1  lcl|NC_013966.1_cds_WP_049914857.1_842      21 HVO_RS03600  3600  402900  403319       pHV4   0
## 6 lcl|NC_013967.1_cds_WP_004044573.1_1416      21 HVO_RS06390  6390  318645  319904 Chromosome   0
## 5 lcl|NC_013967.1_cds_WP_004044076.1_1893      21 HVO_RS08825  8825  777003  778211 Chromosome   0
## 4 lcl|NC_013967.1_cds_WP_004043754.1_2206      21 HVO_RS10405 10405 1079634 1080065 Chromosome   0
## 3 lcl|NC_013967.1_cds_WP_004043693.1_2266      21 HVO_RS10705 10705 1139167 1139376 Chromosome   0
## 2 lcl|NC_013967.1_cds_WP_004043634.1_2319      21 HVO_RS10975 10975 1187908 1188336 Chromosome   0
##                                     genes cluster   locus_tag   rs  start   stop        chr op1
## 1  lcl|NC_013966.1_cds_WP_049914856.1_840      22 HVO_RS03590 3590 402149 402334       pHV4   0
## 3 lcl|NC_013967.1_cds_WP_004045003.1_1100      22 HVO_RS04810 4810  21243  22379 Chromosome   0
## 2 lcl|NC_013967.1_cds_WP_004044276.1_1703      22 HVO_RS07870 7870 588886 589224 Chromosome   0</code></pre>
<pre class="r"><code># for i in cluster_*.txt; do awk &#39;/\t1$/{if (a &amp;&amp; a !~ /\t1$/) print a; print} {a=$0}&#39; $i &gt; filtered_$i; done
# grep &quot;Chromosome&quot; filtered_cluster_3_operons.txt | awk -F&quot;\t&quot; &#39;{print $3}&#39; &gt; genes_c3.txt
# grep -f genes_c3.txt ../../full_annot.txt | grep -v -i &quot;ribosomal protein&quot; |\
#   awk -F&quot;\t&quot; &#39;{OFS=&quot;\t&quot;} {print $1,$4,$7,$8,$13,$15,$38,$42,$43,$45,$46,$47,$48,$49,$50,$51,$52,$53,$54,$55,$56}&#39; |\
#   sort -k 2 | uniq &gt; c3_op_annot.txt
# </code></pre>
<pre class="r"><code>genes &lt;- read.delim(&quot;data/genes_for_map.txt&quot;, header = TRUE)

ggplot(genes, aes(xmin = start, xmax = stop, y = chr, fill = locus_tag, label = gene)) +
  geom_gene_arrow() +
  geom_gene_label(align = &quot;left&quot;) +
  facet_wrap(~ chr, scales = &quot;free&quot;, ncol = 1) +
  scale_fill_brewer(palette = &quot;Set3&quot;) +
  theme_genes()</code></pre>
<p><img src="hvo_mating_files/figure-html/gene%20map-1.png" width="672" /></p>
<pre class="r"><code>genes$direction &lt;- ifelse(genes$strand == &quot;+&quot;, 1, -1)
ggplot(
  subset(genes, chr == &quot;Chromosome&quot;),
  aes(xmin = start, xmax = stop, y = strand, fill = locus_tag, label = gene, forward = direction)
  ) +
  geom_gene_arrow() +
  geom_gene_label(align = &quot;center&quot;) +
  scale_fill_brewer(palette = &quot;Set3&quot;) +
  theme_genes() +
  theme(legend.title = element_blank()) +
  theme(legend.key.size = unit(0.3, &quot;cm&quot;))</code></pre>
<p><img src="hvo_mating_files/figure-html/gene%20map-2.png" width="672" /></p>
<pre class="r"><code>ggsave(&quot;plots/gene_map.pdf&quot;, height = 2, width = 6, units = &quot;in&quot;)</code></pre>
<pre class="r"><code>### from above script
#explo.plot(myPCA, factor = &quot;time&quot;)
library(scales)
library(ggplot2)
###
mating &lt;- read.table(&quot;data/mating.txt&quot;, header=T, com=&#39;&#39;, check.names=F)
ggplot(mating, aes(time2,average, group=1)) +
  geom_errorbar(aes(ymin=average-stderr,ymax=average+stderr), width=0.1) +
  geom_line() +
  geom_point() +
  scale_x_continuous(breaks = c(0,2,4,8,24), labels = c(&quot;0&quot;,&quot;2&quot;,&quot;4&quot;,&quot;8&quot;,&quot;24&quot;)) +
  # scale_x_continuous(breaks = c(0,2,4,8,24), labels = NULL) +
  theme_bw() +
  theme(panel.grid.major.x = element_blank(),
        panel.grid.minor.x = element_blank(),
        panel.grid.major.y = element_blank(),
        panel.grid.minor.y = element_blank(),
        panel.border = element_rect(colour = &#39;black&#39;, size = 0.75),
        text = element_text(size=18, color = &quot;black&quot;)
        # axis.ticks.x = element_blank(),
        # axis.ticks.length = unit(3, &quot;mm&quot;),
        # axis.text.x = element_text(margin=margin(2,2,5,5,&quot;mm&quot;)),
        # axis.text.y = element_text(margin=margin(2,2,5,5,&quot;mm&quot;)),
        # axis.title = element_blank(),
        # axis.text = element_blank()
        ) +
  xlab(&quot;Time (h)&quot;) +
  ylab(&quot;Mean % mated&quot;)</code></pre>
<embed src="hvo_mating_files/figure-html/wet plots-1.pdf" width="576" type="application/pdf" />
<pre class="r"><code>ggsave(&quot;plots/mean_mated.pdf&quot;)</code></pre>
<pre><code>## Saving 6 x 3 in image</code></pre>
<pre class="r"><code>mating &lt;- read.table(&quot;data/mating_all.txt&quot;, header=T, com=&#39;&#39;, check.names=F)
ggplot(mating, aes(time2,mated, group=1)) +
  geom_point() +
  scale_x_continuous(breaks = c(0,2,4,8,24), labels = c(&quot;0&quot;,&quot;2&quot;,&quot;4&quot;,&quot;8&quot;,&quot;24&quot;)) +
  # scale_x_continuous(breaks = c(0,2,4,8,24), labels = NULL) +
  geom_smooth(method=&#39;lm&#39;, se = FALSE) +
  theme_bw() +
  theme(panel.grid.major.x = element_blank(),
        panel.grid.minor.x = element_blank(),
        panel.grid.major.y = element_blank(),
        panel.grid.minor.y = element_blank(),
        panel.border = element_rect(colour = &#39;black&#39;, size = 0.75),
        text = element_text(size=18),
        # axis.ticks.x = element_blank(),
        axis.ticks.length = unit(3, &quot;mm&quot;),
        # axis.text.x = element_text(margin=margin(2,2,5,5,&quot;mm&quot;)),
        # axis.text.y = element_text(margin=margin(2,2,5,5,&quot;mm&quot;)),
        # axis.title = element_blank(),
        axis.text = element_text(color = &quot;black&quot;)
        ) +
  scale_y_continuous(labels = scientific) +
  xlab(&quot;Time (h)&quot;) +
  ylab(&quot;Mating efficiency&quot;)</code></pre>
<embed src="hvo_mating_files/figure-html/wet plots-2.pdf" width="576" type="application/pdf" />
<pre class="r"><code>ggsave(&quot;plots/percent_mated.pdf&quot;)</code></pre>
<pre><code>## Saving 6 x 3 in image</code></pre>
<pre class="r"><code>growth &lt;- read.table(&quot;data/growth.txt&quot;, header=T, com=&#39;&#39;, check.names=F)
ggplot(growth, aes(time2,cfu_mean, group=1)) +
  geom_errorbar(aes(ymin=cfu_mean-cfu_err,ymax=cfu_mean+cfu_err), width=0.1) +
  geom_point() +
  scale_x_continuous(breaks = c(0,2,4,8,24), labels = c(&quot;0&quot;,&quot;2&quot;,&quot;4&quot;,&quot;8&quot;,&quot;24&quot;)) +
  # scale_x_continuous(breaks = c(0,2,4,8,24), labels = NULL) +
  geom_line() +
  theme_bw() +
  theme(panel.grid.major.x = element_blank(),
        panel.grid.minor.x = element_blank(),
        panel.grid.major.y = element_blank(),
        panel.grid.minor.y = element_blank(),
        panel.border = element_rect(colour = &#39;black&#39;, size = 0.75),
        text = element_text(size=18),
        # axis.ticks.x = element_blank(),
        axis.ticks.length = unit(3, &quot;mm&quot;),
        # axis.text.x = element_text(margin=margin(2,2,5,5,&quot;mm&quot;)),
        # axis.text.y = element_text(margin=margin(2,2,5,5,&quot;mm&quot;)),
        # axis.title = element_blank(),
        axis.text = element_text(color = &quot;black&quot;)
        ) +
  scale_y_log10() +
  xlab(&quot;time (h)&quot;) +
  ylab(substitute(paste(&quot;mean CFU&quot;, &quot; (log&quot;[10], &quot;)&quot;)))</code></pre>
<embed src="hvo_mating_files/figure-html/wet plots-3.pdf" width="576" type="application/pdf" />
<pre class="r"><code>ggsave(&quot;plots/growth.pdf&quot;)</code></pre>
<pre><code>## Saving 6 x 3 in image</code></pre>
<pre class="r"><code>mating &lt;- read.table(&quot;data/mated_full.txt&quot;, header=T, com=&#39;&#39;, check.names=F)
mating_lm &lt;- lm(mated ~ 0 + time, data = mating)
ggplot(mating, aes(time,mated, group=1)) +
  geom_point() +
  scale_x_continuous(breaks = c(-1,0,2,4,8,24), labels = c(&quot;sh&quot;,&quot;0&quot;,&quot;2&quot;,&quot;4&quot;,&quot;8&quot;,&quot;24&quot;)) +
  # scale_x_continuous(breaks = c(0,2,4,8,24), labels = NULL) +
  #geom_smooth(method=&#39;glm&#39;, se = FALSE) +
  geom_abline(intercept=0, slope=mating_lm$coefficients[1], color=&#39;darkred&#39;, size=1.1) + 
  theme_bw() +
  theme(panel.grid.major.x = element_blank(),
        panel.grid.minor.x = element_blank(),
        panel.grid.major.y = element_blank(),
        panel.grid.minor.y = element_blank(),
        panel.border = element_rect(colour = &#39;black&#39;, size = 0.75),
        text = element_text(size=18),
        # axis.ticks.x = element_blank(),
        axis.ticks.length = unit(3, &quot;mm&quot;),
        # axis.text.x = element_text(margin=margin(2,2,5,5,&quot;mm&quot;)),
        # axis.text.y = element_text(margin=margin(2,2,5,5,&quot;mm&quot;)),
        # axis.title = element_blank(),
        axis.text = element_text(color = &quot;black&quot;)
        ) +
  scale_y_continuous(labels = scientific) +
 # scale_y_log10() + 
  xlab(&quot;Time (h)&quot;) +
  ylab(&quot;Mating efficiency&quot;)</code></pre>
<embed src="hvo_mating_files/figure-html/wet plots-4.pdf" width="576" type="application/pdf" />
<pre class="r"><code>ggsave(&quot;plots/mating_efficiency.pdf&quot;)</code></pre>
<pre><code>## Saving 6 x 3 in image</code></pre>
<pre class="r"><code>data_summary &lt;- function(data, varname, groupnames){
  require(plyr)
  summary_func &lt;- function(x, col){
    c(mean = mean(x[[col]], na.rm=TRUE),
      se = sd(x[[col]], na.rm=TRUE) / sqrt(length(x[[col]])))
  }
  data_sum&lt;-ddply(data, groupnames, .fun=summary_func,
                  varname)
  data_sum &lt;- rename(data_sum, c(&quot;mean&quot; = varname))
 return(data_sum)
}

mating2 &lt;- data_summary(mating, varname=&quot;mated&quot;, 
                    groupnames=&quot;time&quot;)

ggplot(mating2, aes(time, mated, group=1)) +
  geom_point() +
  scale_x_continuous(breaks = c(-1,0,2,4,8,24), labels = c(&quot;sh&quot;,&quot;0&quot;,&quot;2&quot;,&quot;4&quot;,&quot;8&quot;,&quot;24&quot;)) +
  # scale_x_continuous(breaks = c(0,2,4,8,24), labels = NULL) +
  geom_errorbar(aes(ymin=mated-se,ymax=mated+se), width=0.1) +
  theme_bw() +
  theme(panel.grid.major.x = element_blank(),
        panel.grid.minor.x = element_blank(),
        panel.grid.major.y = element_blank(),
        panel.grid.minor.y = element_blank(),
        panel.border = element_rect(colour = &#39;black&#39;, size = 0.75),
        text = element_text(size=18),
        # axis.ticks.x = element_blank(),
        axis.ticks.length = unit(3, &quot;mm&quot;),
        # axis.text.x = element_text(margin=margin(2,2,5,5,&quot;mm&quot;)),
        # axis.text.y = element_text(margin=margin(2,2,5,5,&quot;mm&quot;)),
        # axis.title = element_blank(),
        axis.text = element_text(color = &quot;black&quot;)
        ) +
  scale_y_continuous(labels = scientific) +
  xlab(&quot;Time (h)&quot;) +
  ylab(&quot;Mating efficiency&quot;)</code></pre>
<embed src="hvo_mating_files/figure-html/wet plots-5.pdf" width="576" type="application/pdf" />
<pre class="r"><code>ggsave(&quot;plots/mating_efficiency_err.pdf&quot;)</code></pre>
<pre><code>## Saving 6 x 3 in image</code></pre>
<pre class="r"><code>library(ISwR)
mating_lm &lt;- lm(mated ~ 0 + time, data = mating)
summary(mating_lm)</code></pre>
<pre><code>## 
## Call:
## lm(formula = mated ~ 0 + time, data = mating)
## 
## Residuals:
##        Min         1Q     Median         3Q        Max 
## -2.772e-05 -4.235e-06  2.304e-06  1.177e-05  1.837e-04 
## 
## Coefficients:
##       Estimate Std. Error t value Pr(&gt;|t|)   
## time 2.304e-06  7.177e-07    3.21  0.00284 **
## ---
## Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1
## 
## Residual standard error: 4.52e-05 on 35 degrees of freedom
## Multiple R-squared:  0.2274, Adjusted R-squared:  0.2053 
## F-statistic:  10.3 on 1 and 35 DF,  p-value: 0.002844</code></pre>
<pre class="r"><code>confint(mating_lm, &#39;time&#39;, level=0.95)</code></pre>
<pre><code>##             2.5 %       97.5 %
## time 8.465496e-07 3.760532e-06</code></pre>
<pre class="r"><code>mating_noz &lt;- read.table(&quot;data/mated_full_noz.txt&quot;, header=T, com=&#39;&#39;, check.names=F)
mating_lm_noz &lt;- lm(mated ~ time, data = mating_noz)
summary(mating_lm_noz)</code></pre>
<pre><code>## 
## Call:
## lm(formula = mated ~ time, data = mating_noz)
## 
## Residuals:
##        Min         1Q     Median         3Q        Max 
## -4.114e-05 -2.555e-05 -1.556e-05  2.187e-06  1.513e-04 
## 
## Coefficients:
##               Estimate Std. Error t value Pr(&gt;|t|)   
## (Intercept)  4.204e-05  1.477e-05   2.846  0.00939 **
## time        -1.136e-07  1.150e-06  -0.099  0.92222   
## ---
## Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1
## 
## Residual standard error: 4.871e-05 on 22 degrees of freedom
## Multiple R-squared:  0.0004432,  Adjusted R-squared:  -0.04499 
## F-statistic: 0.009754 on 1 and 22 DF,  p-value: 0.9222</code></pre>
<pre class="r"><code>confint(mating_lm_noz, &#39;time&#39;, level=0.95)</code></pre>
<pre><code>##              2.5 %       97.5 %
## time -2.498371e-06 2.271227e-06</code></pre>
<pre class="r"><code>ggplot(mating_noz, aes(time, mated, group=1)) +
  geom_point(size = 3) +
  scale_x_continuous(breaks = c(0,2,4,8,24), labels = c(&quot;&quot;,&quot;2&quot;,&quot;4&quot;,&quot;8&quot;,&quot;24&quot;), limits = c(0,24)) +
  # scale_x_continuous(breaks = c(0,2,4,8,24), labels = NULL) +
  geom_smooth(method=&#39;glm&#39;, se = FALSE, size = 2) +
  #geom_abline(intercept=0, slope=mating_lm$coefficients[1], color=&#39;darkred&#39;, size=1.1) + 
  theme_bw() +
  theme(panel.grid.major.x = element_blank(),
        panel.grid.minor.x = element_blank(),
        panel.grid.major.y = element_blank(),
        panel.grid.minor.y = element_blank(),
        panel.border = element_rect(colour = &#39;black&#39;, size = 1.25),
        text = element_text(size=18),
        axis.ticks.length = unit(6, &quot;mm&quot;),
        axis.ticks = element_line(size = 1),
        # axis.text.x = element_text(margin=margin(2,2,5,5,&quot;mm&quot;)),
        # axis.text.y = element_text(margin=margin(2,2,5,5,&quot;mm&quot;)),
        # axis.title = element_blank(),
        axis.text = element_text(color = &quot;black&quot;)
        # axis.text=element_blank()
        ) +
  scale_y_continuous(labels = scientific) +
 # scale_y_log10() + 
  xlab(&quot;Time (h)&quot;) +
  ylab(&quot;Mating efficiency&quot;)</code></pre>
<embed src="hvo_mating_files/figure-html/wet plots-6.pdf" width="576" type="application/pdf" />
<pre class="r"><code>ggsave(&quot;plots/mating_efficiency_nozs.pdf&quot;)</code></pre>
<pre><code>## Saving 6 x 3 in image</code></pre>
</div>
<div id="end" class="section level2">
<h2>end</h2>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
